<div class="container publications">
  <h1 class="center pageTitle">Publications</h1>
  <ul>
    <li>Posey J, et al. “Resolution of Disease Phenotypes Resulting from Multilocus Genomic Variation” NEJM (2017)</li>
    <li>D Smedley, et al. “Next-generation diagnostics and disease-gene discovery with the Exomiser” Nature Protocols (2015)</li>
    <li>N Akawi, et al. “Discovery of four recessive developmental disorders using probabilistic genotype and phenotype matching among 4,125 families” Nature Genetics (2015) 47</li>
    <li>A Kundaje, W Meuleman, J Ernst, M Bilenky et al. “Integrative analysis of 111 reference human epigenomes” Nature (2015) 518,317–330</li>
    <li>Zemojtel, Tomasz, et al. “Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome.” Science translational medicine 6.252 (2014): 252ra123-252ra123.</li>
    <li>Robinson, Peter N., et al. “Improved exome prioritization of disease genes through cross-species phenotype comparison.” Genome research 24.2 (2014): 340-348.</li>
    <li>Köhler, Sebastian, et al. “Clinical interpretation of CNVs with cross-species phenotype data.” Journal of medical genetics 51.11 (2014): 766-772.</li>
    <li>Bayés, Àlex, et al. “Characterization of the proteome, diseases and evolution of the human postsynaptic density.” Nature neuroscience 14.1 (2011): 19-21.</li>
    <li>Castellano, Sergi, et al. “Patterns of coding variation in the complete exomes of three Neandertals.” Proceedings of the National Academy of Sciences 111.18 (2014): 6666-6671.</li>
    <li>Corpas, Manuel, et al. “Interpretation of genomic copy number variants using DECIPHER.” Current Protocols in Human Genetics (2012): 8-14.</li>
    <li>Sifrim, Alejandro, et al. “eXtasy: variant prioritization by genomic data fusion.” Nature methods 10.11 (2013): 1083-1084.</li>
    <li>Lappalainen, Ilkka, et al. “DbVar and DGVa: public archives for genomic structural variation.” Nucleic acids research 41.D1 (2013): D936-D941.</li>
    <li>Firth, Helen V., and Caroline F. Wright. “The deciphering developmental disorders (DDD) study.” Developmental Medicine &amp; Child Neurology 53.8 (2011): 702-703.</li>
    <li>Welter, Danielle, et al. “The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.” Nucleic acids research 42.D1 (2014): D1001-D1006.</li>
    <li>Zhou, XueZhong, et al. “Human symptoms–disease network.” Nature communications 5 (2014).</li>
    <li>Renkema, Kirsten Y., et al. “Next-generation sequencing for research and diagnostics in kidney disease.” Nature Reviews Nephrology (2014).</li>
    <li>Cutting, Garry R. “Annotating DNA Variants Is the Next Major Goal for Human Genetics.” The American Journal of Human Genetics 94.1 (2014): 5-10.</li>
    <li>Köhler, Sebastian, et al. “Clinical diagnostics in human genetics with semantic similarity searches in ontologies.” The American Journal of Human Genetics 85.4 (2009): 457-464.</li>
    <li>Amberger, Joanna, Carol Bocchini, and Ada Hamosh. “A new face and new challenges for Online Mendelian Inheritance in Man (OMIM®).” Human mutation 32.5 (2011): 564-567.</li>
    <li>de Lima Morais, David A., et al. “SUPERFAMILY 1.75 including a domain-centric gene ontology method.” Nucleic acids research (2010): gkq1130.</li>
    <li>Li, Mulin Jun, et al. “GWASdb: a database for human genetic variants identified by genome-wide association studies.” Nucleic acids research (2011): gkr1182.</li>
    <li>Chen, Chao‐Kung, et al. “MouseFinder: candidate disease genes from mouse phenotype data.” Human mutation 33.5 (2012): 858-866.</li>
    <li>Boycott, Kym M., et al. “Rare-disease genetics in the era of next-generation sequencing: discovery to translation.” Nature Reviews Genetics 14.10 (2013): 681-691.</li>
    <li>de Leeuw, Nicole, et al. “Diagnostic interpretation of array data using public databases and internet sources.” Human mutation 33.6 (2012): 930-940.</li>
    <li>Mottaz, Anaïs, et al. “Easy retrieval of single amino-acid polymorphisms and phenotype information using SwissVar.” Bioinformatics 26.6 (2010): 851-852.</li>
    <li>Hochheiser, Harry, et al. “The FaceBase Consortium: a comprehensive program to facilitate craniofacial research.” Developmental biology 355.2 (2011): 175-182.</li>
    <li>Riggs, Erin Rooney, et al. “Phenotypic information in genomic variant databases enhances clinical care and research: the International Standards for Cytogenomic Arrays Consortium experience.” Human mutation 33.5 (2012): 787-796.</li>
    <li>Landrum, Melissa J., et al. “ClinVar: public archive of relationships among sequence variation and human phenotype.” Nucleic acids research (2013): gkt1113.</li>
    <li>Rubinstein, Wendy S., et al. “The NIH genetic testing registry: a new, centralized database of genetic tests to enable access to comprehensive information and improve transparency.” Nucleic acids research (2012): gks1173.</li>
    <li>Doelken, Sandra C., et al. “Phenotypic overlap in the contribution of individual genes to CNV pathogenicity revealed by cross-species computational analysis of single-gene mutations in humans, mice and zebrafish.” Disease models &amp; mechanisms 6.2 (2013): 358-372.</li>
    <li>Poot, Martin, and Ron Hochstenbach. “A three-step workflow procedure for the interpretation of array-based comparative genome hybridization results in patients with idiopathic mental retardation and congenital anomalies.” Genetics in Medicine 12.8 (2010): 478-485.</li>
    <li>Hamosh, Ada, et al. “PhenoDB: A New Web‐Based Tool for the Collection, Storage, and Analysis of Phenotypic Features.” Human mutation 34.4 (2013): 566-571.</li>
    <li>Vulto‐van Silfhout, Anneke T., et al. “Clinical Significance of De Novo and Inherited Copy‐Number Variation.” Human mutation 34.12 (2013): 1679-1687.</li>
    <li>Girdea, Marta, et al. “Phenotips: Patient phenotyping software for clinical and research use.” Human mutation 34.8 (2013): 1057-1065.</li>
    <li>Haworth, Andrea, et al. “Call for participation in the neurogenetics consortium within the Human Variome Project.” neurogenetics 12.3 (2011): 169-173.</li>
    <li>Vulto-van Silfhout, Anneke T., et al. “An update on ECARUCA, the European cytogeneticists association register of unbalanced chromosome aberrations.” European journal of medical genetics 56.9 (2013): 471-474.</li>
    <li>Singleton, Marc V., et al. “Phevor Combines Multiple Biomedical Ontologies for Accurate Identification of Disease-Causing Alleles in Single Individuals and Small Nuclear Families.” The American Journal of Human Genetics 94.4 (2014): 599-610.</li>
    <li>Wu, Jiaxin, Yanda Li, and Rui Jiang. “Integrating multiple genomic data to predict disease-causing nonsynonymous single nucleotide variants in exome sequencing studies.” PLoS genetics 10.3 (2014): e1004237.</li>
    <li>Rehm, Heidi L. “Disease-targeted sequencing: a cornerstone in the clinic.” Nature Reviews Genetics 14.4 (2013): 295-300.</li>
    <li>Masuya, Hiroshi, et al. “The RIKEN integrated database of mammals.” Nucleic acids research 39.suppl 1 (2011): D861-D870.</li>
    <li>Köhler, Sebastian, et al. “Ontological phenotype standards for neurogenetics.” Human mutation 33.9 (2012): 1333-1339.</li>
    <li>Wain, Karen E., et al. “The laboratory-clinician team: A professional call to action to improve communication and collaboration for optimal patient care in chromosomal microarray testing.” Journal of genetic counseling 21.5 (2012): 631-637.</li>
    <li>Gottlieb, Assaf, et al. “PREDICT: a method for inferring novel drug indications with application to personalized medicine.” Molecular systems biology 7.1 (2011).</li>
    <li>Reese, Martin G., et al. “A standard variation file format for human genome sequences.” Genome Biol 11.8 (2010): R88.</li>
    <li>Grant, Seth GN. “Synaptopathies: diseases of the synaptome.” Current opinion in neurobiology 22.3 (2012): 522-529.</li>
    <li>de Bono, Bernard, et al. “The RICORDO approach to semantic interoperability for biomedical data and models: strategy, standards and solutions.” BMC Research Notes 4.1 (2011): 313.</li>
    <li>de Bono, Bernard, Pierre Grenon, and Stephen John Sammut. “ApiNATOMY: A novel toolkit for visualizing multiscale anatomy schematics with phenotype‐related information.” Human mutation 33.5 (2012): 837-848.</li>
    <li>Carss, Keren J., et al. “Exome sequencing improves genetic diagnosis of structural fetal abnormalities revealed by ultrasound.” Human molecular genetics 23.12 (2014): 3269-3277.</li>
    <li>Robinson, Peter N., et al. “The Human Phenotype Ontology: a tool for annotating and analyzing human hereditary disease.” The American Journal of Human Genetics 83.5 (2008): 610-615.</li>
    <li>Robinson, Peter N., and S. Mundlos. “The human phenotype ontology.” Clinical genetics 77.6 (2010): 525-534.</li>
    <li>Washington, Nicole L., et al. “Linking human diseases to animal models using ontology-based phenotype annotation.” PLoS biology 7.11 (2009): e1000247.</li>
    <li>Köhler, Sebastian, et al. “The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data.” Nucleic acids research(2013): gkt1026.</li>
    <li>Mungall, Christopher J., et al. “Integrating phenotype ontologies across multiple species.” Genome biology 11.1 (2010): R2.</li>
    <li>Gkoutos, Georgios V., et al. “Entity/quality-based logical definitions for the human skeletal phenome using PATO.” Engineering in Medicine and Biology Society, 2009. EMBC 2009. Annual International Conference of the IEEE. IEEE, 2009.</li>
    <li>Schindelman, Gary, et al. “Worm Phenotype Ontology: integrating phenotype data within and beyond the C. elegans community.” BMC bioinformatics 12.1 (2011): 32.</li>
    <li>Reese, Martin G., et al. “A standard variation file format for human genome sequences.” Genome Biol 11.8 (2010): R88.</li>
    <li>Hoehndorf, Robert, Paul N. Schofield, and Georgios V. Gkoutos. “PhenomeNET: a whole-phenome approach to disease gene discovery.”Nucleic acids research 39.18 (2011): e119-e119.</li>
    <li>Schriml, Lynn Marie, et al. “Disease Ontology: a backbone for disease semantic integration.” Nucleic acids research 40.D1 (2012): D940-D946.</li>
    <li>Oti, Martin, Martijn A. Huynen, and Han G. Brunner. “The biological coherence of human phenome databases.” The American Journal of Human Genetics 85.6 (2009): 801-808.</li>
    <li>Shimoyama, Mary, et al. “Three ontologies to define phenotype measurement data.” Bioinformatics and Computational Biology 3 (2012): 87.</li>
    <li>Dahdul, Wasila M., et al. “Evolutionary characters, phenotypes and ontologies: curating data from the systematic biology literature.” PLoS One 5.5 (2010): e10708.</li>
    <li>Schlicker, Andreas, Thomas Lengauer, and Mario Albrecht. “Improving disease gene prioritization using the semantic similarity of Gene Ontology terms.”Bioinformatics 26.18 (2010): i561-i567.</li>
    <li>Tiffin, Nicki, Miguel A. Andrade-Navarro, and Carolina Perez-Iratxeta. “Linking genes to diseases: it’s all in the data.” Genome Med 1.8 (2009): 77.</li>
    <li>Moreau, Yves, and Léon-Charles Tranchevent. “Computational tools for prioritizing candidate genes: boosting disease gene discovery.” Nature Reviews Genetics 13.8 (2012): 523-536.</li>
    <li>Mungall, Christopher J., et al. “Uberon, an integrative multi-species anatomy ontology.” Genome Biol 13.1 (2012): R5.</li>
    <li>Li, Yongjin, and Jagdish C. Patra. “Genome-wide inferring gene–phenotype relationship by walking on the heterogeneous network.” Bioinformatics 26.9 (2010): 1219-1224.</li>
    <li>Köhler, Sebastian, et al. “Improving ontologies by automatic reasoning and evaluation of logical definitions.” BMC bioinformatics 12.1 (2011): 418.</li>
    <li>Wang, Jing, et al. “WEB-based GEne SeT AnaLysis Toolkit (WebGestalt): update 2013.” Nucleic acids research (2013): gkt439.</li>
    <li>Wang, Xiujuan, Natali Gulbahce, and Haiyuan Yu. “Network-based methods for human disease gene prediction.” Briefings in functional genomics 10.5 (2011): 280-293.</li>
    <li>Oellrich, Anika, et al. “Improving disease gene prioritization by comparing the semantic similarity of phenotypes in mice with those of human diseases.” PloS one 7.6 (2012): e38937.</li>
    <li>Fang, Hai, and Julian Gough. “dcGO: database of domain-centric ontologies on functions, phenotypes, diseases and more.” Nucleic acids research 41.D1 (2013): D536-D544.</li>
    <li>Hoehndorf, Robert, et al. “Semantic integration of physiology phenotypes with an application to the Cellular Phenotype Ontology.” Bioinformatics 28.13 (2012): 1783-1789.</li>
    <li>Schofield, Paul N., et al. “New approaches to the representation and analysis of phenotype knowledge in human diseases and their animal models.” Briefings in functional genomics 10.5 (2011): 258-265.</li>
    <li>Robinson, Peter N., P. Krawitz, and S. Mundlos. “Strategies for exome and genome sequence data analysis in disease‐gene discovery projects.” Clinical genetics 80.2 (2011): 127-132.</li>
    <li>Shihab, Hashem A., et al. “Predicting the functional, molecular, and phenotypic consequences of amino acid substitutions using hidden Markov models.”Human mutation 34.1 (2013): 57-65.</li>
  </ul>
</div>

