<div class="container faq">
<div class="row">
  <div class="col-lg-12">
      <h1 class="center pageTitle">Frequently Asked Questions</h1>
      <br>
      <h3>
        HPO
      </h3>
    <mat-accordion>
      <mat-expansion-panel [expanded]="openPanel == 1" id = "hpo-relation">
        <mat-expansion-panel-header>
          <mat-panel-title>
            <strong>How is HPO related to the Monarch Initiative?</strong>
          </mat-panel-title>
        </mat-expansion-panel-header>
        <p>The HPO is the clinical window of the <a href="https://monarchinitiative.org/" target="__blank">Monarch Initiative</a> which is an NIH supported project that integrates, aligns, and re-distributes
          cross-species gene, genotype, variant, disease, and phenotype data. <a href="https://monarchinitiative.org/" target="__blank">The Monarch Browser</a> allows users to browse not only human diseases but to see their links to other organisms, genes, phenotypes, and pathways, which
          may be of especial interest to translational researchers.</p>
      </mat-expansion-panel>
      <mat-expansion-panel [expanded]="openPanel == 2" id = "content-change">
        <mat-expansion-panel-header>
          <mat-panel-title>
            <strong>Am I allowed to change the contents of HPO?</strong>
          </mat-panel-title>
        </mat-expansion-panel-header>
        <p>The HPO is a community-driven project. We request that all proposed changes be suggested on our <a target="__blank" href="https://github.com/obophenotype/human-phenotype-ontology/issues">GitHub tracker</a> so that they will be of use to all users of the HPO. </p>
      </mat-expansion-panel>
      <mat-expansion-panel [expanded]="openPanel == 3" id = "collaboration">
        <mat-expansion-panel-header>
          <mat-panel-title>
            <strong>How to collaborate with the HPO ?</strong>
          </mat-panel-title>
        </mat-expansion-panel-header>
        <p>We welcome the participation of interested colleagues. We anticipate that the structure of the HPO will continue to be refined and completed for some time to come.
          Groups or persons with expert knowledge in a particular domain of human phenotyping in a medical genetics setting are invited to contribute their knowledge on a collaborative basis.
          Please contact us to discuss details.</p>
        <p>We are currently transitioning from annotations derived in a semiautomated fashion from the OMIM database to manually curated annotations. Given the number of human hereditary diseases,
          this is no small task. We encourage participation of colleagues with an interest in specific hereditary diseases to improve the annotations for ‘their’ disease. Please get in contact with us to discuss this form of collaboration.
          We are also preparing an online system for manual annotations.</p>
        <p>The issue tracker is <a href="https://github.com/obophenotype/human-phenotype-ontology/issues" target="__blank">available on Github.</a></p>
        <h4>Collaborations for translations</h4>
        <p>We are working with several people to translate more content of HPO into a multitude of languages. Several translations are being actively developed at the moment (e.g. german).</p>
        <p>The translations are considered a project coordinated by the HPO team and are hosted on <a href="https://github.com/Phenomics/HPO-translations" target="__blank">GitHub</a>. Please do not start translating HPO content without contacting us beforehand.
          This might cause duplicated translations and other problems. </p> <p> Please contact us <a routerLink="/contact">here</a>.</p>
      </mat-expansion-panel>
    </mat-accordion>
      <h3>
        General
      </h3>
    <mat-accordion>
      <mat-expansion-panel [expanded]="openPanel == 4" id = "ontology">
        <mat-expansion-panel-header>
          <mat-panel-title>
            <strong>What is an ontology?</strong>
          </mat-panel-title>
        </mat-expansion-panel-header>
        <p>
          A terminology contains a list of items (often called "terms") that represent the concepts of a domain. An ontology is a terminology that additionally specifies formal semantic relationships of concepts.
          For instance, a terminology of wines would include a list of items such as Chardonnay, Cabernet Sauvignon, Rioja, Riesling, Sauvignon blanc, etc., but an ontology of wine might have a term for "red wine" and "Rioja"; It also can specify that "Rioja" is a type of "red wine".
          Ontologies and terminologies can be used as tools for standardizing and exchanging data. Ontologies such as the HPO typically enable sophisticated computational algorithms that exploit the semantic relations between terms.
        </p>
      </mat-expansion-panel>
    </mat-accordion>
      <h3>
        Annotation
      </h3>
      <mat-accordion>

        <mat-expansion-panel [expanded]="openPanel == 5" id = "annotation-use">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How do I use HPO Annotations?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>The terms of the HPO describe phenotypic abnormalities and do not directly describe diseases. Instead, the HPO project uses so-called annotations to describe the connection between diseases and HPO terms. For instance, the disease,
            <a routerLink="/browse/disease/ORPHA:171719">Cutis laxa-Marfanoid syndrome </a> is annotated by the HPO terms for <a routerLink="/browse/term/HP:0002827">Hip dislocation</a>, <a routerLink="/browse/term/HP:0001371">Flexion contracture</a> and many more.</p>
        </mat-expansion-panel>

        <mat-expansion-panel [expanded]="openPanel == 6" id="genes-phenotypes">
          <mat-expansion-panel-header>
              <mat-panel-title>
              <strong>What is the difference between genes_to_phenotypes.txt and phenotype_to_genes.txt ?</strong>
              </mat-panel-title>
          </mat-expansion-panel-header>
          <p>A question that is asked very often, deals with the asymmetry of the files:</p>
          <ul>
            <li><i>genes_to_phenotypes.txt</i></li>
            <li><i>phenotype_to_genes.txt</i></li>
          </ul>
          <p>The difference is that one file uses the transitivity of the annotations and the other does not. I.e. in <i>genes_to_phenotypes.txt</i> we list for each gene the most specific HPO-classes (and not all the ancestors).
            In <i>phenotype_to_genes.txt</i> we show each phenotype that has at least one gene associated with it. Additionally, we have mapped genes to the ancestor classes of each associated phenotype. For example given that:</p>
          <ul>
            <li>HP_1 subclass_of HP_3</li>
            <li>HP_1 subclass_of HP_3</li>
          </ul>
          <p>then <i>genes_to_phenotypes.txt</i> </p>
          <ul>
            <li><strong>geneA</strong> annotated to HP_1</li>
            <li><strong>geneB</strong> annotated to HP_2</li>
          </ul>
          <p>then <i>phenotype_to_genes.txt</i> contains:</p>
          <ul>
            <li>HP_3 annotates <strong>geneA</strong> and <strong>geneB</strong></li>
          </ul>
        </mat-expansion-panel>

        <mat-expansion-panel [expanded]="openPanel == 7" id = "frequencies">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>What is the difference between “ALL_FREQUENCIES” and “FREQUENT_FEATURES”?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>Each annotation has a frequency associated, i.e. how many patients with that disease have this clinical feature. “ALL_FREQUENCIES” denotes that we do not use any filter.
            “FREQUENT_FEATURES” means that we exclude all features seen in less than 50% of the patients.</p>
         </mat-expansion-panel>

          <mat-expansion-panel [expanded]="openPanel == 8" id = "association-rules">
              <mat-expansion-panel-header>
                <mat-panel-title>
                  <strong>What is your procedure for associating genes with HPO phenotypes?</strong>
                </mat-panel-title>
              </mat-expansion-panel-header>
              <p>The following file is used to associate genes with phenotypes.</p>
              <ul>
                <li>
                  <i>genes_to_phenotypes.txt</i>
                </li>
              </ul>
              <p>
                  In many cases, there are multiple diseases that are associated with mutations in the same gene (for instance, mutations in the LMNA gene are associated with multiple diseases including Cardiomyopathy,
                  dilated, 1A [MIM:115200], Charcot-Marie-Tooth disease, type 2B1 [605588], Emery-Dreifuss muscular dystrophy 2, AD [181350], and Hutchinson-Gilford progeria [176670]. Each of these diseases is associated with a
                  different spectrum of phenotypic features. The gene-to-phenotype association in our databases takes the set of all phenotypic features (HPO terms) associated with one or more diseases caused by mutation in the gene under
                  consideration. Thus, LMNA is associated with Alopecia (which is a feature of Hutchinson-Gilford progeria) as well as Ventricular arrhythmia (which is a feature of Cardiomyopathy, dilated, 1A).
              </p>
          </mat-expansion-panel>
      </mat-accordion>
   </div>
  </div>
</div>
