"use strict";(self.webpackChunkhpo=self.webpackChunkhpo||[]).push([[381],{7734:(W,C,r)=>{r.r(C),r.d(C,{DataModule:()=>p});var n=r(6895),T=r(2474),o=r(4006),t=r(3917),e=r(4650);let q=(()=>{class g{constructor(){}ngOnInit(){}}return g.\u0275fac=function(d){return new(d||g)},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-annotation"]],decls:544,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://phenotips.org/","target","__blank"],["href","http://www.patientarchive.org/#/home","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/30476213","target","_blank"],["align","center",1,"table-spacer"],["routerLink","/browse/term/HP:0003577"],["routerLink","/browse/term/HP:0025257"],[1,"no-indent-list"],["href","https://omim.org/","target","__blank"],["href","http://www.orpha.net/consor/cgi-bin/index.php","target","__blank"],["href","https://decipher.sanger.ac.uk/","target","__blank"],["routerLink","/browse/term/HP:0003674"],["routerLink","/browse/term/HP:0040279"],["routerLink","/browse/term/HP:0012823"],["href","https://hpo-annotation-qc.readthedocs.io/en/latest/annotationFormat.html#phenotype-hpoa-format","target","_blank"],["href","https://www.ebi.ac.uk/ols/ontologies/mondo","target","__blank"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Annotation & File Format"),e.qZA(),e.TgZ(3,"p"),e._uU(4," In addition to an ontology of phenotypic abnormalities, the HPO project provides a large set of phenotype annotations, i.e. statements that link a particular term from the HPO to specific diseases or genes. These annotations are made for the most appropriate (specific) term of the HPO, as all of the ancestor terms are implicitly annotated as well. This means that if a patient has "),e.TgZ(5,"em"),e._uU(6,"Aculeiform cataract"),e.qZA(),e._uU(7," (HP:0010926), the patient can also be said to have the parent term of Aculeiform cataract, viz., "),e.TgZ(8,"em"),e._uU(9,"Nuclear cataract"),e.qZA(),e._uU(10,' (HP:0100018) as well as the "grandmother" and "great-grandmother" terms '),e.TgZ(11,"em"),e._uU(12,"Zonular cataract"),e.qZA(),e._uU(13," (HP:0010920) and "),e.TgZ(14,"em"),e._uU(15,"Cataract"),e.qZA(),e._uU(16,' (HP:0000518). This is a necessary consequence of the way the HPO-based algorithms were developed, but it also reflects the "fuzziness" that is frequently encountered in clinical medicine. For instance, a general practitioner might make the diagnosis of Cataract and refer a patient to an ophthalmologist who would make the more precise diagnosis of Aculeiform cataract. '),e.qZA(),e.TgZ(17,"p"),e._uU(18,"Users of HPO-based software such as Phenomizer and Exomiser will see the disease annotations displayed in various contexts. This page describes the format of the file provided by the HPO project for download and will mainly be of interest to bioinformaticians or computer scientists who want to use the file for their own analysis. "),e.qZA(),e.TgZ(19,"p"),e._uU(20,"We represent clinical annotations using a simple tab-delimited format. This document describes the process of assigning HPO terms to disease entities. Each line in the annotation file represents a link between a disease entity such as "),e.TgZ(21,"i"),e._uU(22,"Noonan syndrome"),e.qZA(),e._uU(23," and one of the clinical features characteristically seen in that disease. Each of the features of a disease is to be listed on a separate line. Note that this file (and format) is intended to be used for the annotation of disease entities (e.g."),e.TgZ(24,"i"),e._uU(25,"Noonan syndrome"),e.qZA(),e._uU(26,") and not individuals (such as a person who has been diagnosed with "),e.TgZ(27,"i"),e._uU(28,"Noonan syndrome"),e.qZA(),e._uU(29,"). We recommend looking at "),e.TgZ(30,"a",2),e._uU(31,"PhenoTips"),e.qZA(),e._uU(32," or "),e.TgZ(33,"a",3),e._uU(34,"Patient Archive"),e.qZA(),e._uU(35," if you are aiming at annotating clinical findings of individuals with hereditary diseases."),e.qZA(),e.TgZ(36,"h3"),e._uU(37,"Current Annotation File format (2019 onwards)"),e.qZA(),e._uU(38," The HPO transitioned to a new annotation file format in 2019 (Details in "),e.TgZ(39,"a",4),e._uU(40," K\xf6hler et al, 2019"),e.qZA(),e._uU(41,"). "),e.TgZ(42,"table",5)(43,"thead")(44,"tr")(45,"th"),e._uU(46,"Column"),e.qZA(),e.TgZ(47,"th"),e._uU(48,"Content"),e.qZA(),e.TgZ(49,"th"),e._uU(50,"Required"),e.qZA(),e.TgZ(51,"th"),e._uU(52,"Example"),e.qZA()()(),e.TgZ(53,"tbody")(54,"tr")(55,"td"),e._uU(56,"1"),e.qZA(),e.TgZ(57,"td"),e._uU(58,"DatabaseId"),e.qZA(),e.TgZ(59,"td"),e._uU(60,"required"),e.qZA(),e.TgZ(61,"td"),e._uU(62,"MIM:154700"),e.qZA()(),e.TgZ(63,"tr")(64,"td"),e._uU(65,"2"),e.qZA(),e.TgZ(66,"td"),e._uU(67,"DB_Name"),e.qZA(),e.TgZ(68,"td"),e._uU(69,"required"),e.qZA(),e.TgZ(70,"td"),e._uU(71,"Achondrogenesis, type IB"),e.qZA()(),e.TgZ(72,"tr")(73,"td"),e._uU(74,"3"),e.qZA(),e.TgZ(75,"td"),e._uU(76,"Qualifier"),e.qZA(),e.TgZ(77,"td"),e._uU(78,"optional"),e.qZA(),e.TgZ(79,"td"),e._uU(80,"NOT"),e.qZA()(),e.TgZ(81,"tr")(82,"td"),e._uU(83,"4"),e.qZA(),e.TgZ(84,"td"),e._uU(85,"HPO_ID"),e.qZA(),e.TgZ(86,"td"),e._uU(87,"required"),e.qZA(),e.TgZ(88,"td"),e._uU(89,"HP:0002487"),e.qZA()(),e.TgZ(90,"tr")(91,"td"),e._uU(92,"5"),e.qZA(),e.TgZ(93,"td"),e._uU(94,"DB_Reference"),e.qZA(),e.TgZ(95,"td"),e._uU(96,"required"),e.qZA(),e.TgZ(97,"td"),e._uU(98,"OMIM:154700 or PMID:15517394"),e.qZA()(),e.TgZ(99,"tr")(100,"td"),e._uU(101,"6"),e.qZA(),e.TgZ(102,"td"),e._uU(103,"Evidence_Code"),e.qZA(),e.TgZ(104,"td"),e._uU(105,"required"),e.qZA(),e.TgZ(106,"td"),e._uU(107,"IEA"),e.qZA()(),e.TgZ(108,"tr")(109,"td"),e._uU(110,"7"),e.qZA(),e.TgZ(111,"td"),e._uU(112,"Onset modifier"),e.qZA(),e.TgZ(113,"td"),e._uU(114,"optional"),e.qZA(),e.TgZ(115,"td"),e._uU(116,"HP:0003577"),e.qZA()(),e.TgZ(117,"tr")(118,"td"),e._uU(119,"8"),e.qZA(),e.TgZ(120,"td"),e._uU(121,"Frequency"),e.qZA(),e.TgZ(122,"td"),e._uU(123,"optional"),e.qZA(),e.TgZ(124,"td")(125,"a",6),e._uU(126,"HP:0003577"),e.qZA(),e._uU(127," or 12/45 or 22%"),e.qZA()(),e.TgZ(128,"tr")(129,"td"),e._uU(130,"9"),e.qZA(),e.TgZ(131,"td"),e._uU(132,"Sex"),e.qZA(),e.TgZ(133,"td"),e._uU(134,"optional"),e.qZA(),e.TgZ(135,"td"),e._uU(136,"MALE or FEMALE"),e.qZA()(),e.TgZ(137,"tr")(138,"td"),e._uU(139,"10"),e.qZA(),e.TgZ(140,"td"),e._uU(141,"Modifier"),e.qZA(),e.TgZ(142,"td"),e._uU(143,"optional"),e.qZA(),e.TgZ(144,"td")(145,"a",7),e._uU(146,"HP:0025257"),e.qZA(),e._uU(147,"(\u201c;\u201d-separated list)"),e.qZA()(),e.TgZ(148,"tr")(149,"td"),e._uU(150,"11"),e.qZA(),e.TgZ(151,"td"),e._uU(152,"Aspect"),e.qZA(),e.TgZ(153,"td"),e._uU(154,"required"),e.qZA(),e.TgZ(155,"td"),e._uU(156,"P, C, I, or M"),e.qZA()(),e.TgZ(157,"tr")(158,"td"),e._uU(159,"12"),e.qZA(),e.TgZ(160,"td"),e._uU(161,"BiocurationBy"),e.qZA(),e.TgZ(162,"td"),e._uU(163,"required"),e.qZA(),e.TgZ(164,"td"),e._uU(165,"HPO:skoehler[YYYY-MM-DD]"),e.qZA()()()(),e.TgZ(166,"ol",8)(167,"li")(168,"strong"),e._uU(169,"DatabaseId"),e.qZA(),e.TgZ(170,"p"),e._uU(171,"This field refers to the database and database identifier of the disease being annotated. Annotations from "),e.TgZ(172,"a",9),e._uU(173,"OMIM"),e.qZA(),e._uU(174,", "),e.TgZ(175,"a",10),e._uU(176,"Orphanet"),e.qZA(),e._uU(177,", "),e.TgZ(178,"a",11),e._uU(179,"DECIPHER"),e.qZA(),e._uU(180," are currently available. "),e.qZA()(),e.TgZ(181,"li")(182,"strong"),e._uU(183,"Qualifer:"),e.qZA(),e.TgZ(184,"p"),e._uU(185,"This optional field can be used to qualify the annotation shown in field 5. The only allowed values of this field are NOT and (empty string). "),e.TgZ(186,"em"),e._uU(187,"NOT"),e.qZA(),e._uU(188,": The disorder being annotated is NOT characterized by the feature associated with HPO_ID in column 5. Note that annotations with the NOT modifier are moved to separate file!"),e.qZA()(),e.TgZ(189,"li")(190,"strong"),e._uU(191,"HPO_ID:"),e.qZA(),e.TgZ(192,"p"),e._uU(193,"This field is for the HPO identifier for the term attributed to the DB_Object_ID. This field is mandatory, cardinality 1."),e.qZA()(),e.TgZ(194,"li")(195,"strong"),e._uU(196,"DB_Reference:"),e.qZA(),e.TgZ(197,"p"),e._uU(198,"This required field indicates the source of the information used for the annotation. This may be the clinical experience of the annotator or may be taken from an article as indicated by a PubMed id. Each collaborating center of the Human Phenotype Ontology consortium is assigned a HPO:Ref id. In addition, if appropriate, a PubMed id for an article describing the clinical abnormality may be used."),e.qZA()(),e.TgZ(199,"li")(200,"strong"),e._uU(201,"Evidence:"),e.qZA(),e.TgZ(202,"p"),e._uU(203,"This required field indicates the level of evidence supporting the annotation. Annotations that have been extracted by parsing the Clinical Features sections of the omim.txt file are assigned the evidence code "),e.TgZ(204,"strong"),e._uU(205,"IEA"),e.qZA(),e._uU(206," (inferred from electronic annotation). Please note that you need to contact OMIM in order to reuse these annotations in other software products. Other codes include "),e.TgZ(207,"strong"),e._uU(208,"PCS"),e.qZA(),e._uU(209," for published clinical study. This should be used for information extracted from articles in the medical literature. Generally, annotations of this type will include the PubMed id of the published study in the DB_Reference field. "),e.TgZ(210,"strong"),e._uU(211,"ICE"),e.qZA(),e._uU(212," can be used for annotations based on individual clinical experience. This may be appropriate for disorders with a limited amount of published data. This must be accompanied by an entry in the DB:Reference field denoting the individual or center performing the annotation together with an identifier. For instance, GH:007 might be used to refer to the seventh such annotation made by a specialist from Gotham Hospital (assuming the prefix GH has been registered with the HPO). Finally we have "),e.TgZ(213,"strong"),e._uU(214,"TAS"),e.qZA(),e._uU(215,", which stands for \u201ctraceable author statement\u201d, usually reviews or disease entries (e.g. OMIM) that only refers to the original publication."),e.qZA()(),e.TgZ(216,"li")(217,"strong"),e._uU(218,"Onset"),e.qZA(),e.TgZ(219,"p"),e._uU(220,"A term-id from the HPO-sub-ontology below the term "),e.TgZ(221,"a",12),e._uU(222,"Age of onset"),e.qZA(),e._uU(223,". "),e.qZA()(),e.TgZ(224,"li")(225,"strong"),e._uU(226,"Frequency"),e.qZA(),e.TgZ(227,"p"),e._uU(228,"A term-id from the HPO-sub-ontology below the term "),e.TgZ(229,"a",13),e._uU(230,"Frequency"),e.qZA(),e._uU(231,"."),e.qZA(),e._uU(232," There are three allowed options for this field. "),e.TgZ(233,"ol")(234,"li"),e._uU(235," A term-id from the HPO-sub-ontology below the term "),e.TgZ(236,"a",13),e._uU(237,"Frequency"),e.qZA(),e._uU(238,". "),e.qZA(),e.TgZ(239,"li"),e._uU(240," A count of patients affected within a cohort. For instance, 7/13 would indicate that 7 of the 13 patients with the specified disease were found to have the phenotypic abnormality referred to by the HPO term in question in the study referred to by the DB_Reference "),e.qZA(),e.TgZ(241,"li"),e._uU(242," A percentage value such as 17%, again referring to the percentage of patients found to have the phenotypic abnormality referred to by the HPO term in question in the study referred to by the DB_Reference. If possible, the 7/13 format is preferred over the percentage format if the exact data is available. "),e.qZA()()(),e.TgZ(243,"li")(244,"strong"),e._uU(245,"Sex:"),e.qZA(),e.TgZ(246,"p"),e._uU(247,"This field contains the strings MALE or FEMALE if the annotation in question is limited to males or females. This field refers to the phenotypic (and not the chromosomal) sex, and does not intend to capture the further complexities of sex determination. If a phenotype is limited to one or the other sex, then the corresponding term from the "),e.TgZ(248,"a",14),e._uU(249,"Clinical modifier"),e.qZA(),e._uU(250," subontology should also be used in the Modifier field."),e.qZA()(),e.TgZ(251,"li")(252,"strong"),e._uU(253,"Modifier:"),e.qZA(),e.TgZ(254,"p"),e._uU(255,"A term from the "),e.TgZ(256,"a",14),e._uU(257,"Clinical modifier"),e.qZA(),e._uU(258," subontology"),e.qZA()(),e.TgZ(259,"li")(260,"strong"),e._uU(261,"Aspect:"),e.qZA(),e.TgZ(262,"p"),e._uU(263,"one of P (Phenotypic abnormality), I (inheritance), C (onset and clinical course). This field is mandatory; cardinality 1"),e.qZA()(),e.TgZ(264,"li")(265,"strong"),e._uU(266,"BiocurationBy"),e.qZA(),e.TgZ(267,"p"),e._uU(268,"This refers to the center or user making the annotation and the date on which the annotation was made; format is YYYY-MM-DD this field is mandatory. Multiple entries can be separated by a semicolon if an annotation was revised, e.g., HPO:skoehler[2010-04-21];HPO:lcarmody[2019-06-02]"),e.qZA()()(),e.TgZ(269,"p"),e._uU(270,"See the "),e.TgZ(271,"a",15),e._uU(272,"online documentation"),e.qZA(),e._uU(273," for more information about this format."),e.qZA(),e.TgZ(274,"h3"),e._uU(275,"2009-2018 Annotation file format"),e.qZA(),e.TgZ(276,"p"),e._uU(277,"We will maintain the previous annotation format for the foreseeable future as a service to our users but ask users to update their pipelines to the new format."),e.qZA(),e.TgZ(278,"p")(279,"i"),e._uU(280,"The flat file format comprises 14 tab-delimited fields. Please note there are several differences compared with our previous annotation file format that was in use from 2009 to March, 2018."),e.qZA()(),e.TgZ(281,"table",5)(282,"thead")(283,"tr")(284,"th"),e._uU(285,"Column"),e.qZA(),e.TgZ(286,"th"),e._uU(287,"Content"),e.qZA(),e.TgZ(288,"th"),e._uU(289,"Required"),e.qZA(),e.TgZ(290,"th"),e._uU(291,"Example"),e.qZA()()(),e.TgZ(292,"tbody")(293,"tr")(294,"td"),e._uU(295,"1"),e.qZA(),e.TgZ(296,"td"),e._uU(297,"DB"),e.qZA(),e.TgZ(298,"td"),e._uU(299,"required"),e.qZA(),e.TgZ(300,"td"),e._uU(301,"OMIM, ORPHA, DECIPHER, MONDO"),e.qZA()(),e.TgZ(302,"tr")(303,"td"),e._uU(304,"2"),e.qZA(),e.TgZ(305,"td"),e._uU(306,"DB_Object_ID"),e.qZA(),e.TgZ(307,"td"),e._uU(308,"required"),e.qZA(),e.TgZ(309,"td"),e._uU(310,"154700"),e.qZA()(),e.TgZ(311,"tr")(312,"td"),e._uU(313,"3"),e.qZA(),e.TgZ(314,"td"),e._uU(315,"DB_Name"),e.qZA(),e.TgZ(316,"td"),e._uU(317,"required"),e.qZA(),e.TgZ(318,"td"),e._uU(319,"Achondrogenesis, type IB"),e.qZA()(),e.TgZ(320,"tr")(321,"td"),e._uU(322,"4"),e.qZA(),e.TgZ(323,"td"),e._uU(324,"Qualifier"),e.qZA(),e.TgZ(325,"td"),e._uU(326,"optional"),e.qZA(),e.TgZ(327,"td"),e._uU(328,"NOT"),e.qZA()(),e.TgZ(329,"tr")(330,"td"),e._uU(331,"5"),e.qZA(),e.TgZ(332,"td"),e._uU(333,"HPO_ID"),e.qZA(),e.TgZ(334,"td"),e._uU(335,"required"),e.qZA(),e.TgZ(336,"td"),e._uU(337,"HP:0002487"),e.qZA()(),e.TgZ(338,"tr")(339,"td"),e._uU(340,"6"),e.qZA(),e.TgZ(341,"td"),e._uU(342,"DB_Reference"),e.qZA(),e.TgZ(343,"td"),e._uU(344,"required"),e.qZA(),e.TgZ(345,"td"),e._uU(346,"OMIM:154700 or PMID:15517394"),e.qZA()(),e.TgZ(347,"tr")(348,"td"),e._uU(349,"7"),e.qZA(),e.TgZ(350,"td"),e._uU(351,"Evidence_Code"),e.qZA(),e.TgZ(352,"td"),e._uU(353,"required"),e.qZA(),e.TgZ(354,"td"),e._uU(355,"IEA"),e.qZA()(),e.TgZ(356,"tr")(357,"td"),e._uU(358,"8"),e.qZA(),e.TgZ(359,"td"),e._uU(360,"Onset modifier"),e.qZA(),e.TgZ(361,"td"),e._uU(362,"optional"),e.qZA(),e.TgZ(363,"td"),e._uU(364,"HP:0003577"),e.qZA()(),e.TgZ(365,"tr")(366,"td"),e._uU(367,"9"),e.qZA(),e.TgZ(368,"td"),e._uU(369,"Frequency"),e.qZA(),e.TgZ(370,"td"),e._uU(371,"optional"),e.qZA(),e.TgZ(372,"td")(373,"a",6),e._uU(374,"HP:0003577"),e.qZA(),e._uU(375," or 12/45 or 22%"),e.qZA()(),e.TgZ(376,"tr")(377,"td"),e._uU(378,"10"),e.qZA(),e.TgZ(379,"td"),e._uU(380,"Sex"),e.qZA(),e.TgZ(381,"td"),e._uU(382,"optional"),e.qZA(),e.TgZ(383,"td"),e._uU(384,"MALE or FEMALE"),e.qZA()(),e.TgZ(385,"tr")(386,"td"),e._uU(387,"11"),e.qZA(),e.TgZ(388,"td"),e._uU(389,"Modifier"),e.qZA(),e.TgZ(390,"td"),e._uU(391,"optional"),e.qZA(),e.TgZ(392,"td")(393,"a",7),e._uU(394,"HP:0025257"),e.qZA(),e._uU(395,"(\u201c;\u201d-separated list)"),e.qZA()(),e.TgZ(396,"tr")(397,"td"),e._uU(398,"12"),e.qZA(),e.TgZ(399,"td"),e._uU(400,"Aspect"),e.qZA(),e.TgZ(401,"td"),e._uU(402,"required"),e.qZA(),e.TgZ(403,"td"),e._uU(404,"P, C, or I"),e.qZA()(),e.TgZ(405,"tr")(406,"td"),e._uU(407,"13"),e.qZA(),e.TgZ(408,"td"),e._uU(409,"Date_Created"),e.qZA(),e.TgZ(410,"td"),e._uU(411,"required"),e.qZA(),e.TgZ(412,"td"),e._uU(413,"YYYY-MM-DD"),e.qZA()(),e.TgZ(414,"tr")(415,"td"),e._uU(416,"14"),e.qZA(),e.TgZ(417,"td"),e._uU(418,"Assigned_By"),e.qZA(),e.TgZ(419,"td"),e._uU(420,"required"),e.qZA(),e.TgZ(421,"td"),e._uU(422,"HPO"),e.qZA()()()(),e.TgZ(423,"ol",8)(424,"li")(425,"strong"),e._uU(426,"DB:"),e.qZA(),e.TgZ(427,"p"),e._uU(428,"This field refers to the database from which the identifier in DB_Object_ID (column 2) is drawn. Annotations from "),e.TgZ(429,"a",9),e._uU(430,"OMIM"),e.qZA(),e._uU(431,", "),e.TgZ(432,"a",10),e._uU(433,"Orphanet"),e.qZA(),e._uU(434,", "),e.TgZ(435,"a",11),e._uU(436,"DECIPHER"),e.qZA(),e._uU(437," are currently available. Annotations to diseases in "),e.TgZ(438,"a",16),e._uU(439,"Monarch Disease Ontology (MONDO)"),e.qZA(),e._uU(440," will soon be available. "),e.qZA()(),e.TgZ(441,"li")(442,"strong"),e._uU(443,"DB_Object_ID:"),e.qZA(),e.TgZ(444,"p"),e._uU(445,"This is the identifier of the annotated disease within the database indicated in column 1. Note that for OMIM identifiers, the symbol preceding the MIM number is omitted (*,#,+,%)."),e.qZA()(),e.TgZ(446,"li")(447,"strong"),e._uU(448,"DB_Name:"),e.qZA(),e.TgZ(449,"p"),e._uU(450,"This is the name of the disease associated with the DB_Object_ID in the database. Only the accepted name should be used, synonyms should not be listed here."),e.qZA()(),e.TgZ(451,"li")(452,"strong"),e._uU(453,"Qualifer:"),e.qZA(),e.TgZ(454,"p"),e._uU(455,"This optional field can be used to qualify the annotation shown in field 5. The only allowed values of this field are NOT and (empty string). "),e.TgZ(456,"em"),e._uU(457,"NOT"),e.qZA(),e._uU(458,": The disorder being annotated is NOT characterized by the feature associated with HPO_ID in column 5. Note that annotations with the NOT modifier are moved to separate file!"),e.qZA()(),e.TgZ(459,"li")(460,"strong"),e._uU(461,"HPO_ID:"),e.qZA(),e.TgZ(462,"p"),e._uU(463,"This field is for the HPO identifier for the term attributed to the DB_Object_ID. This field is mandatory, cardinality 1."),e.qZA()(),e.TgZ(464,"li")(465,"strong"),e._uU(466,"DB_Reference:"),e.qZA(),e.TgZ(467,"p"),e._uU(468,"This required field indicates the source of the information used for the annotation. This may be the clinical experience of the annotator or may be taken from an article as indicated by a PubMed id. Each collaborating center of the Human Phenotype Ontology consortium is assigned a HPO:Ref id. In addition, if appropriate, a PubMed id for an article describing the clinical abnormality may be used."),e.qZA()(),e.TgZ(469,"li")(470,"strong"),e._uU(471,"Evidence:"),e.qZA(),e.TgZ(472,"p"),e._uU(473,"This required field indicates the level of evidence supporting the annotation. Annotations that have been extracted by parsing the Clinical Features sections of the omim.txt file are assigned the evidence code "),e.TgZ(474,"strong"),e._uU(475,"IEA"),e.qZA(),e._uU(476," (inferred from electronic annotation). Please note that you need to contact OMIM in order to reuse these annotations in other software products. Other codes include "),e.TgZ(477,"strong"),e._uU(478,"PCS"),e.qZA(),e._uU(479," for published clinical study. This should be used for information extracted from articles in the medical literature. Generally, annotations of this type will include the PubMed id of the published study in the DB_Reference field. "),e.TgZ(480,"strong"),e._uU(481,"ICE"),e.qZA(),e._uU(482," can be used for annotations based on individual clinical experience. This may be appropriate for disorders with a limited amount of published data. This must be accompanied by an entry in the DB:Reference field denoting the individual or center performing the annotation together with an identifier. For instance, GH:007 might be used to refer to the seventh such annotation made by a specialist from Gotham Hospital (assuming the prefix GH has been registered with the HPO). Finally we have "),e.TgZ(483,"strong"),e._uU(484,"TAS"),e.qZA(),e._uU(485,", which stands for \u201ctraceable author statement\u201d, usually reviews or disease entries (e.g. OMIM) that only refers to the original publication."),e.qZA()(),e.TgZ(486,"li")(487,"strong"),e._uU(488,"Onset"),e.qZA(),e.TgZ(489,"p"),e._uU(490,"A term-id from the HPO-sub-ontology below the term "),e.TgZ(491,"a",12),e._uU(492,"Age of onset"),e.qZA(),e._uU(493,". "),e.qZA()(),e.TgZ(494,"li")(495,"strong"),e._uU(496,"Frequency"),e.qZA(),e.TgZ(497,"p"),e._uU(498,"A term-id from the HPO-sub-ontology below the term "),e.TgZ(499,"a",13),e._uU(500,"Frequency"),e.qZA(),e._uU(501,"."),e.qZA(),e._uU(502," There are three allowed options for this field. "),e.TgZ(503,"ol")(504,"li"),e._uU(505," A term-id from the HPO-sub-ontology below the term "),e.TgZ(506,"a",13),e._uU(507,"Frequency"),e.qZA(),e._uU(508,". "),e.qZA(),e.TgZ(509,"li"),e._uU(510," A count of patients affected within a cohort. For instance, 7/13 would indicate that 7 of the 13 patients with the specified disease were found to have the phenotypic abnormality referred to by the HPO term in question in the study referred to by the DB_Reference "),e.qZA(),e.TgZ(511,"li"),e._uU(512," A percentage value such as 17%, again referring to the percentage of patients found to have the phenotypic abnormality referred to by the HPO term in question in the study referred to by the DB_Reference. If possible, the 7/13 format is preferred over the percentage format if the exact data is available. "),e.qZA()()(),e.TgZ(513,"li")(514,"strong"),e._uU(515,"Sex:"),e.qZA(),e.TgZ(516,"p"),e._uU(517,"This field contains the strings MALE or FEMALE if the annotation in question is limited to males or females. This field refers to the phenotypic (and not the chromosomal) sex, and does not intend to capture the further complexities of sex determination. If a phenotype is limited to one or the other sex, then the corresponding term from the "),e.TgZ(518,"a",14),e._uU(519,"Clinical modifier"),e.qZA(),e._uU(520," subontology should also be used in the Modifier field."),e.qZA()(),e.TgZ(521,"li")(522,"strong"),e._uU(523,"Modifier:"),e.qZA(),e.TgZ(524,"p"),e._uU(525,"A term from the "),e.TgZ(526,"a",14),e._uU(527,"Clinical modifier"),e.qZA(),e._uU(528," subontology"),e.qZA()(),e.TgZ(529,"li")(530,"strong"),e._uU(531,"Aspect:"),e.qZA(),e.TgZ(532,"p"),e._uU(533,"one of P (Phenotypic abnormality), I (inheritance), C (onset and clinical course). This field is mandatory; cardinality 1"),e.qZA()(),e.TgZ(534,"li")(535,"strong"),e._uU(536,"Date_Created:"),e.qZA(),e.TgZ(537,"p"),e._uU(538,"Date on which the annotation was made; format is YYYY-MM-DD this field is mandatory, cardinality 1"),e.qZA()(),e.TgZ(539,"li")(540,"strong"),e._uU(541,"Assigned_By:"),e.qZA(),e.TgZ(542,"p"),e._uU(543,"This refers to the center or user making the annotation."),e.qZA()()()())},dependencies:[t.yS],styles:[".float-left[_ngcontent-%COMP%]{text-align:left}.table-spacer[_ngcontent-%COMP%]   td[_ngcontent-%COMP%], .table-spacer[_ngcontent-%COMP%]   th[_ngcontent-%COMP%]{padding:5px}"]}),g})();var P=r(1576);let A=(()=>{class g{constructor(){}ngOnInit(){}}return g.\u0275fac=function(d){return new(d||g)},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-layperson"]],decls:28,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://monarchinitiative.org/","target","_blank"],["routerLink","/browse/term/HP:0000256"],["href","https://www.ncbi.nlm.nih.gov/pubmed/29632381","target","__blank"],["href","https://doi.org/10.5281/zenodo.1168737","target","__blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/laypersonHPO.jpg",1,"static-images"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Layperson Synonyms"),e.qZA(),e.TgZ(3,"p"),e._uU(4,"In clinical settings, phenotypic abnormalities are often defined using medical terminology, which can be difficult for patients to understand. This so-called terminology gap can impede participation of patients and families in research projects concerned with the phenotype or natural history of diseases, and can even make it difficult for patients to understand data from their own medical charts. "),e.qZA(),e.TgZ(5,"p"),e._uU(6,"The HPO Project (as part of the "),e.TgZ(7,"a",2),e._uU(8,"Monarch Initiative"),e.qZA(),e._uU(9,') has therefore "translated" the terms of the HPO into "plain language". For instance, the term for '),e.TgZ(10,"a",3)(11,"em"),e._uU(12,"Macrocephaly"),e.qZA()(),e._uU(13," can be translated as "),e.TgZ(14,"em"),e._uU(15,"Large skull"),e.qZA(),e._uU(16,". A report about the layperson HPO terminology was published in "),e.TgZ(17,"a",4),e._uU(18,"Nature Genetics"),e.qZA(),e._uU(19," and an open-access preprint version is available "),e.TgZ(20,"a",5),e._uU(21,"here"),e.qZA(),e._uU(22,". "),e.qZA(),e._uU(23," The layperson translations are included in the main HPO release files. For instance, in the hp.obo file the term for Macrocephaly contains synonyms that are marked as layperson (the following is an excerpt of the entire entry for the term): "),e.TgZ(24,"pre"),e._uU(25,'    [Term]\n    id: HP:0000256\n    name: Macrocephaly\n    def: "Occipitofrontal (head) circumference greater than 97th centile compared to appropriate, age matched,\n    sex-matched normal standards. Alternatively, a apparently increased size of the cranium." [pmid:19125436]\n    comment: Macrocephaly can be due to hydrocephalus (increased CFSF), megalencephaly (increased brain volume) or\n    thickening of the skull. Head circumference is measured from just above the glabella\n    (the most prominent point on the frontal bone above the root of the nose) to the most posterior prominent point of\n    the occipital bone using a tape measure. Some standard charts are organized by centiles [Hall et al. [2007]], others\n    by standard deviations [Farkas, [1981]]. It is important to add an indication of how far above the normal standard\n    the head circumference is if an accurate assessment of this can be made. Macrocephaly is an absolute term.\n    The term relative macrocephaly can be used when the head size centile exceeds the centile for height, for example,\n    head size at the 75th centile with height at the 5th centile for age and sex.\n    subset: hposlim_core\n    synonym: "Big calvaria" BROAD layperson [ORCID:0000-0001-5889-4463]\n    synonym: "Big cranium" BROAD layperson [ORCID:0000-0001-5889-4463]\n    synonym: "Big head" BROAD layperson [ORCID:0000-0001-5889-4463]\n    synonym: "Big skull" BROAD layperson [ORCID:0000-0001-5889-4463]\n  '),e.qZA(),e.TgZ(26,"div",6),e._UZ(27,"img",7),e.qZA()())},dependencies:[t.yS,P.xw,P.Wh]}),g})(),m=(()=>{class g{constructor(){}ngOnInit(){}}return g.\u0275fac=function(d){return new(d||g)},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-translations"]],decls:31,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://crowdin.com/project/hpo-translation","target","_blank"],["href","https://github.com/dbcls/bh16/wiki/HumanPhenotypeOntologyJapanese","target","_blank"],["href","http://www.chinahpo.org/wiki/index.php/%E9%A6%96%E9%A1%B5","target","_blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/27899602","target","_blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/14681409","target","_blank"],["href","http://www.irdirc.org/?s=ichpt","target","_blank"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Translations and Mapping"),e.qZA(),e.TgZ(3,"h2"),e._uU(4,"Translations"),e.qZA(),e.TgZ(5,"p"),e._uU(6,"The HPO has been translated into many languages including (in alphabetical order) Chinese, Dutch, French, German, Italian, Japanese, Portugese, Russion, Spanish, and Turkish. Some of the translations can be downloaded from the "),e.TgZ(7,"a",2),e._uU(8,"HPO Translation"),e.qZA(),e._uU(9," webpage. Information about the Japanese translation is available on the "),e.TgZ(10,"a",3),e._uU(11,"HPO Japanese GitHub page"),e.qZA(),e._uU(12,". The Chinese HPO project offers a "),e.TgZ(13,"a",4),e._uU(14,"Wiki page"),e.qZA(),e._uU(15,". "),e.qZA(),e.TgZ(16,"h2"),e._uU(17,"Mappings"),e.qZA(),e.TgZ(18,"p"),e._uU(19,"In contrast to many clinical terminologies, the HPO was purpose-built to facilitate differential diagnostics and scientific investigations such as cross-species comparisons with model organism data from mouse and zebrafish. Moreover, the HPO has an extremely broad coverage of concepts. In an evaluation of HPO content versus the numerous vocabularies integrated within the Unified Medical Language System (UMLS), Winnenburg and Bodenreider showed that the coverage of HPO phenotype concepts in the UMLS is 54% and only 30% in SNOMED CT (ISMB 2014 SIG session cited "),e.TgZ(20,"a",5),e._uU(21,"here"),e.qZA(),e._uU(22,"). The "),e.TgZ(23,"a",6),e._uU(24,"UMLS"),e.qZA(),e._uU(25," is a terminology integration system developed by the U.S. National Library of Medicine that integrates many standard biomedical terminologies. In order to improve the coverage of phenotype data, the UMLS has now integrated the entire HPO starting with the 2015AB release. This enables an easy process to map HPO-encoded data to standard health-care terminologies such as SNOMED CT. "),e.qZA(),e.TgZ(26,"p"),e._uU(27,"HPO has contributed to the establishment of the "),e.TgZ(28,"a",7),e._uU(29,"International Consortium of Human Phenotype Terminologies (ICHPT)"),e.qZA(),e._uU(30," to provide the community with standards that achieve interoperability among databases incorporating human phenotypic features. "),e.qZA()())},styles:[".translation[_ngcontent-%COMP%]{margin-top:0;list-style:none;-webkit-padding-start:10px;padding-inline-start:10px}.lyfe-language-btn[_ngcontent-%COMP%]{text-decoration:none}.lyfe-language-btn[_ngcontent-%COMP%]:hover{color:#fff!important}.indigenous[_ngcontent-%COMP%]{width:300px}"]}),g})();var v=r(7730),N=r(7392);let R=(()=>{class g{constructor(d){this.utilityService=d,this.hpoaUrl="http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa"}ngOnInit(){this.utilityService.getMostRecentReleaseHPO().subscribe(d=>{this.version=d})}}return g.\u0275fac=function(d){return new(d||g)(e.Y36(v.t))},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-annotations"]],decls:35,vars:1,consts:[["fxLayout","row wrap",1,"container"],["fxFlexFill","",1,"pageTitle","center"],["fxFlex","40",1,"download-banner-wrapper","mat-elevation-z4",3,"click"],[1,"download-banner-title"],["fxFlex","100"],["href","https://www.ncbi.nlm.nih.gov/pubmed/30476213","target","_blank"],["href","https://hpo-annotation-qc.readthedocs.io/en/latest/annotationFormat.html#phenotype-hpoa-format","target","_blank"],["href","http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa","target","__blank"],["href","http://purl.obolibrary.org/obo/hp/hpoa/genes_to_phenotype.txt","target","__blank"],[1,"reduce-margin"],["href","http://purl.obolibrary.org/obo/hp/hpoa/phenotype_to_genes.txt","target","__blank"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Download Annotations"),e.qZA(),e.TgZ(3,"div",2),e.NdJ("click",function(){return U.utilityService.openExternalUrl(U.hpoaUrl)}),e.TgZ(4,"mat-icon"),e._uU(5,"download"),e.qZA(),e.TgZ(6,"h3",3),e._uU(7),e.qZA()(),e.TgZ(8,"div",4)(9,"h3"),e._uU(10,"Current format"),e.qZA(),e.TgZ(11,"p"),e._uU(12,"The HPO project has transitioned to a new annotation format in 2019 that is described in "),e.TgZ(13,"a",5),e._uU(14,"K\xf6hler et al (2019) Nucleic Acids Res"),e.qZA(),e._uU(15,". A detailed format description is available "),e.TgZ(16,"a",6),e._uU(17,"here"),e.qZA(),e._uU(18,"."),e.qZA(),e.TgZ(19,"ul")(20,"li")(21,"a",7),e._uU(22,"http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa"),e.qZA()()(),e.TgZ(23,"h3"),e._uU(24,"Additional resources"),e.qZA(),e.TgZ(25,"p"),e._uU(26,"We provide files that link genes and phenotypes in different ways."),e.qZA(),e.TgZ(27,"a",8),e._uU(28,"http://purl.obolibrary.org/obo/hp/hpoa/genes_to_phenotype.txt"),e.qZA(),e._uU(29," provides a link between genes and HPO terms. All phenotype terms associated with any disease that is associated with variants in a gene are assigned to that gene in this file. Other files are available on our Jenkins server that filter terms according to provenance of the annotation and frequency of the features in the disease."),e._UZ(30,"br")(31,"hr",9),e.TgZ(32,"a",10),e._uU(33,"http://purl.obolibrary.org/obo/hp/hpoa/phenotype_to_genes.txt"),e.qZA(),e._uU(34," is analogous, but instead provides links from HPO terms to genes. "),e.qZA()()),2&d&&(e.xp6(7),e.hij("Download HPO annotations (",U.version,")"))},dependencies:[N.Hw,P.xw,P.s9,P.yH],styles:[".reduce-margin[_ngcontent-%COMP%]{margin:5px}.download-banner-wrapper[_ngcontent-%COMP%]{display:flex}"]}),g})();var z=r(4859);let S=(()=>{class g{constructor(){}ngOnInit(){}}return g.\u0275fac=function(d){return new(d||g)},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-indigenous"]],decls:89,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["fxLayout","row",1,"row"],["fxFlex","50"],[1,"translation"],["src","assets/indigenous.png",1,"static-images","indigenous"],["fxLayout.gt-sm","row wrap","fxLayoutAlign.gt-sm","space-evenly center"],["mat-raised-button","","href","http://www.lyfelanguages.com/About.html","color","primary","target","_blank",1,"lyfe-language-btn"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Indigenous Language Translations"),e.qZA(),e.TgZ(3,"h2"),e._uU(4,"Lyfe Languages Project"),e.qZA(),e.TgZ(5,"p"),e._uU(6,"Lyfe Languages commenced during the 2019 United Nations International Year of Indigenous Languages as part of Project Y. It involves the translation of Human Phenotype Ontology (HPO) terms into Indigenous languages, including Australian Aboriginal & Torres Strait Islander languages in Australia and Indigenous Ghanaian Languages. These translations will enable a more effective and personal approach in communication between health care providers and patients. The complexity of HPO can be lost on most people, therefore translation into language will improve patient understanding of what they\u2019re experiencing. "),e.qZA(),e.TgZ(7,"p"),e._uU(8,"Translations are being completed in partnership between Champions, Elders, community, community medical centres, local language centres and local land councils. This approach strengthens bonds between all the groups involved, ensuring all voices are heard and that community empowerment is achieved."),e.qZA(),e._uU(9," The approach of this initiative will: "),e.TgZ(10,"ul")(11,"li"),e._uU(12,"Improve rates of diagnosis"),e.qZA(),e.TgZ(13,"li"),e._uU(14,"Educate youth on rare disease and genetic health care"),e.qZA(),e.TgZ(15,"li"),e._uU(16,"Connect youth to culture, and knowledge sharing between younger and older generations"),e.qZA(),e.TgZ(17,"li"),e._uU(18,"Using new technology to connect with ancient knowledge, including through the HPO lingua franca"),e.qZA(),e.TgZ(19,"li"),e._uU(20,"Ensure culturally appropriate health care"),e.qZA(),e.TgZ(21,"li"),e._uU(22,"Retain and empower Indigenous language"),e.qZA()(),e.TgZ(23,"p"),e._uU(24," Translations are being completed in partnership between Champions, Elders, community, community medical centres, local language centres and local land councils. This approach strengthens bonds between all the groups involved, ensuring all voices are heard and that community empowerment is achieved. "),e.qZA(),e.TgZ(25,"div",2)(26,"div",3)(27,"p"),e._uU(28," Currently there are several Aboriginal and Torres Strait Islander languages across Australia being translated including: "),e.qZA(),e.TgZ(29,"ul",4)(30,"li"),e._uU(31,"Nyangumarta"),e.qZA(),e.TgZ(32,"li"),e._uU(33,"Noongar"),e.qZA(),e.TgZ(34,"li"),e._uU(35,"Badimaya"),e.qZA(),e.TgZ(36,"li"),e._uU(37,"Eastern Arrernte"),e.qZA(),e.TgZ(38,"li"),e._uU(39,"Kija"),e.qZA(),e.TgZ(40,"li"),e._uU(41,"Kala Lagaw Ya and Creol"),e.qZA(),e.TgZ(42,"li"),e._uU(43,"Manyjilyjarra"),e.qZA()()(),e.TgZ(44,"div",3),e._UZ(45,"img",5),e.qZA()(),e.TgZ(46,"p"),e._uU(47,"There are over 290 Aboriginal and Torres Strait Islander language groups across Australia. On average, 1 in 10 families speak an Indigenous language at home. In regional parts of Australia, that figure can jump up to 5.8 in 10 families. People living in regional parts of Australia also experience a much more different health journey than those living in metropolitan areas, which includes limited access to primary and specialist health care, large distances to travel, referral to hospital specialists often involving travelling to larger metropolitan hospitals usually by way of airplane. This highlights the importance of the need for translations to ensure the effective delivery of health care."),e.qZA(),e.TgZ(48,"p"),e._uU(49,"Examples of translations completed in Nyangumarta include:"),e.qZA(),e.TgZ(50,"div",6)(51,"div")(52,"strong"),e._uU(53,"HPO Term \u2013 Microcephaly"),e.qZA(),e.TgZ(54,"ul",4)(55,"li"),e._uU(56,"Description \u2013 Small head circumference"),e.qZA(),e.TgZ(57,"li"),e._uU(58,"Translation \u2013 Jurnturtu wupartu"),e.qZA()()(),e.TgZ(59,"div")(60,"strong"),e._uU(61,"HPO Term \u2013 Macrocephaly"),e.qZA(),e.TgZ(62,"ul",4)(63,"li"),e._uU(64,"Description \u2013 Big head circumference"),e.qZA(),e.TgZ(65,"li"),e._uU(66,"Translation \u2013 Jurnturtu wirtu"),e.qZA()()(),e.TgZ(67,"div")(68,"strong"),e._uU(69,"HPO Term \u2013 Hypertelorism"),e.qZA(),e.TgZ(70,"ul",4)(71,"li"),e._uU(72,"Description \u2013 widely spaced eyes"),e.qZA(),e.TgZ(73,"li"),e._uU(74,"Translation \u2013 Pani kakajul"),e.qZA()()(),e.TgZ(75,"div")(76,"strong"),e._uU(77,"HPO Term \u2013 Macrostomia"),e.qZA(),e.TgZ(78,"ul",4)(79,"li"),e._uU(80,"Description \u2013 wide mouth"),e.qZA(),e.TgZ(81,"li"),e._uU(82,"Translation \u2013 Jawa walpurra"),e.qZA()()()(),e.TgZ(83,"p"),e._uU(84," Lyfe Languages continues to grow, evolve and adapt on a daily basis, while gaining popularity and interest on a worldwide scale. Ghana is a country located in the western African Region with a population of about 30 million. Like most African countries, its population is predominantly indigenous with about 100 ethnic groups and over 50 indigenous languages. Currently Rare Disease Ghana Initiative with its Lyfe Language champions are translating the HPO terms into Twi, Ewe, Ga, Dagaare, Dagbani, Kaseem, Nzema and Gonja. "),e.qZA(),e.TgZ(85,"p"),e._uU(86," The ultimate goal of this project is to ensure that medical terminology can be explained to patients in a language that they understand, thus ensuring understanding and appropriate care. "),e.qZA(),e.TgZ(87,"a",7),e._uU(88,"Visit the Lyfe Languages Project"),e.qZA()())},dependencies:[z.zs,P.xw,P.Wh,P.yH],styles:[".lyfe-language-btn[_ngcontent-%COMP%]{text-decoration:none}.lyfe-language-btn[_ngcontent-%COMP%]:hover{color:#fff!important}.indigenous[_ngcontent-%COMP%]{width:300px}"]}),g})();var I=r(2340);const h=t.Bz.forChild([{path:"data",children:[{path:"annotations",component:R},{path:"annotation-format",component:q},{path:"ontology",component:(()=>{class g{constructor(d){this.utilityService=d,this.oboUrl=`${I.N.OBO_PURL_NO_EXT}.obo`,this.owlUrl=`${I.N.OBO_PURL_NO_EXT}.owl`,this.jsonUrl=`${I.N.OBO_PURL_NO_EXT}.json`}ngOnInit(){this.utilityService.getMostRecentReleaseHPO().subscribe(d=>{this.version=d})}}return g.\u0275fac=function(d){return new(d||g)(e.Y36(v.t))},g.\u0275cmp=e.Xpm({type:g,selectors:[["app-ontology"]],decls:21,vars:1,consts:[["fxLayout","row wrap",1,"container","ontology-download"],["fxFlex","100",1,"pageTitle","center"],["fxLayout","column","fxLayoutAlign","space-evenly center","fxFlex","",1,"ontology-download-wrapper"],["fxLayout","row",1,"download-banner-wrapper","mat-elevation-z4",3,"click"],[1,"download-banner-title"]],template:function(d,U){1&d&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Download Ontology"),e.qZA(),e.TgZ(3,"div",2)(4,"h2"),e._uU(5),e.qZA(),e.TgZ(6,"div",3),e.NdJ("click",function(){return U.utilityService.openExternalUrl(U.oboUrl)}),e.TgZ(7,"mat-icon"),e._uU(8,"download"),e.qZA(),e.TgZ(9,"h3",4),e._uU(10,"Download hp.obo"),e.qZA()(),e.TgZ(11,"div",3),e.NdJ("click",function(){return U.utilityService.openExternalUrl(U.owlUrl)}),e.TgZ(12,"mat-icon"),e._uU(13,"download"),e.qZA(),e.TgZ(14,"h3",4),e._uU(15,"Download hp.owl"),e.qZA()(),e.TgZ(16,"div",3),e.NdJ("click",function(){return U.utilityService.openExternalUrl(U.jsonUrl)}),e.TgZ(17,"mat-icon"),e._uU(18,"download"),e.qZA(),e.TgZ(19,"h3",4),e._uU(20,"Download hp.json"),e.qZA()()()()),2&d&&(e.xp6(5),e.hij("v",U.version,""))},dependencies:[N.Hw,P.xw,P.Wh,P.yH],styles:[".ontology-download[_ngcontent-%COMP%]   .purl[_ngcontent-%COMP%]{font-size:1.2rem;font-weight:700}.ontology-download[_ngcontent-%COMP%]   .ontology-download-wrapper[_ngcontent-%COMP%]{margin:1rem 0 0}"]}),g})()},{path:"layperson",component:A},{path:"translations",component:m},{path:"indigenous-languages",component:S}]}]);let Z=(()=>{class g{}return g.\u0275fac=function(d){return new(d||g)},g.\u0275mod=e.oAB({type:g}),g.\u0275inj=e.cJS({imports:[t.Bz,h,t.Bz]}),g})(),p=(()=>{class g{}return g.\u0275fac=function(d){return new(d||g)},g.\u0275mod=e.oAB({type:g}),g.\u0275inj=e.cJS({imports:[n.ez,Z,T.c,o.u5,o.UX]}),g})()},4475:(W,C,r)=>{r.d(C,{v:()=>e});var n=r(4650),T=r(3917),o=r(4859),t=r(7084);let e=(()=>{class q{constructor(A){this.route=A,this.panelHash={1:"hpo-relation",2:"content-change",3:"collaboration",4:"ontology",5:"susceptibility",6:"annotation-use",7:"genes-phenotypes",8:"frequencies",9:"association-rules"}}ngOnInit(){this.route.params.subscribe(A=>{this.navigateToQuestion(+A.id)})}navigateToQuestion(A){Object.keys(this.panelHash).includes(A.toString())&&(this.openPanel=A,document.getElementById(this.panelHash[A]).scrollIntoView())}}return q.\u0275fac=function(A){return new(A||q)(n.Y36(T.gz))},q.\u0275cmp=n.Xpm({type:q,selectors:[["app-faq"]],decls:264,vars:11,consts:[[1,"container","faq"],[1,"center","pageTitle"],[1,"center"],["mat-button","","href","https://github.com/TheJacksonLaboratory/hpo-web/issues/new","target","__blank",1,"faq-btn"],[1,"faq-container"],["id","hpo-relation",3,"expanded"],["href","https://monarchinitiative.org/","target","__blank"],["id","content-change",3,"expanded"],["target","__blank","href","https://github.com/obophenotype/human-phenotype-ontology/issues"],["id","collaboration",3,"expanded"],["href","https://github.com/obophenotype/human-phenotype-ontology/issues","target","__blank"],["href","https://github.com/Phenomics/HPO-translations","target","__blank"],["routerLink","/contact"],["href","https://omim.org/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/18950739","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/26119816","target","__blank"],["id","ontology",3,"expanded"],["id","susceptibility",3,"expanded"],["href","https://www.ebi.ac.uk/gwas/efotraits/EFO_0000270","target","_blank"],["href","https://omim.org/entry/600807","target","_blank"],["href","https://www.ebi.ac.uk/gwas/efotraits/HP_0000175","target","_blank"],["href","https://omim.org/entry/604370","target","_blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/28632866","target","_blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/19057201/","target","_blank"],["href","https://hpo.jax.org/app/browse/disease/OMIM:193670","target","_blank"],["href","https://hpo.jax.org/app/browse/disease/OMIM:146500","target","_blank"],["id","ontology-structure",3,"expanded"],["routerLink","/browse/term/HP:0001964"],["routerLink","/browse/term/HP:0006494"],["routerLink","/browse/term/HP:0001711"],["routerLink","/browse/term/HP:0001713"],["id","ontology-research",3,"expanded"],["href","http://www.geneontology.org/","target","__blank"],["id","annotation-use",3,"expanded"],["routerLink","/browse/disease/ORPHA:171719"],["routerLink","/browse/term/HP:0002827"],["routerLink","/browse/term/HP:0001371"],["id","genes-phenotypes",3,"expanded"],["id","frequencies",3,"expanded"],["id","association-rules",3,"expanded"]],template:function(A,m){1&A&&(n.TgZ(0,"div",0)(1,"h1",1),n._uU(2,"Frequently Asked Questions"),n.qZA(),n._UZ(3,"br"),n.TgZ(4,"div",2)(5,"a",3),n._uU(6,"Request a Feature or Report a Bug"),n.qZA()(),n.TgZ(7,"div",4)(8,"h3"),n._uU(9," Ontology "),n.qZA(),n.TgZ(10,"mat-accordion")(11,"mat-expansion-panel",5)(12,"mat-expansion-panel-header")(13,"mat-panel-title")(14,"strong"),n._uU(15,"How is HPO related to the Monarch Initiative?"),n.qZA()()(),n.TgZ(16,"p"),n._uU(17,"The HPO is the clinical window of the "),n.TgZ(18,"a",6),n._uU(19,"Monarch Initiative"),n.qZA(),n._uU(20," which is an NIH supported project that integrates, aligns, and re-distributes cross-species gene, genotype, variant, disease, and phenotype data. "),n.TgZ(21,"a",6),n._uU(22,"The Monarch Browser"),n.qZA(),n._uU(23," allows users to browse not only human diseases but to see their links to other organisms, genes, phenotypes, and pathways, which may be of especial interest to translational researchers."),n.qZA()(),n.TgZ(24,"mat-expansion-panel",7)(25,"mat-expansion-panel-header")(26,"mat-panel-title")(27,"strong"),n._uU(28,"Am I allowed to change the contents of HPO?"),n.qZA()()(),n.TgZ(29,"p"),n._uU(30,"The HPO is a community-driven project. We request that all proposed changes be suggested on our "),n.TgZ(31,"a",8),n._uU(32,"GitHub tracker"),n.qZA(),n._uU(33," so that they will be of use to all users of the HPO. "),n.qZA()(),n.TgZ(34,"mat-expansion-panel",9)(35,"mat-expansion-panel-header")(36,"mat-panel-title")(37,"strong"),n._uU(38,"How to collaborate with the HPO ?"),n.qZA()()(),n.TgZ(39,"p"),n._uU(40,"We welcome the participation of interested colleagues. We anticipate that the structure of the HPO will continue to be refined and completed for some time to come. Groups or persons with expert knowledge in a particular domain of human phenotyping in a medical genetics setting are invited to contribute their knowledge on a collaborative basis. Please contact us to discuss details."),n.qZA(),n.TgZ(41,"p"),n._uU(42,"We are currently transitioning from annotations derived in a semiautomated fashion from the OMIM database to manually curated annotations. Given the number of human hereditary diseases, this is no small task. We encourage participation of colleagues with an interest in specific hereditary diseases to improve the annotations for \u2018their\u2019 disease. Please get in contact with us to discuss this form of collaboration. We are also preparing an online system for manual annotations."),n.qZA(),n.TgZ(43,"p"),n._uU(44,"The issue tracker is "),n.TgZ(45,"a",10),n._uU(46,"available on Github."),n.qZA()(),n.TgZ(47,"h4"),n._uU(48,"Collaborations for translations"),n.qZA(),n.TgZ(49,"p"),n._uU(50,"We are working with several people to translate more content of HPO into a multitude of languages. Several translations are being actively developed at the moment (e.g. german)."),n.qZA(),n.TgZ(51,"p"),n._uU(52,"The translations are considered a project coordinated by the HPO team and are hosted on "),n.TgZ(53,"a",11),n._uU(54,"GitHub"),n.qZA(),n._uU(55,". Please do not start translating HPO content without contacting us beforehand. This might cause duplicated translations and other problems. "),n.qZA(),n.TgZ(56,"p"),n._uU(57," Please contact us "),n.TgZ(58,"a",12),n._uU(59,"here"),n.qZA(),n._uU(60,"."),n.qZA()(),n.TgZ(61,"mat-expansion-panel")(62,"mat-expansion-panel-header")(63,"mat-panel-title")(64,"strong"),n._uU(65,"What is the medical focus of the Human Phenotype Ontology?"),n.qZA()()(),n.TgZ(66,"p"),n._uU(67," The medical focus of the HPO in its initial decade (2007-2017) was on rare, mainly Mendelian diseases. The construction of the initial version of the HPO in 2007/2008 was performed by generating an ontology based on descriptions in the Clinical Synopsis of the "),n.TgZ(68,"a",13),n._uU(69,"Online Mendelian Inheritance in Man (OMIM) database"),n.qZA(),n._uU(70,". Since the "),n.TgZ(71,"a",14),n._uU(72,"initial publication of the HPO in 2008"),n.qZA(),n._uU(73,", the HPO team has held regular workshops with clinicians to refine and extend the clinical terminology of the HPO in specific areas such as cardiology or immunology. We have added textual definitions, computational logical definitions, and roughly 6,000 new terms since then. "),n.qZA(),n.TgZ(74,"p"),n._uU(75," The focus of the HPO will be extended to other areas of medicine in coming years. A "),n.TgZ(76,"a",15),n._uU(77,"pilot project"),n.qZA(),n._uU(78," on common-disease annotations was published in 2015. "),n.qZA()(),n.TgZ(79,"mat-expansion-panel",16)(80,"mat-expansion-panel-header")(81,"mat-panel-title")(82,"strong"),n._uU(83,"What is an ontology?"),n.qZA()()(),n.TgZ(84,"p"),n._uU(85,' A terminology contains a list of items (often called "terms") that represent the concepts of a domain. An ontology is a terminology that additionally specifies formal semantic relationships of concepts. For instance, a terminology of wines would include a list of items such as Chardonnay, Cabernet Sauvignon, Rioja, Riesling, Sauvignon blanc, etc., but an ontology of wine might have a term for "red wine" and "Rioja"; It also can specify that "Rioja" is a type of "red wine". Ontologies and terminologies can be used as tools for standardizing and exchanging data. Ontologies such as the HPO typically enable sophisticated computational algorithms that exploit the semantic relations between terms. '),n.qZA()(),n.TgZ(86,"mat-expansion-panel",17)(87,"mat-expansion-panel-header")(88,"mat-panel-title")(89,"strong"),n._uU(90,"How does the HPO project model susceptibility?"),n.qZA()()(),n.TgZ(91,"p"),n._uU(92," The word susceptibility can be defined as a state of being liable to be influenced or harmed by a particular thing. In genetics, the word susceptibility can be used with a variety of meanings. "),n.qZA(),n.TgZ(93,"ol")(94,"li"),n._uU(95,"To refer to an allele that confers some (usually small) increased risk of developing a polygenic (complex) disease. A disease such as Asthma for instance currently has 733 such loci listed in "),n.TgZ(96,"a",18),n._uU(97,"GWAS catalog"),n.qZA(),n._uU(98,", as well as ten entries in "),n.TgZ(99,"a",19),n._uU(100,"OMIM"),n.qZA(),n._uU(101,'. The HPO project does not consider these "susceptibility" entries in OMIM to be separate diseases. '),n.qZA(),n.TgZ(102,"li"),n._uU(103," Similarly, some congenital anomalies such as Cleft palate are polygenic diseases, with over 150 loci in "),n.TgZ(104,"a",20),n._uU(105,"GWAS catalog"),n.qZA(),n._uU(106,". These loci are treated analogously. "),n.qZA(),n.TgZ(107,"li"),n._uU(108,"Susceptibility to a cancer that is inherited as a Mendelian trait; the classic example is "),n.TgZ(109,"a",21),n._uU(110,"BRCA1"),n.qZA(),n._uU(111,". The susceptibility associated with a pathogenic BRCA1 mutation is substantially higher than the susceptibility associated with an allele related to a polygenic disease; the susceptibility is also transmitted as a Mendelian trait. The clinical aspects may differ in important ways for distinct cancer susceptibility loci. For instance, the risk of ovarian cancer is higher for BRCA1 mutation carriers than for BRCA2 mutation carriers ( "),n.TgZ(112,"a",22),n._uU(113,"Kuchenbaecker et al, 2017"),n.qZA(),n._uU(114,"). "),n.qZA(),n.TgZ(115,"li"),n._uU(116,"Inherited susceptibility to certain kinds of infectious disease may occur as a Mendelian trait. For example, WHIM syndrome confers suceptibility to human papilloma virus infection induced warts, condyloma acuminata and carcinomas ("),n.TgZ(117,"a",23),n._uU(118,"Kawai and Malech, 2009"),n.qZA(),n._uU(119,"). Susceptibility to infections may also be a polygenic trait. The HPO models Mendelian traits as diseases, e.g., "),n.TgZ(120,"a",24),n._uU(121,"WHIM syndrome"),n.qZA(),n._uU(122,"). "),n.qZA(),n.TgZ(123,"li"),n._uU(124,"In some cases, susceptibility to diseases such as neurodegeneration can be inherited as a Mendelian trait, including Huntington disease, some forms of Parkinson disease, and Alzheimer disease. "),n.TgZ(125,"a",25),n._uU(126,"Mutliple system atrophy"),n.qZA(),n._uU(127," may be occur sporadically or in rare cases be inherited as a Mendelian trait. For this reason, HPO models this as a distinct disease. "),n.qZA()()(),n.TgZ(128,"mat-expansion-panel",26)(129,"mat-expansion-panel-header")(130,"mat-panel-title")(131,"strong"),n._uU(132,"How are the ontology terms structured?"),n.qZA()()(),n.TgZ(133,"p"),n._uU(134," Most ontologies are structured as directed acyclic graphs (DAG), which are similar to hierarchies but differ in that a more specialized term (child) can be related to more than one less specialized term (parent). Cycles (cyclic paths in the graph) are not allowed. The relationship of the terms of the HPO to one another is displayed in the DAG. For instance, the term "),n.TgZ(135,"a",27),n._uU(136,"Aplasia/Hypoplasia of metatarsal bones"),n.qZA(),n._uU(137," is a child of both "),n.TgZ(138,"a",28),n._uU(139,"Aplasia/Hypoplasia involving bones of the feet"),n.qZA(),n._uU(140," and "),n.TgZ(141,"a",27),n._uU(142,"Abnormal metatarsal morphology"),n.qZA(),n._uU(143,". The ability to encode multiple parents in a DAG adds to the flexibility and descriptiveness of the ontology. This would not be possible with a simple hierarchical system. The "),n.TgZ(144,"strong"),n._uU(145,"\xa0is-a\xa0"),n.qZA(),n._uU(146," relationship is transitive, meaning that annotations are inherited up all paths to the root. For instance, "),n.TgZ(147,"a",29),n._uU(148,"Abnormal morphology of the left ventricle"),n.qZA(),n.TgZ(149,"strong"),n._uU(150,"\xa0is-a\xa0"),n.qZA(),n.TgZ(151,"a",30),n._uU(152,"Abnormal cardiac ventricle morphology"),n.qZA(),n._uU(153,". "),n.qZA(),n.TgZ(154,"p"),n._uU(155," Each term in the HPO describes a clinical abnormality. These may be general terms, such as "),n.TgZ(156,"em"),n._uU(157,"Abnormal ear morphology"),n.qZA(),n._uU(158," or very specific ones such as "),n.TgZ(159,"em"),n._uU(160,"Chorioretinal atrophy"),n.qZA(),n._uU(161,". The terms have a unique ID such as HP:0001140 and a label such as Epibulbar dermoid. Most terms have textual definitions such as An epibulbar dermoid is a benign tumor typically found at the junction of the cornea and sclera (limbal epibullar dermoid). The source of the definition must be indicated. Many terms have synonyms. For instance, Epibulbar dermoids is taken to be a synonym of Epibulbar dermoids."),n.qZA()(),n.TgZ(162,"mat-expansion-panel",31)(163,"mat-expansion-panel-header")(164,"mat-panel-title")(165,"strong"),n._uU(166,"What is an ontology and how is it used for medical genetics research?"),n.qZA()()(),n.TgZ(167,"p"),n._uU(168," The word ontology is derived from Greek words meaning the study of existence and being. More recently, the word ontology has been used in computer science to describe systems that describe concepts within some domain and relationships between those concepts. For instance, the "),n.TgZ(169,"a",32),n._uU(170,"Gene Ontology consortium"),n.qZA(),n._uU(171," has developed an extensive ontology describing molecular functions, biological processes, and cellular locations over the last decade and a number of groups have supplied annotations using the GO terms to gene products of many organisms. The study of human phenotypes in the context of hereditary and common disease has the potential to lead to great insight on the function of genes and genetic networks. The HPO provides computational resources that allow large-scale computational analysis of the human phenome. "),n.qZA()()(),n.TgZ(172,"h3"),n._uU(173," Annotation "),n.qZA(),n.TgZ(174,"mat-accordion")(175,"mat-expansion-panel",33)(176,"mat-expansion-panel-header")(177,"mat-panel-title")(178,"strong"),n._uU(179,"How do I use HPO Annotations?"),n.qZA()()(),n.TgZ(180,"p"),n._uU(181,"The terms of the HPO describe phenotypic abnormalities and do not directly describe diseases. Instead, the HPO project uses so-called annotations to describe the connection between diseases and HPO terms. For instance, the disease, "),n.TgZ(182,"a",34),n._uU(183,"Cutis laxa-Marfanoid syndrome "),n.qZA(),n._uU(184," is annotated by the HPO terms for "),n.TgZ(185,"a",35),n._uU(186,"Hip dislocation"),n.qZA(),n._uU(187,", "),n.TgZ(188,"a",36),n._uU(189,"Flexion contracture"),n.qZA(),n._uU(190," and many more."),n.qZA()(),n.TgZ(191,"mat-expansion-panel",37)(192,"mat-expansion-panel-header")(193,"mat-panel-title")(194,"strong"),n._uU(195,"What is the difference between genes_to_phenotypes.txt and phenotype_to_genes.txt ?"),n.qZA()()(),n.TgZ(196,"p"),n._uU(197,"A question that is asked very often, deals with the asymmetry of the files:"),n.qZA(),n.TgZ(198,"ul")(199,"li")(200,"i"),n._uU(201,"genes_to_phenotypes.txt"),n.qZA()(),n.TgZ(202,"li")(203,"i"),n._uU(204,"phenotype_to_genes.txt"),n.qZA()()(),n.TgZ(205,"p"),n._uU(206,"The difference is that one file uses the transitivity of the annotations and the other does not. I.e. in "),n.TgZ(207,"i"),n._uU(208,"genes_to_phenotypes.txt"),n.qZA(),n._uU(209," we list for each gene the most specific HPO-classes (and not all the ancestors). In "),n.TgZ(210,"i"),n._uU(211,"phenotype_to_genes.txt"),n.qZA(),n._uU(212," we show each phenotype that has at least one gene associated with it. Additionally, we have mapped genes to the ancestor classes of each associated phenotype. For example given that: "),n.qZA(),n.TgZ(213,"ul")(214,"li"),n._uU(215,"HP_1 subclass_of HP_3"),n.qZA(),n.TgZ(216,"li"),n._uU(217,"HP_1 subclass_of HP_3"),n.qZA()(),n.TgZ(218,"p"),n._uU(219,"then "),n.TgZ(220,"i"),n._uU(221,"genes_to_phenotypes.txt"),n.qZA()(),n.TgZ(222,"ul")(223,"li")(224,"strong"),n._uU(225,"geneA"),n.qZA(),n._uU(226," annotated to HP_1"),n.qZA(),n.TgZ(227,"li")(228,"strong"),n._uU(229,"geneB"),n.qZA(),n._uU(230," annotated to HP_2"),n.qZA()(),n.TgZ(231,"p"),n._uU(232,"then "),n.TgZ(233,"i"),n._uU(234,"phenotype_to_genes.txt"),n.qZA(),n._uU(235," contains:"),n.qZA(),n.TgZ(236,"ul")(237,"li"),n._uU(238,"HP_3 annotates "),n.TgZ(239,"strong"),n._uU(240,"geneA"),n.qZA(),n._uU(241," and "),n.TgZ(242,"strong"),n._uU(243,"geneB"),n.qZA()()()(),n.TgZ(244,"mat-expansion-panel",38)(245,"mat-expansion-panel-header")(246,"mat-panel-title")(247,"strong"),n._uU(248,"What is the difference between \u201cALL_FREQUENCIES\u201d and \u201cFREQUENT_FEATURES\u201d?"),n.qZA()()(),n.TgZ(249,"p"),n._uU(250,"Each annotation has a frequency associated, i.e. how many patients with that disease have this clinical feature. \u201cALL_FREQUENCIES\u201d denotes that we do not use any filter. \u201cFREQUENT_FEATURES\u201d means that we exclude all features seen in less than 50% of the patients."),n.qZA()(),n.TgZ(251,"mat-expansion-panel",39)(252,"mat-expansion-panel-header")(253,"mat-panel-title")(254,"strong"),n._uU(255,"What is your procedure for associating genes with HPO phenotypes?"),n.qZA()()(),n.TgZ(256,"p"),n._uU(257,"The following file is used to associate genes with phenotypes."),n.qZA(),n.TgZ(258,"ul")(259,"li")(260,"i"),n._uU(261,"genes_to_phenotypes.txt"),n.qZA()()(),n.TgZ(262,"p"),n._uU(263," In many cases, there are multiple diseases that are associated with mutations in the same gene (for instance, mutations in the LMNA gene are associated with multiple diseases including Cardiomyopathy, dilated, 1A [MIM:115200], Charcot-Marie-Tooth disease, type 2B1 [605588], Emery-Dreifuss muscular dystrophy 2, AD [181350], and Hutchinson-Gilford progeria [176670]. Each of these diseases is associated with a different spectrum of phenotypic features. The gene-to-phenotype association in our databases takes the set of all phenotypic features (HPO terms) associated with one or more diseases caused by mutation in the gene under consideration. Thus, LMNA is associated with Alopecia (which is a feature of Hutchinson-Gilford progeria) as well as Ventricular arrhythmia (which is a feature of Cardiomyopathy, dilated, 1A). "),n.qZA()()()()()),2&A&&(n.xp6(11),n.Q6J("expanded",1==m.openPanel),n.xp6(13),n.Q6J("expanded",2==m.openPanel),n.xp6(10),n.Q6J("expanded",3==m.openPanel),n.xp6(45),n.Q6J("expanded",4==m.openPanel),n.xp6(7),n.Q6J("expanded",5==m.openPanel),n.xp6(42),n.Q6J("expanded",5==m.openPanel),n.xp6(34),n.Q6J("expanded",6==m.openPanel),n.xp6(13),n.Q6J("expanded",6==m.openPanel),n.xp6(16),n.Q6J("expanded",7==m.openPanel),n.xp6(53),n.Q6J("expanded",8==m.openPanel),n.xp6(7),n.Q6J("expanded",9==m.openPanel))},dependencies:[T.yS,o.zs,t.pp,t.ib,t.yz,t.yK],styles:[".faq[_ngcontent-%COMP%]   .faq-container[_ngcontent-%COMP%]{width:50%;margin:0 auto}.faq[_ngcontent-%COMP%]   .mat-expansion-panel[_ngcontent-%COMP%]{margin-bottom:5px;border-radius:0!important}.faq[_ngcontent-%COMP%]   .mat-expanded[_ngcontent-%COMP%]   .mat-expansion-panel-header[_ngcontent-%COMP%]{background-color:#29657c}.faq[_ngcontent-%COMP%]   .mat-expanded[_ngcontent-%COMP%]   .mat-expansion-panel-header-title[_ngcontent-%COMP%]{color:#fff}.faq-btn[_ngcontent-%COMP%]{border-radius:0;background-color:#b84448;color:#fff!important;text-decoration:none}.faq-btn[_ngcontent-%COMP%]:hover{opacity:.8}"]}),q})()},7475:(W,C,r)=>{r.r(C),r.d(C,{ResourcesModule:()=>_e});var n=r(6895),T=r(2474),o=r(4006),t=r(3917),e=r(4650),q=r(7084);let P=(()=>{class s{constructor(){}ngOnInit(){}}return s.\u0275fac=function(l){return new(l||s)},s.\u0275cmp=e.Xpm({type:s,selectors:[["app-clinician-guide"]],decls:79,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://www.ncbi.nlm.nih.gov/pubmed/22504886","target","_blank"],["routerLink","/browse/term/HP:0000819"],["href","https://www.ncbi.nlm.nih.gov/pubmed/8721568","target","_blank"],["routerLink","/browse/term/HP:0100436"],["routerLink","/browse/term/HP:0010175"],["routerLink","/tools/exomiser"]],template:function(l,_){1&l&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Clinician Guide"),e.qZA(),e.TgZ(3,"p"),e._uU(4," The Human Phenotype Ontology (HPO) project offers a variety of resources for working with the HPO. Several of the tools generate lists of diseases that best match a set of HPO terms (some of these tools additionally analyze lists of variants identified by exome or genome sequencing). The choice of terms made by clinicians or researchers using the tools can influence the output of the tools, and therefore, it is important to choose the terms well. "),e.qZA(),e.TgZ(5,"mat-accordion")(6,"mat-expansion-panel")(7,"mat-expansion-panel-header")(8,"mat-panel-title")(9,"strong"),e._uU(10,'How does the HPO define "phenotype"?'),e.qZA()()(),e.TgZ(11,"p"),e._uU(12,"The word phenotype is used with many different meanings. In biology, the most widely accepted definition of phenotype is "),e.TgZ(13,"em"),e._uU(14,"the observable traits of an organism"),e.qZA(),e._uU(15,". In medical contexts, however, the word "),e.TgZ(16,"em"),e._uU(17,"phenotype"),e.qZA(),e._uU(18," is more often used to refer to some deviation from normal morphology, physiology, or behavior. Many phenotypic descriptions in medical publications describe the phenotype in imprecise ways. For instance, a description such as "),e.TgZ(19,"em"),e._uU(20,"myopathic electromyography"),e.qZA(),e._uU(21," is used instead of describing the reasons for this diagnosis, which can include reduced duration and reduced amplitude of the action potentials, increased spontaneous activity with fibrillations, positive sharp waves, or a reduced number of motor units in the muscle. In contrast, "),e.TgZ(22,"strong"),e._uU(23,"deep phenotyping"),e.qZA(),e._uU(24," is defined as the precise and comprehensive analysis of phenotypic abnormalities in which the individual components of the phenotype are observed and described (See "),e.TgZ(25,"a",2),e._uU(26,"Deep Phenotyping for Precision Medicine"),e.qZA(),e._uU(27,"). The HPO strives to provide a computational resource to enable deep phenotyping, and many of the HPO-based computational tools exploit deep phenotyping data."),e.qZA()(),e.TgZ(28,"mat-expansion-panel")(29,"mat-expansion-panel-header")(30,"mat-panel-title")(31,"strong"),e._uU(32,"What is the distinction between a disease and a phenotypic feature?"),e.qZA()()(),e.TgZ(33,"p"),e._uU(34," THe HPO considers a disease to be an entity that has a known or unknown cause, is characterized by one or more phenotypic features which can affect all or only a subset of individuals with the disease, a time course over which the phenotypic features may have onset and evolve, and in some cases one or more indicated treatments and a response to treatment. For instance, if the disease entity is the common cold, then the cause is a virus, the phenotypic features include fever, cough, runny nose, and fatigue, the time course usually is a relatively acute onset with manifestations dragging on for days to about a week, and the treatment may include bed rest, aspirin, or nasal sprays. "),e.qZA(),e.TgZ(35,"p"),e._uU(36," In contrast, a phenotypic feature, such as fever, is a manifestation (component) of many diseases. HPO terms therefore do not describe disease entities, but rather the individual manifestations of the diseases. "),e.qZA(),e.TgZ(37,"p"),e._uU(38," There is a somewhat arbitrary distinction between diseases and phenotypic features. For instance, "),e.TgZ(39,"a",3)(40,"em"),e._uU(41,"Diabetes mellitus"),e.qZA()(),e._uU(42," can be conceptualized as a disease, but it is also a feature of other diseases such as Bardet Biedl syndrome. The HPO takes a practical stance and provides terms for such entities. "),e.qZA()(),e.TgZ(43,"mat-expansion-panel")(44,"mat-expansion-panel-header")(45,"mat-panel-title")(46,"strong"),e._uU(47,"How to choose HPO terms for data entry?"),e.qZA()()(),e.TgZ(48,"p"),e._uU(49,'For each phenotype, choose the most specific term possible, based on the definition and not simply based on the term label. Before selecting a term, examine any of the more specific terms contained underneath it (if you are using the browser on this Website to search for terms, follow the links in the excerpt of the HPO hierarchy shown on the left side of the HPO term pages). If any of those are appropriate, select the more specific term. You do not need to choose both a parent and child term. Some HPO-based software allows users to annotations for manifestations that were investigated and specifically not observed (e.g., by clicking on a \u201cN\u201d/"No" button). These "No" annotations are especially useful to include in cases where there are only a few observable phenotypes. '),e.qZA(),e.TgZ(50,"p"),e._uU(51,' The more specific the item chosen, the better the specificity of the whole phenotypic profile. If there are no phenotypes in a given category, consider making a high level "N" (No) annotation. For example, if a full visual exam finds no visual impairment, then you could choose N-"Visual impairment". Choose terms that are thought to be pathological or unusual, even if they are not so in the greater populace. For example, '),e.TgZ(52,"a",3)(53,"em"),e._uU(54,"Blue irides"),e.qZA()(),e._uU(55,' is a common phenotype in the population, but as an HPO annotation, it is intended to represent "unusually" blue irides given the population background of an affected individual, e.g., with '),e.TgZ(56,"a",4),e._uU(57,"Williams syndrome"),e.qZA(),e._uU(58,". "),e.qZA(),e.TgZ(59,"p"),e._uU(60,"Clinicians should also use their judgment in entering terms. For instance, if a patient has severe myopia, this feature is more likely to be related to an underlying disease than if a patient was found to have a very mild degree of myopia (a feature which is extremely common in the population and may just be a coincidence). "),e.qZA(),e.TgZ(61,"p"),e._uU(62,'Users of HPO software often ask how many terms they need to enter for HPO-based software to "work". It is difficult to provide a general answer to this question, but given that a medical record may contain up to hundreds of "abnormal" observations that would be difficult to enter by hand in software, some guidelines can be useful. We recommend that users enter HPO terms for all "major" phenotypic abnormalities, perhaps 5-7 in the first round. In some cases it may be advisable to enter higher level terms; for instance, in some forms of skeletal dysplasia, affected individuals may have several abnormalities of all or most phalanges. Although very specific HPO terms exist for these (e.g., '),e.TgZ(63,"a",5)(64,"em"),e._uU(65,"Bullet-shaped middle phalanx of the 5th toe"),e.qZA()(),e._uU(66," and so on for the other toes), it may be preferable to enter "),e.TgZ(67,"a",6)(68,"em"),e._uU(69,"Bullet-shaped toe phalanx"),e.qZA()(),e._uU(70," if many phalanges of the toes are affected. We recommend trying several combinations of phenotypes to get a good feeling for how term choice can affect the results. "),e.qZA()()(),e.TgZ(71,"h4")(72,"em"),e._uU(73,"Patient Health Information Disclaimer"),e.qZA()(),e.TgZ(74,"p"),e._uU(75,"The HPO project offers a number of demonstration tools on the internet that intend to show prospective users how to run the tools and interpret the output. We note that in no case should Protected Health Information be transmitted to these tools across the internet. Instead, downloadable versions of these tools (such as the "),e.TgZ(76,"a",7),e._uU(77,"Exomiser"),e.qZA(),e._uU(78,") should be used behind appropriate firewalls."),e.qZA()())},dependencies:[t.yS,q.pp,q.ib,q.yz,q.yK]}),s})(),A=(()=>{class s{constructor(){}ngOnInit(){}}return s.\u0275fac=function(l){return new(l||s)},s.\u0275cmp=e.Xpm({type:s,selectors:[["app-definitions"]],decls:242,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://github.com/obophenotype/human-phenotype-ontology/issues","target","__blank"],["routerLink","/browse/term/HP:0005120"],["routerLink","/browse/term/HP:0001631"],["routerLink","/browse/term/HP:0045075"],["routerLink","/browse/term/HP:0000653"],["href","https://www.ncbi.nlm.nih.gov/pubmed/22504886","target","__blank"],["href","https://github.com/obophenotype/human-phenotype-ontology/issues","target","_blank"],[1,"contribute-example"],["href","https://hpo.jax.org/app/browse/term/HP:0003674","target","_blank"],["routerLink","/browse/term/HP:0004389"],["routerLink","/browse/term/HP:0001943"],["routerLink","/browse/term/HP:0025153"],["routerLink","/browse/term/HP:0002611"],["routerLink","/browse/term/HP:0003259"],["routerLink","/browse/term/HP:0003138"],["routerLink","/tools/workbench"]],template:function(l,_){1&l&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Collaborating with HPO"),e.qZA(),e.TgZ(3,"h2"),e._uU(4,"Contributing"),e.qZA(),e.TgZ(5,"p"),e._uU(6," We welcome the participation of interested colleagues. We have been collaborating with many clinicians and translational researchers to refine and extend the HPO resources since 2009, and anticipate continuing this process for the foreseeable future. Groups or persons with expert knowledge in a particular domain of human phenotyping in a medical genetics setting are invited to contribute their knowledge on a collaborative basis. "),e.qZA(),e.TgZ(7,"h2"),e._uU(8,"New term requests"),e.qZA(),e.TgZ(9,"p"),e._uU(10,"The simplest way to contribute to the HPO is to make a new term request on our issue tracker "),e.TgZ(11,"a",2),e._uU(12,"on Github"),e.qZA(),e._uU(13,". The following text is intended to serve as a guide for anyone who would like to contribute to the HPO project by making new term requests. "),e.qZA(),e.TgZ(14,"h3"),e._uU(15,"Does the term you are looking for already exist?"),e.qZA(),e.TgZ(16,"p"),e._uU(17,"First, please look at the current HPO and check whether the term is already there. Use the browser on this website."),e.qZA(),e.TgZ(18,"h3"),e._uU(19,"Is the term you are looking for a synonym of an existing term?"),e.qZA(),e.TgZ(20,"p"),e._uU(21,"If you do not immediately find the term you are looking for, please look for synonyms. One way of doing this is to go to a likely parent of the term and peruse all of the children to spot a likely synonym. For instance, if you are looking for a term entitled "),e.TgZ(22,"em"),e._uU(23,"Defect in the atrial septum"),e.qZA(),e._uU(24," and do not find it, go to the term "),e.TgZ(25,"a",3),e._uU(26,"Abnormal cardiac atrium morphology"),e.qZA(),e._uU(27,". and look through all of the children. With some luck you will find the term "),e.TgZ(28,"a",4),e._uU(29,"Atrial septal defect"),e.qZA(),e._uU(30," and recognize that "),e.TgZ(31,"em"),e._uU(32,"Defect in the atrial septum"),e.qZA(),e._uU(33," is listed as a synonym of "),e.TgZ(34,"em"),e._uU(35,"Atrial septal defect"),e.qZA(),e._uU(36,". "),e.qZA(),e.TgZ(37,"h2"),e._uU(38,"Bundled terms"),e.qZA(),e.TgZ(39,"p"),e._uU(40," If you find a description in a publication such as "),e.TgZ(41,"em"),e._uU(42,"Sparse eyebrows and eyelashes"),e.qZA(),e._uU(43,", note that the description is referring to two separate phenotypic features. The HPO would encode this using two different terms, "),e.TgZ(44,"a",5),e._uU(45,"Sparse eyebrow"),e.qZA(),e._uU(46," and "),e.TgZ(47,"a",6),e._uU(48,"Sparse eyelashes"),e.qZA(),e._uU(49,". The idea is that one HPO term should refer to an "),e.TgZ(50,"em"),e._uU(51,"atomic"),e.qZA(),e._uU(52," phenotypic abnormality rather than to a collection of abnormalities observed in an individual patient. Please "),e.TgZ(53,"em"),e._uU(54,"debundle"),e.qZA(),e._uU(55," the description and proceed as described above. This is a central tenet of "),e.TgZ(56,"a",7),e._uU(57,"Deep phenotyping"),e.qZA(),e._uU(58,". "),e.qZA(),e.TgZ(59,"h3"),e._uU(60,"Anatomy of a good term suggestion"),e.qZA(),e.TgZ(61,"p"),e._uU(62,"A good term request provides the information shown in Table 1. Please use our "),e.TgZ(63,"a",8),e._uU(64,"GitHub issue tracker"),e.qZA(),e._uU(65,", choosing the \u201cNew Term\u201d template."),e.qZA(),e.TgZ(66,"table",9)(67,"tr")(68,"th"),e._uU(69,"Item"),e.qZA(),e.TgZ(70,"th"),e._uU(71,"Example"),e.qZA()(),e.TgZ(72,"tr")(73,"td"),e._uU(74,"Preferred term label"),e.qZA(),e.TgZ(75,"td"),e._uU(76,"Elevated D-dimers"),e.qZA()(),e.TgZ(77,"tr")(78,"td"),e._uU(79,"Synonyms"),e.qZA(),e.TgZ(80,"td"),e._uU(81,"Elevated D-dimer level"),e.qZA()(),e.TgZ(82,"tr")(83,"td"),e._uU(84,"Definition"),e.qZA(),e.TgZ(85,"td"),e._uU(86,"An increased concentration of D-dimers, a marker of fibrin degradation, in the blood circulation. PMID:19008457 "),e.qZA()(),e.TgZ(87,"tr")(88,"td"),e._uU(89,"Comment"),e.qZA(),e.TgZ(90,"td"),e._uU(91,"If appropriate, provide additional information or context to support the definition"),e.qZA()(),e.TgZ(92,"tr")(93,"td"),e._uU(94,"Parent term"),e.qZA(),e.TgZ(95,"td"),e._uU(96,"Abnormality of fibrinolysis HP:0040224 (it is easiest to copy the link to this term from the HPO website). "),e.qZA()(),e.TgZ(97,"tr")(98,"td"),e._uU(99,"Diseases characterized by this term"),e.qZA(),e.TgZ(100,"td"),e._uU(101,"2/2 individuals with Pseudo-TORCH syndrome 3 (OMIM:618886); PMID:31836668"),e.qZA()(),e.TgZ(102,"figcaption")(103,"strong"),e._uU(104,"Table 1"),e.qZA(),e._uU(105,". Components of a complete and well-structured new term request."),e.qZA()(),e.TgZ(106,"h3"),e._uU(107,"How to suggest new disease annotations"),e.qZA(),e.TgZ(108,"p"),e._uU(109,"We welcome suggestions for novel disease annotations (HPOAs) to add to HPO. These can be made by submitting a ticket to the HPO tracker with the information shown in Table 2. Please use our "),e.TgZ(110,"a",8),e._uU(111,"GitHub issue tracker"),e.qZA(),e._uU(112,", choosing the \u201cNew Annotation\u201d template."),e.qZA(),e.TgZ(113,"table",9)(114,"tr")(115,"th"),e._uU(116,"Item"),e.qZA(),e.TgZ(117,"th"),e._uU(118,"Example"),e.qZA()(),e.TgZ(119,"tr")(120,"td"),e._uU(121,"Disease ID"),e.qZA(),e.TgZ(122,"td"),e._uU(123,"OMIM:265380"),e.qZA()(),e.TgZ(124,"tr")(125,"td"),e._uU(126,"Disease Name"),e.qZA(),e.TgZ(127,"td"),e._uU(128,"Alveolar capillary dysplasia with misalignment of pulmonary veins1"),e.qZA()(),e.TgZ(129,"tr")(130,"td"),e._uU(131,"HPO ID"),e.qZA(),e.TgZ(132,"td"),e._uU(133,"HP:0001734"),e.qZA()(),e.TgZ(134,"tr")(135,"td"),e._uU(136,"HPO Term Name"),e.qZA(),e.TgZ(137,"td"),e._uU(138,"Annular pancreas"),e.qZA()(),e.TgZ(139,"tr")(140,"td"),e._uU(141,"Frequency"),e.qZA(),e.TgZ(142,"td"),e._uU(143,"2/14"),e.qZA()(),e.TgZ(144,"tr")(145,"td"),e._uU(146,"Onset"),e.qZA(),e.TgZ(147,"td"),e._uU(148,"Congenital onset (use terms from the HPO "),e.TgZ(149,"a",10),e._uU(150,"Onset"),e.qZA(),e._uU(151," hierarchy) "),e.qZA()(),e.TgZ(152,"tr")(153,"td"),e._uU(154,"PMID"),e.qZA(),e.TgZ(155,"td"),e._uU(156,"PMID:19500772"),e.qZA()(),e.TgZ(157,"tr")(158,"td"),e._uU(159,"Comment"),e.qZA(),e.TgZ(160,"td"),e._uU(161,"Any other information or context"),e.qZA()(),e.TgZ(162,"figcaption")(163,"strong"),e._uU(164,"Table 2"),e.qZA(),e._uU(165,". Components of a complete and well-structured new disease annotation request. "),e.qZA()(),e.TgZ(166,"h2"),e._uU(167,"The distinction between diseases and phenotypes"),e.qZA(),e.TgZ(168,"p"),e._uU(169,"The community uses the word "),e.TgZ(170,"em"),e._uU(171,"phenotype"),e.qZA(),e._uU(172," with multiple meanings. The HPO defines a disease as an entity that has all four of the following features:"),e.qZA(),e.TgZ(173,"ul")(174,"li"),e._uU(175,"an etiology (whether identified or as yet unknown)"),e.qZA(),e.TgZ(176,"li"),e._uU(177,"a time course"),e.qZA(),e.TgZ(178,"li"),e._uU(179,"a set of phenotypic features"),e.qZA(),e.TgZ(180,"li"),e._uU(181,"if treatments exist, there is a characteristic response to them"),e.qZA()(),e._uU(182," A phenotype (better: "),e.TgZ(183,"em"),e._uU(184,"phenotypic feature"),e.qZA(),e._uU(185,") is a component of a disease. HPO terms can be used to describe the set of phenotypic features that characterize a disease. For instance, if the disease is the common cold, then the phenotypes would be runny nose, fever, cough, fatigue, etc. Therefore, in the following description: "),e.TgZ(186,"pre"),e._uU(187,"    a sepsis-like condition with intestinal pseudoobstruction, transient hypoglycemia,\n    cholestatic hepatitis, and transient renal failure (maximum plasma creatinine\n    132 \u03bcmol/L and urea 11 mmol/L at day 5 of life, which normalized on day 10).\n  "),e.qZA(),e._uU(188,' We would conceptualize the "sepsis-like condition" as a disease, and would use HPO terms to describe '),e.TgZ(189,"ul")(190,"li")(191,"a",11),e._uU(192,"Intestinal pseudo-obstruction"),e.qZA()(),e.TgZ(193,"li")(194,"a",12),e._uU(195,"Hypoglycemia"),e.qZA(),e._uU(196," with the modifier "),e.TgZ(197,"a",13),e._uU(198,"Transient"),e.qZA(),e._uU(199,". "),e.qZA(),e.TgZ(200,"li")(201,"a",14),e._uU(202,"Cholestatic liver disease"),e.qZA(),e._uU(203,"."),e.qZA(),e.TgZ(204,"li")(205,"a",15),e._uU(206,"Elevated serum creatinine"),e.qZA(),e._uU(207,"."),e.qZA(),e.TgZ(208,"li")(209,"a",16),e._uU(210,"Increased blood urea nitrogen"),e.qZA(),e._uU(211,"."),e.qZA()(),e.TgZ(212,"p"),e._uU(213,"Thus, the HPO considers a disease to be an entity that has a known or unknown cause, is characterized by one or more phenotypic features which can affect all or only a subset of individuals with the disease, a time course over which the phenotypic features may have onset and evolve, and in some cases one or more indicated treatments and a response to treatment. For instance, if the disease entity is the common cold, then the cause is a virus, the phenotypic features include fever, cough, runny nose, and fatigue, the time course usually is a relatively acute onset with manifestations dragging on for days to about a week, and the treatment may include bed rest, aspirin, or nasal sprays. In contrast, a phenotypic feature such as fever is a manifestation of many diseases. There is a grey zone between diseases and phenotypic features. For instance, diabetes mellitus can be conceptualized as a disease, but it is also a feature of other diseases such as Bardet Biedl syndrome. The HPO takes a practical stance and provides terms for such entities."),e.qZA(),e.TgZ(214,"h2"),e._uU(215,"What is the desired level of granularity of an HPO term?"),e.qZA(),e.TgZ(216,"p"),e._uU(217," In general, each HPO term refers to a recognizable entity that occurs in multiple patients and often in multiple diseases. We do not try to create an HPO term for very specific manifestations in individual patients. For instance, "),e.TgZ(218,"em"),e._uU(219,"hamartomatous proliferation containing malformed hair follicles in various stages of development"),e.qZA(),e._uU(220," would be too detailed to be a good HPO term. "),e.qZA(),e.TgZ(221,"h2"),e._uU(222,"Anatomy of an HPO term"),e.qZA(),e.TgZ(223,"p"),e._uU(224," Once you have convinced yourself that the item you need is not already present in the HPO, please provide us with the following information "),e.qZA(),e.TgZ(225,"ul")(226,"li"),e._uU(227,"Preferred Label: What is the name of the term? This should be the name most commonly used by the community."),e.qZA(),e.TgZ(228,"li"),e._uU(229,"Synonyms. If you are aware of synonyms for you term, please include them in your term request."),e.qZA(),e.TgZ(230,"li"),e._uU(231,"Definition. Please try to formulate a definition of your term that will be comprehensible to non-specialists. "),e.qZA(),e.TgZ(232,"li"),e._uU(233,"References. Please include a PubMed ID if possible, so that other users of the HPO can find more informa stion about your term. "),e.qZA(),e.TgZ(234,"li"),e._uU(235," Parent term. If possible, please suggest where your new term should be placed within the existing ontology. It is sufficient to write the name(s) of the parent term(s) (i.e., you do not need to tell us the HPO ID, e.g., HP:1234567). "),e.qZA()(),e.TgZ(236,"h2"),e._uU(237,"HPO Workbench"),e.qZA(),e._uU(238," We are developing a new app called "),e.TgZ(239,"a",17),e._uU(240," HPO Workbench"),e.qZA(),e._uU(241," for making term suggestions. While the app is still in testing stages, it is stable and has been used for production purposes. HPO Workbench is Java app for working with the HPO that allows users to browse the ontology and disease annotations. In order to make suggestions, it is possible to make GitHub issues on our tracker directly from HPO Workbench. Alternatively, it is possible to export Excel files with all terms from a subhierarchy of interest (e.g., all liver-related terms).\n"),e.qZA())},dependencies:[t.yS],styles:[".contribute-example[_ngcontent-%COMP%], .contribute-example[_ngcontent-%COMP%]   tr[_ngcontent-%COMP%], .contribute-example[_ngcontent-%COMP%]   td[_ngcontent-%COMP%], .contribute-example[_ngcontent-%COMP%]   th[_ngcontent-%COMP%]{border:0!important}.contribute-example[_ngcontent-%COMP%]   th[_ngcontent-%COMP%]{background-color:#d3d3d3;text-align:left}.contribute-example[_ngcontent-%COMP%]   td[_ngcontent-%COMP%]:nth-child(1){font-weight:700}.contribute-example[_ngcontent-%COMP%]   figcaption[_ngcontent-%COMP%]{margin:1rem 0 0}"]}),s})();var m=r(4475);r(4147);var R=r(4850),z=r(529);let S=(()=>{class s{constructor(l){this.http=l}getPublications(){return this.http.get("https://raw.githubusercontent.com/monarch-initiative/hpo-web-config/master/hpocitations.txt",{responseType:"text"}).pipe((0,R.U)(l=>this.buildPublications(l.split("\n"))))}buildPublications(l){const _=[];if(l)for(const b of l){const w=b.split("\t");if(!w[0].startsWith("#")&&""!==b){let k=w.shift();k=k.split(",").length>=3?k.split(",").slice(0,4).join(",")+" et al.":k;const E={authors:k,title:w.shift(),journal:w.shift(),year:+w.shift(),volume:w.shift(),pages:w.shift(),pmid:w.shift(),inhouse:"T"===w.shift(),hpo:"T"===w.shift(),monarch:"T"===w.shift(),topicList:w.shift().split(";")};_.push(E)}}return _.sort((b,w)=>b.year>w.year?-1:b.year<w.year?1:0)}}return s.\u0275fac=function(l){return new(l||s)(e.LFG(z.eN))},s.\u0275prov=e.Yz7({token:s,factory:s.\u0275fac,providedIn:"root"}),s})();var I=r(5254),B=r(1709),D=r(1177);class Z{constructor(u,l){this.keySelector=u,this.flushes=l}call(u,l){return l.subscribe(new p(u,this.keySelector,this.flushes))}}class p extends D.Ds{constructor(u,l,_){super(u),this.keySelector=l,this.values=new Set,_&&this.add((0,D.ft)(_,new D.IY(this)))}notifyNext(){this.values.clear()}notifyError(u){this._error(u)}_next(u){this.keySelector?this._useKeySelector(u):this._finalizeNext(u,u)}_useKeySelector(u){let l;const{destination:_}=this;try{l=this.keySelector(u)}catch(b){return void _.error(b)}this._finalizeNext(l,u)}_finalizeNext(u,l){const{values:_}=this;_.has(u)||(_.add(u),this.destination.next(l))}}var g=r(2014),y=r(2821),d=r(7314),U=r(4843);function te(s,u,l){return 0===l?[u]:(s.push(u),s)}var x=r(2143),j=r(6308),J=r(4859),Y=r(9549),oe=r(4385),ae=r(3238);function Q(s,u){if(1&s&&(e.TgZ(0,"mat-option",23),e._uU(1),e.qZA()),2&s){const l=u.$implicit;e.Q6J("value",l),e.xp6(1),e.Oqu(l)}}function ie(s,u){if(1&s&&(e.TgZ(0,"p",24),e._uU(1),e.qZA()),2&s){const l=e.oxw();e.xp6(1),e.AsE("Displaying ",l.publications.filteredData.length," of ",l.publications.data.length," Publications")}}function X(s,u){1&s&&(e.TgZ(0,"mat-header-cell",25),e._uU(1," Year"),e.qZA())}function V(s,u){if(1&s&&(e.TgZ(0,"mat-cell"),e._uU(1),e.qZA()),2&s){const l=u.$implicit;e.xp6(1),e.hij(" ",l.year," ")}}function M(s,u){1&s&&(e.TgZ(0,"mat-header-cell"),e._uU(1," Title"),e.qZA())}function re(s,u){if(1&s&&(e.TgZ(0,"mat-cell"),e._uU(1),e.qZA()),2&s){const l=u.$implicit;e.xp6(1),e.hij(" ",l.title," ")}}function K(s,u){1&s&&(e.TgZ(0,"mat-header-cell"),e._uU(1," Authors"),e.qZA())}function se(s,u){if(1&s&&(e.TgZ(0,"mat-cell"),e._uU(1),e.qZA()),2&s){const l=u.$implicit;e.xp6(1),e.hij(" ",l.authors," ")}}function le(s,u){1&s&&(e.TgZ(0,"mat-header-cell",25),e._uU(1," Journal"),e.qZA())}function ce(s,u){if(1&s&&(e.TgZ(0,"mat-cell"),e._uU(1),e.qZA()),2&s){const l=u.$implicit;e.xp6(1),e.hij(" ",l.journal," ")}}function he(s,u){1&s&&e._UZ(0,"mat-header-row")}function ue(s,u){1&s&&e._UZ(0,"mat-row")}const me=t.Bz.forChild([{path:"resources",children:[{path:"clinician-guide",component:P},{path:"contributing",component:A},{path:"faq",component:m.v},{path:"publications",component:(()=>{class s{constructor(l){this.pubService=l,this.displayedColumns=["authors","year","title","journal"],this.category="all",this.topics=new o.p4,this.topicList=[]}ngOnInit(){this.pubService.getPublications().subscribe(l=>{this.publications=new x.by(l),this.publications.sort=this.sort,this.publications.filterPredicate=this.filterPredicate,this.findUniquePublicationTopics(l)})}findUniquePublicationTopics(l){(0,I.D)(l).pipe((0,B.VS)(b=>b.topicList),function h(s,u){return l=>l.lift(new Z(s,u))}(),(0,R.U)(b=>b.toUpperCase()),function ne(){return function G(s,u){return arguments.length>=2?function(_){return(0,U.z)((0,g.R)(s,u),(0,y.h)(1),(0,d.d)(u))(_)}:function(_){return(0,U.z)((0,g.R)((b,w,k)=>s(b,w,k+1)),(0,y.h)(1))(_)}}(te,[])}()).subscribe(b=>{this.topicList=b})}filterPredicate(l,_){const b=_.split("~");if(0==("all"===(ee=b[0])||"us"===ee&&!0===l.inhouse))return!1;var ee;const E=""===b[1].split(",")[0]||"null"===b[1].split(",")[0]?0:b[1].split(",").length,$=l.topicList.map(L=>b[1].includes(L.toUpperCase()));return E<l.topicList.length?$.filter(L=>L).length===E:E===l.topicList.length?$.every(L=>!0===L):0===E}applyPublicationsFilter(){this.publications.filter=this.category+"~"+this.topics.value}}return s.\u0275fac=function(l){return new(l||s)(e.Y36(S))},s.\u0275cmp=e.Xpm({type:s,selectors:[["app-publications"]],viewQuery:function(l,_){if(1&l&&e.Gf(j.YE,7),2&l){let b;e.iGM(b=e.CRH())&&(_.sort=b.first)}},decls:37,vars:8,consts:[[1,"container","publications"],[1,"center","pageTitle"],[1,"table"],[1,"filters"],[1,"categories"],[3,"value","selectionChange","valueChange"],["value","all"],["value","us"],[1,"topics"],["panelClass","topics-panel","multiple","",3,"formControl","selectionChange"],[3,"value",4,"ngFor","ngForOf"],["class","count",4,"ngIf"],["matSort","",1,"mat-elevation-z8",3,"dataSource"],["matColumnDef","year"],["mat-sort-header","",4,"matHeaderCellDef"],[4,"matCellDef"],["matColumnDef","title"],[4,"matHeaderCellDef"],["matColumnDef","authors"],["matColumnDef","journal"],[4,"matHeaderRowDef","matHeaderRowDefSticky"],[4,"matRowDef","matRowDefColumns"],["mat-raised-button","","routerLink","/citation",1,"mat-btn"],[3,"value"],[1,"count"],["mat-sort-header",""]],template:function(l,_){1&l&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Publications"),e.qZA(),e.TgZ(3,"div",2)(4,"div",3)(5,"mat-form-field",4)(6,"mat-label"),e._uU(7,"Categories"),e.qZA(),e.TgZ(8,"mat-select",5),e.NdJ("selectionChange",function(){return _.applyPublicationsFilter()})("valueChange",function(w){return _.category=w}),e.TgZ(9,"mat-option",6),e._uU(10,"All"),e.qZA(),e.TgZ(11,"mat-option",7),e._uU(12,"By Us"),e.qZA()()(),e.TgZ(13,"mat-form-field",8)(14,"mat-label"),e._uU(15,"Topics"),e.qZA(),e.TgZ(16,"mat-select",9),e.NdJ("selectionChange",function(){return _.applyPublicationsFilter()}),e.YNc(17,Q,2,2,"mat-option",10),e.qZA()(),e.YNc(18,ie,2,2,"p",11),e.qZA(),e.TgZ(19,"mat-table",12),e.ynx(20,13),e.YNc(21,X,2,0,"mat-header-cell",14),e.YNc(22,V,2,1,"mat-cell",15),e.BQk(),e.ynx(23,16),e.YNc(24,M,2,0,"mat-header-cell",17),e.YNc(25,re,2,1,"mat-cell",15),e.BQk(),e.ynx(26,18),e.YNc(27,K,2,0,"mat-header-cell",17),e.YNc(28,se,2,1,"mat-cell",15),e.BQk(),e.ynx(29,19),e.YNc(30,le,2,0,"mat-header-cell",14),e.YNc(31,ce,2,1,"mat-cell",15),e.BQk(),e.YNc(32,he,1,0,"mat-header-row",20),e.YNc(33,ue,1,0,"mat-row",21),e.qZA()(),e._UZ(34,"br"),e.TgZ(35,"a",22),e._uU(36,"View HPO Citation"),e.qZA()()),2&l&&(e.xp6(8),e.Q6J("value",_.category),e.xp6(8),e.Q6J("formControl",_.topics),e.xp6(1),e.Q6J("ngForOf",_.topicList),e.xp6(1),e.Q6J("ngIf",_.publications),e.xp6(1),e.Q6J("dataSource",_.publications),e.xp6(13),e.Q6J("matHeaderRowDef",_.displayedColumns)("matHeaderRowDefSticky",!0),e.xp6(1),e.Q6J("matRowDefColumns",_.displayedColumns))},dependencies:[n.sg,n.O5,t.yS,J.zs,Y.KE,Y.hX,x.BZ,x.fO,x.as,x.w1,x.Dz,x.nj,x.ge,x.ev,x.XQ,x.Gk,j.YE,j.nU,oe.gD,ae.ey,o.JJ,o.oH],styles:[".publications[_ngcontent-%COMP%]   ul[_ngcontent-%COMP%]{list-style-type:none;line-height:30px;text-align:justify}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]{max-height:600px;overflow-y:scroll}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-header-cell[_ngcontent-%COMP%]{padding:15px 5px 0 10px;background-color:#4e8ba2;color:#fff!important}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-header-cell[_ngcontent-%COMP%]:first-child{padding-left:24px}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-column-year[_ngcontent-%COMP%]{max-width:-moz-fit-content;max-width:fit-content;text-align:center}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-column-journal[_ngcontent-%COMP%]{max-width:15%}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   mat-row[_ngcontent-%COMP%]   .mat-column-journal[_ngcontent-%COMP%]{padding:0 15px}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   mat-row[_ngcontent-%COMP%]   .mat-column-title[_ngcontent-%COMP%]{padding:0 10px}.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   mat-row[_ngcontent-%COMP%]   .mat-column-year[_ngcontent-%COMP%]{padding:0 10px}@media screen and (max-width: 767px){.publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-column-journal[_ngcontent-%COMP%], .publications[_ngcontent-%COMP%]   mat-table[_ngcontent-%COMP%]   .mat-column-authors[_ngcontent-%COMP%]{display:none}}.publications[_ngcontent-%COMP%]   .filters[_ngcontent-%COMP%]{margin-top:15px}.publications[_ngcontent-%COMP%]   .filters[_ngcontent-%COMP%]   .topics[_ngcontent-%COMP%]{margin-left:25px}.publications[_ngcontent-%COMP%]   .filters[_ngcontent-%COMP%]   .count[_ngcontent-%COMP%]{float:right}@media screen and (max-width: 767px){.publications[_ngcontent-%COMP%]   .filters[_ngcontent-%COMP%]   .topics[_ngcontent-%COMP%]{margin-left:0}.publications[_ngcontent-%COMP%]   .filters[_ngcontent-%COMP%]   .count[_ngcontent-%COMP%]{float:none;float:initial}}a.mat-btn[_ngcontent-%COMP%]{text-decoration:none;color:#fff!important;background-color:#ec9736}"]}),s})()},{path:"phenopacket",component:(()=>{class s{constructor(){}ngOnInit(){}}return s.\u0275fac=function(l){return new(l||s)},s.\u0275cmp=e.Xpm({type:s,selectors:[["app-phenopacket"]],decls:32,vars:0,consts:[[1,"container","phenopacket"],[1,"center","pageTitle"],["target","__blank","href","https://phenopacket-schema.readthedocs.io/en/latest/"],["target","__blank","href","https://www.ga4gh.org/"],["href","https://www.iso.org/standard/79991.html","target","__blank"],["href","https://pubmed.ncbi.nlm.nih.gov/35705716/","target","__blank"],[1,"phenopacket-figure"],["alt","GA4GH Phenopacket","src","https://onlinelibrary.wiley.com/cms/asset/1cc0a141-da65-45a3-b7b0-6316b7b02069/ggn2202200016-fig-0002-m.jpg"],[1,"phenopacket-figure-caption"],["target","__blank","href","https://onlinelibrary.wiley.com/doi/full/10.1002/ggn2.202200016"]],template:function(l,_){1&l&&(e.TgZ(0,"div",0)(1,"h1",1),e._uU(2,"Global Alliance for Genomic Health Phenopacket"),e.qZA(),e.TgZ(3,"p"),e._uU(4,"The Monarch Initiative, the consortium that includes the HPO team, has led the development of the the "),e.TgZ(5,"a",2),e._uU(6,"Phenopacket Schema"),e.qZA(),e._uU(7," for the "),e.TgZ(8,"a",3),e._uU(9,"Global Alliance for Genomics and Health (GA4GH)"),e.qZA(),e._uU(10,". The Phenopacket Schema was approved by the GA4GH in 2022 and additional certified by the International Standards Organization (ISO) as "),e.TgZ(11,"a",4),e._uU(12,"ISO 4454:2022 Genomics informatics \u2014 Phenopackets: A format for phenotypic data exchange"),e.qZA(),e._uU(13,"."),e.qZA(),e.TgZ(14,"p"),e._uU(15,"More information is available in "),e.TgZ(16,"a",5),e._uU(17,"Jacobsen JOB, et al. The GA4GH Phenopacket schema defines a computable representation of clinical data. Nat Biotechnol. 2022 Jun;40(6):817-820."),e.qZA(),e._uU(18,". "),e.qZA(),e.TgZ(19,"p"),e._uU(20,"A Phenopacket characterizes an individual person or biosample, linking that individual to detailed phenotypic descriptions, genetic information, diagnoses, and treatments. The Phenopacket schema supports the FAIR principles (findable, accessible, interoperable, and reusable), and computability. Specifically, Phenopackets are designed to be both human and machine-interpretable, enabling computing operations and validation on the basis of defined relationships between diagnoses, lab measurements, and genotypic information."),e.qZA(),e.TgZ(21,"p"),e._uU(22,"The Phenopacket schema enables comparison of sets of phenotypic attributes from individual patients. Such comparisons can aid in diagnosis and facilitate patient classification and stratification for identifying new diseases and treatments. The Phenopacket schema is designed to support interoperability between people, organizations, and systems that comprise the worldwide effort to understand human disease. The structure of the information in a phenopacket was designed for integration within clinical laboratories, journals, data repositories, patient registries, electronic health records (EHRs), and knowledge bases. Increasing the volume of computable data across a diversity of systems will support global disease analysis by integrating genotype, phenotype, and other multi-modal data for precision health applications."),e.qZA(),e.TgZ(23,"figure",6),e._UZ(24,"img",7),e.TgZ(25,"figcaption",8),e._uU(26,"Phenopacket Schema overview. The GA4GH Phenopacket Schema is a hierarchical structure that consists of two required fields, ID and MetaData, as well as eight optional fields, Individual, Disease, Interpretation, Biosample, PhenotypicFeature, Measurement, MedicalAction, and files."),e.qZA()(),e.TgZ(27,"p"),e._uU(28," The Phenopacket provides a standardized way of reporting HPO terms and providing additional context. A detailed introduction in how to represent clinical data as a phenopacket is provided in "),e.TgZ(29,"a",9),e._uU(30,"Ladewig et al., GA4GH Phenopackets: A Practical Introduction, Advanced Genetics 2022"),e.qZA(),e._uU(31,". "),e.qZA()())},styles:[".phenopacket[_ngcontent-%COMP%]   .phenopacket-figure[_ngcontent-%COMP%]{margin:0 auto;text-align:center}.phenopacket[_ngcontent-%COMP%]   .phenopacket-figure[_ngcontent-%COMP%]   .phenopacket-figure-caption[_ngcontent-%COMP%]{margin:inherit;text-align:left;width:600px}.phenopacket[_ngcontent-%COMP%]   .phenopacket-figure[_ngcontent-%COMP%]   img[_ngcontent-%COMP%]{width:600px}@media screen and (max-width: 640px){.phenopacket[_ngcontent-%COMP%]   .phenopacket-figure[_ngcontent-%COMP%]{display:none}}"]}),s})()}]}]);let fe=(()=>{class s{}return s.\u0275fac=function(l){return new(l||s)},s.\u0275mod=e.oAB({type:s}),s.\u0275inj=e.cJS({imports:[t.Bz,me,t.Bz]}),s})(),_e=(()=>{class s{}return s.\u0275fac=function(l){return new(l||s)},s.\u0275mod=e.oAB({type:s}),s.\u0275inj=e.cJS({providers:[S],imports:[n.ez,fe,T.c,o.u5,o.UX]}),s})()},7381:(W,C,r)=>{r.r(C),r.d(C,{StaticModule:()=>we});var n=r(6895),T=r(3917),o=r(2109),t=r(4650),e=r(4859),q=r(7392),P=r(3162),A=r(6338),m=r(1576),v=r(5829),N=r(6505);function R(a,c){if(1&a&&(t.TgZ(0,"mat-list-item")(1,"div",47)(2,"a",48),t._uU(3),t.qZA()(),t.TgZ(4,"span",49),t._uU(5),t.qZA()()),2&a){const i=c.$implicit;t.xp6(2),t.MGl("routerLink","/news/",i.monthYear,""),t.xp6(1),t.Oqu(i.teaserTitle),t.xp6(2),t.Oqu(i.date)}}function z(a,c){if(1&a&&(t.TgZ(0,"div")(1,"mat-list",44),t.YNc(2,R,6,3,"mat-list-item",45),t.qZA(),t.TgZ(3,"div",13)(4,"a",46),t._uU(5,"View All News"),t.qZA()()()),2&a){const i=t.oxw();t.xp6(2),t.Q6J("ngForOf",i.teaserNews)}}function S(a,c){1&a&&(t.TgZ(0,"div"),t._uU(1," Temporarily Unavailable. Please "),t.TgZ(2,"a",50),t._uU(3,"contact us "),t.qZA(),t._uU(4," if this does not return soon. "),t.qZA())}function I(a,c){1&a&&t._UZ(0,"mat-progress-bar",51)}let B=(()=>{class a{constructor(i){this.newsService=i,this.loadingNews=!1}ngOnInit(){this.loadingNews=!0,this.newsService.getTeaserNews().subscribe(i=>{this.teaserNews=i},i=>{this.newsError=!0,this.loadingNews=!1,console.error(i)},()=>this.loadingNews=!1)}}return a.\u0275fac=function(i){return new(i||a)(t.Y36(o.Y))},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-home"]],decls:108,vars:3,consts:[["fxLayout","row"],["fxFlex.gt-md","65","fxFlexOffset.gt-md","17","fxFlex","90","fxFlexOffset","5",1,"home-search"],[1,"search-container","mat-elevation-z4"],["src","assets/hpo-logo-white.png",1,"main-logo"],[1,"searchbar-container"],["fxLayout","row wrap","fxLayoutAlign","space-evenly",1,"info-section"],["fxFlex.gt-md","45","fxFlex","80"],[1,"info-body"],["routerLink","/browse/term/HP:0001631"],["href","https://monarchinitiative.org/","target","__blank"],["href","https://www.ga4gh.org/howwework/driver-projects.html","target","__blank"],["href"," https://www.ga4gh.org/","target","__blank"],["href","https://www.ga4gh.org/howwework/strategic-roadmap.html"],[1,"button-wrapper"],["mat-raised-button","","routerLink","about",1,"mat-btn"],["fxFlex.gt-md","25","fxFlex","80"],[1,"info-body","teaser"],[4,"ngIf"],["mode","indeterminate",4,"ngIf"],["fxLayout","row wrap",1,"home-highlight-menu"],["routerLink","/tools/exomiser","fxFlex.gt-md","35","fxFlexOffset.gt-md","10","fxFlex","90","fxFlexOffset","5",1,"home-item","dark"],["fxFlex","row","fxLayoutAlign","space-between center"],["src","assets/highlight-images/exomiser.png",1,"tool-icon"],[1,"home-body"],[1,"title"],[1,"description"],["fxShow","","fxHide.xs","",1,"material-icons"],["routerLink","/tools/phenomizer","fxFlex.gt-md","35","fxFlexOffset.gt-md","10","fxFlex","90","fxFlexOffset","5",1,"home-item","light"],["src","assets/highlight-images/phenomizer.png",1,"tool-icon"],["routerLink","/tools/genomiser","fxFlex.gt-md","35","fxFlexOffset.gt-md","10","fxFlex","90","fxFlexOffset","5",1,"home-item","light"],["src","assets/highlight-images/genomiser.png",1,"tool-icon"],["routerLink","/browse/profile-search","fxFlex.gt-md","35","fxFlexOffset.gt-md","10","fxFlex","90","fxFlexOffset","5",1,"home-item","dark"],["src","assets/highlight-images/patientarchive.png",1,"tool-icon"],["fxLayout","row","fxLayoutAlign","space-between center"],["fxFlex","80","fxFlexOffset","10","routerLink","resources/phenopacket",1,"phenopacket-banner","mat-elevation-z1"],["src","assets/phenopackets.png",1,"phenopacket-logo"],["fxHide.lt-sm","",1,"description"],[1,"text"],[1,"mailing-list-wrapper"],["href","https://groups.io/g/human-phenotype-ontology","target","_blank"],[1,"mailing-list"],[1,"mailing-list-body"],["href","http://www.irdirc.org/research/irdirc-recognized-resources/","target","__blank"],["src","assets/irdirc-logo.jpg",1,"irdirc"],[1,"list"],[4,"ngFor","ngForOf"],["mat-raised-button","","routerLink","/news",1,"mat-btn"],[1,"teaser-title"],[3,"routerLink"],[1,"teaser-date"],["routerLink","/contact"],["mode","indeterminate"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"div",1)(2,"div",2),t._UZ(3,"img",3),t.TgZ(4,"div",4),t._UZ(5,"searchbar"),t.qZA()()()(),t.TgZ(6,"div",5)(7,"div",6)(8,"h2"),t._uU(9,"The Human Phenotype Ontology"),t.qZA(),t._UZ(10,"hr"),t.TgZ(11,"div",7)(12,"p"),t._uU(13,"The Human Phenotype Ontology (HPO) provides a standardized vocabulary of phenotypic abnormalities encountered in human disease. Each term in the HPO describes a phenotypic abnormality, such as "),t.TgZ(14,"a",8),t._uU(15,"Atrial septal defect"),t.qZA(),t._uU(16,". The HPO is currently being developed using the medical literature, Orphanet, DECIPHER, and OMIM. HPO currently contains over 13,000 terms and over 156,000 annotations to hereditary diseases. The HPO project and others have developed software for phenotype-driven differential diagnostics, genomic diagnostics, and translational research. The HPO is a flagship product of the "),t.TgZ(17,"a",9),t._uU(18,"Monarch Initiative"),t.qZA(),t._uU(19,", an NIH-supported international consortium dedicated to semantic integration of biomedical and model organism data with the ultimate goal of improving biomedical research. The HPO, as a part of the Monarch Initiative, is a central component of one of the "),t.TgZ(20,"a",10),t._uU(21,"13 driver projects"),t.qZA(),t._uU(22," in the "),t.TgZ(23,"a",11),t._uU(24,"Global Alliance for Genomics and Health"),t.qZA(),t._uU(25," (GA4GH) "),t.TgZ(26,"a",12),t._uU(27,"strategic roadmap"),t.qZA(),t._uU(28,"."),t.qZA(),t.TgZ(29,"div",13)(30,"a",14),t._uU(31,"Learn More About HPO"),t.qZA()()()(),t.TgZ(32,"div",15)(33,"h2"),t._uU(34,"News & Updates"),t.qZA(),t._UZ(35,"hr"),t.TgZ(36,"div",16),t.YNc(37,z,6,1,"div",17),t.YNc(38,S,5,0,"div",17),t.YNc(39,I,1,0,"mat-progress-bar",18),t.qZA()()(),t.TgZ(40,"div",19)(41,"div",20)(42,"div",21),t._UZ(43,"img",22),t.TgZ(44,"div",23)(45,"h2",24),t._uU(46,"Exomiser"),t.qZA(),t.TgZ(47,"p",25)(48,"i"),t._uU(49,"Evaluate variants based on the predicted pathogenicity."),t.qZA()()(),t.TgZ(50,"i",26),t._uU(51," chevron_right "),t.qZA()()(),t.TgZ(52,"div",27)(53,"div",21),t._UZ(54,"img",28),t.TgZ(55,"div",23)(56,"h2",24),t._uU(57,"Phenomizer"),t.qZA(),t.TgZ(58,"p",25)(59,"i"),t._uU(60,"Rank disease differential diagnosis by clinical features."),t.qZA()()(),t.TgZ(61,"i",26),t._uU(62," chevron_right "),t.qZA()()(),t.TgZ(63,"div",29)(64,"div",21),t._UZ(65,"img",30),t.TgZ(66,"div",23)(67,"h2",24),t._uU(68,"Genomiser"),t.qZA(),t.TgZ(69,"p",25)(70,"i"),t._uU(71,"Analyze genome sequence data for non-coding variants."),t.qZA()()(),t.TgZ(72,"i",26),t._uU(73," chevron_right "),t.qZA()()(),t.TgZ(74,"div",31)(75,"div",21),t._UZ(76,"img",32),t.TgZ(77,"div",23)(78,"h2",24),t._uU(79,"Profile Search"),t.qZA(),t.TgZ(80,"p",25)(81,"i"),t._uU(82,"Discover diseases with a phenotype profile."),t.qZA()()(),t.TgZ(83,"i",26),t._uU(84," chevron_right "),t.qZA()()()(),t._UZ(85,"br")(86,"br"),t.TgZ(87,"div",33)(88,"div",34),t._UZ(89,"img",35),t.TgZ(90,"div",36)(91,"h4"),t._uU(92,"Phenopackets"),t.qZA(),t.TgZ(93,"span",37),t._uU(94,"A Global Alliance for Genomics and Health (GA4GH) international standard for phenotypic data exchange."),t.qZA()()()(),t._UZ(95,"br")(96,"br"),t.TgZ(97,"div",38)(98,"a",39)(99,"div",40)(100,"mat-icon"),t._uU(101,"mail_outline"),t.qZA(),t.TgZ(102,"div",41)(103,"h4"),t._uU(104,"HPO Mailing List!"),t.qZA(),t._uU(105," Get official announcements and updates from the core developers. "),t.qZA()()()(),t.TgZ(106,"a",42),t._UZ(107,"img",43),t.qZA()),2&i&&(t.xp6(37),t.Q6J("ngIf",f.teaserNews),t.xp6(1),t.Q6J("ngIf",f.newsError),t.xp6(1),t.Q6J("ngIf",f.loadingNews))},dependencies:[n.sg,n.O5,e.zs,q.Hw,P.pW,A.i$,A.Tg,m.xw,m.Wh,m.UT,m.yH,v.b8,N.g,T.rH,T.yS],styles:[".container.container-main[_ngcontent-%COMP%]   .container-title[_ngcontent-%COMP%]{font-size:20px;margin-top:0;font-weight:700}.container.container-main[_ngcontent-%COMP%]   .container-body[_ngcontent-%COMP%]{font-size:16px;text-align:justify}.container.container-main[_ngcontent-%COMP%]   .container-body[_ngcontent-%COMP%]   a.btn[_ngcontent-%COMP%]{border-radius:0}.empty-space[_ngcontent-%COMP%]{height:25px;width:100%}.quick-links-card[_ngcontent-%COMP%]   mat-list-item[_ngcontent-%COMP%]{background-color:#d0d3d4;opacity:.7;margin-bottom:5px}.quick-links-card[_ngcontent-%COMP%]   mat-list-item[_ngcontent-%COMP%]:hover{background-color:#d0d3d4;opacity:.9}.quick-links-card[_ngcontent-%COMP%]   mat-card-title[_ngcontent-%COMP%]   h3[_ngcontent-%COMP%]{margin:0}a.mat-btn[_ngcontent-%COMP%]{text-decoration:none;background-color:#ec9736}.quick-links-card[_ngcontent-%COMP%]   a[mat-list-item][_ngcontent-%COMP%]{background-color:#d0d3d4;opacity:.7;margin-bottom:5px}.quick-links-card[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]:hover{text-decoration:none}.quick-links-card[_ngcontent-%COMP%]   a[mat-list-item][_ngcontent-%COMP%]:hover{background-color:#d0d3d4;opacity:.9}.news-updates[_ngcontent-%COMP%]{margin-top:5px}.home-search[_ngcontent-%COMP%]   .search-container[_ngcontent-%COMP%]{margin:0 auto;max-height:100%;height:250px;background-color:#29657c;border:solid 2px #325364}.home-search[_ngcontent-%COMP%]   .search-container[_ngcontent-%COMP%]   .main-logo[_ngcontent-%COMP%]{margin:1rem auto;height:100px;display:block}@media screen and (min-width: 767px){.home-search[_ngcontent-%COMP%]   .search-container[_ngcontent-%COMP%]   .searchbar-container[_ngcontent-%COMP%]{width:50%;margin:0 auto}}@media screen and (max-width: 767px){.home-search[_ngcontent-%COMP%]   .search-container[_ngcontent-%COMP%]   .main-logo[_ngcontent-%COMP%]{height:75px}}.info-section[_ngcontent-%COMP%]{margin-top:25px}.info-section[_ngcontent-%COMP%] > div[_ngcontent-%COMP%]{margin:1rem}.info-body[_ngcontent-%COMP%]   p[_ngcontent-%COMP%], .info-body[_ngcontent-%COMP%]   .list[_ngcontent-%COMP%]{min-height:150px}.info-body[_ngcontent-%COMP%]   .list[_ngcontent-%COMP%]{-webkit-margin-before:1em;-webkit-margin-after:1em;-webkit-margin-start:0;-webkit-margin-end:0;-moz-margin-start:0;padding:0}.teaser-title[_ngcontent-%COMP%]{font-weight:700;font-size:16px;font-style:italic}.teaser-date[_ngcontent-%COMP%]{justify-content:flex-end;margin:0 0 0 auto}.mailing-list[_ngcontent-%COMP%]{width:300px;background-color:#ec9736;padding:15px;color:#fff;position:absolute;box-shadow:1px 1px 8px #a9a9a9}.mailing-list-wrapper[_ngcontent-%COMP%]   a[_ngcontent-%COMP%], .mailing-list-wrapper[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]:hover, .mailing-list-wrapper[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]:active, .mailing-list-wrapper[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]:visited, .mailing-list-wrapper[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]:focus{text-decoration:none}.mailing-list-body[_ngcontent-%COMP%]{margin:0 0 0 50px;font-weight:700}.mailing-list-body[_ngcontent-%COMP%]   h4[_ngcontent-%COMP%]{margin:0}.mailing-list[_ngcontent-%COMP%]   .material-icons[_ngcontent-%COMP%]{font-size:40px;margin:10px 0;float:left}a[_ngcontent-%COMP%]   .irdirc[_ngcontent-%COMP%]{height:100px;float:right;box-shadow:1px 1px 8px #a9a9a9}@media (max-width: 767px){.mailing-list[_ngcontent-%COMP%]{position:static;position:initial}a[_ngcontent-%COMP%]   .irdirc[_ngcontent-%COMP%]{height:100px;float:left;box-shadow:1px 1px 8px #a9a9a9}.home-highlight-menu[_ngcontent-%COMP%]{margin:50px 0 0}}.home-highlight-menu[_ngcontent-%COMP%]{margin:50px auto 5px}@media screen and (max-width: 640px){.home-highlight-menu[_ngcontent-%COMP%]{margin:50px 0 5px}}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]{height:100px;color:#fff;cursor:pointer;display:flex;padding-right:15px;padding-left:15px}@media screen and (max-width: 640px){.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]{margin-bottom:5px}}.home-highlight-menu[_ngcontent-%COMP%]   .home-item.dark[_ngcontent-%COMP%]{background:rgb(39,84,103)}.home-highlight-menu[_ngcontent-%COMP%]   .home-item.light[_ngcontent-%COMP%]{background:rgb(41,101,124)}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]   .tool-icon[_ngcontent-%COMP%]{height:100px;width:100px;padding:0 0 0 5px;float:left;display:inline-block}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]:hover{opacity:.8}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]   .home-body[_ngcontent-%COMP%]{float:left;margin:0 0 0 25px;flex:1}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]   .title[_ngcontent-%COMP%]{margin:15px 0 0}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]   .description[_ngcontent-%COMP%]{margin-top:0}.home-highlight-menu[_ngcontent-%COMP%]   .home-item[_ngcontent-%COMP%]   .material-icons[_ngcontent-%COMP%]{font-size:40px}.phenopacket-banner[_ngcontent-%COMP%]{height:200px;padding:25px;display:flex;flex:0;margin-left:25px;cursor:pointer}.phenopacket-banner[_ngcontent-%COMP%]:hover{opacity:.75}.phenopacket-banner[_ngcontent-%COMP%]   .description[_ngcontent-%COMP%]{flex:1 auto;margin:3rem 0 0 3rem;font-size:2rem}.phenopacket-banner[_ngcontent-%COMP%]   .description[_ngcontent-%COMP%]   h4[_ngcontent-%COMP%]{margin:.5rem 0}.phenopacket-banner[_ngcontent-%COMP%]   a[_ngcontent-%COMP%]{display:inherit;color:inherit!important;text-decoration:none}@media screen and (max-width: 958px){.phenopacket-banner[_ngcontent-%COMP%]   a[_ngcontent-%COMP%], .phenopacket-banner[_ngcontent-%COMP%]   .phenopacket-logo[_ngcontent-%COMP%]{margin:0 auto}}.phenopacket-banner[_ngcontent-%COMP%]   .button[_ngcontent-%COMP%]{height:36px;float:right;margin:0 10rem 0 0}"]}),a})();function D(a,c){if(1&a&&(t.TgZ(0,"div",7)(1,"div",8),t._UZ(2,"img",9),t.qZA(),t.TgZ(3,"div",10)(4,"h3"),t._uU(5),t.qZA(),t.TgZ(6,"h4"),t._uU(7),t.qZA(),t.TgZ(8,"a",11)(9,"mat-icon"),t._uU(10,"email_outline"),t.qZA()()()()),2&a){const i=c.$implicit;t.xp6(2),t.Q6J("src",i.src,t.LSH)("alt",i.name),t.xp6(3),t.Oqu(i.name),t.xp6(2),t.Oqu(i.affiliation),t.xp6(1),t.Q6J("href","mailto:"+i.email,t.LSH)}}function h(a,c){if(1&a&&(t.TgZ(0,"div",12)(1,"div",8),t._UZ(2,"img",9),t.qZA(),t.TgZ(3,"div",10)(4,"h3"),t._uU(5),t.qZA(),t.TgZ(6,"h4"),t._uU(7),t.qZA(),t.TgZ(8,"a",11)(9,"mat-icon"),t._uU(10,"email_outline"),t.qZA()()()()),2&a){const i=c.$implicit;t.xp6(2),t.Q6J("src",i.src,t.LSH)("alt",i.name),t.xp6(3),t.Oqu(i.name),t.xp6(2),t.Oqu(i.affiliation),t.xp6(1),t.Q6J("href","mailto:"+i.email,t.LSH)}}let Z=(()=>{class a{constructor(){this.generalContacts=[{name:"Sebastian K\xf6hler",affiliation:"Monarch Initiative & Information Architect at Ada Health",email:"dr.sebastian.koehler@gmail.com",src:"assets/contact-images/sebastian-kohler.jpg"},{name:"Peter Robinson",affiliation:"The Jackson Laboratory for Genomic Medicine",email:"peter.robinson@jax.org",src:"assets/contact-images/peter-robinson.jpg"},{name:"Melissa Haendel",affiliation:"University of Colorado Center for Health AI",email:"melissa@tislab.org",src:"assets/contact-images/melissa-haendel.jpg"}],this.technicalContacts=[{name:"Michael Gargano",affiliation:"The Jackson Laboratory for Genomic Medicine",email:"michael.gargano@jax.org",src:"assets/contact-images/michael-gargano.jpg"}]}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-contact"]],decls:13,vars:2,consts:[[1,"container"],[1,"center","pageTitle"],["fxLayout","row wrap","fxLayoutAlign","center center"],["fxFlex.gt-sm","50","fxFlexOffset","25","fxLayout","column",1,"contact-general"],["class","contact-card","fxLayout","row",4,"ngFor","ngForOf"],["fxFlex.gt-sm","50","fxFlexOffset","25","fxLayout","column",1,"contact-technical"],["class","contact-card",4,"ngFor","ngForOf"],["fxLayout","row",1,"contact-card"],[1,"contact-img"],[3,"src","alt"],["fxFlex","85",1,"contact-bio"],["target","_blank",3,"href"],[1,"contact-card"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"div")(2,"h1",1),t._uU(3,"Contact"),t.qZA(),t.TgZ(4,"div",2)(5,"div",3)(6,"h4"),t._uU(7," For general human phenotype ontology questions: "),t.qZA(),t.YNc(8,D,11,5,"div",4),t.qZA(),t.TgZ(9,"div",5)(10,"h4"),t._uU(11," For technical hpo web application and api questions or bugs: "),t.qZA(),t.YNc(12,h,11,5,"div",6),t.qZA()()()()),2&i&&(t.xp6(8),t.Q6J("ngForOf",f.generalContacts),t.xp6(4),t.Q6J("ngForOf",f.technicalContacts))},dependencies:[n.sg,q.Hw,m.xw,m.Wh,m.UT,m.yH],styles:[".contact-card[_ngcontent-%COMP%]{margin:1rem}.contact-card[_ngcontent-%COMP%]   .contact-img[_ngcontent-%COMP%]{width:80px;height:80px;flex-shrink:0;border-radius:999px;overflow:hidden}.contact-card[_ngcontent-%COMP%]   .contact-img[_ngcontent-%COMP%]   img[_ngcontent-%COMP%]{width:100%;height:100%;-o-object-fit:cover;object-fit:cover}.contact-card[_ngcontent-%COMP%]   .contact-bio[_ngcontent-%COMP%]{margin:0 0 0 1rem}.contact-card[_ngcontent-%COMP%]   .contact-bio[_ngcontent-%COMP%]   h3[_ngcontent-%COMP%], .contact-card[_ngcontent-%COMP%]   .contact-bio[_ngcontent-%COMP%]   h4[_ngcontent-%COMP%]{margin:.5rem .5rem .5rem 0}.contact-card[_ngcontent-%COMP%]   .contact-bio[_ngcontent-%COMP%]   .mat-icon[_ngcontent-%COMP%]{cursor:pointer}"]}),a})(),p=(()=>{class a{constructor(){}ngOnInit(){}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-citation"]],decls:10,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://pubmed.ncbi.nlm.nih.gov/33264411/","target","__blank"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"Citation"),t.qZA(),t.TgZ(3,"p"),t._uU(4,"Please cite our "),t.TgZ(5,"a",2),t._uU(6,"most recent article"),t.qZA(),t._uU(7," if you use the HPO for published work."),t.qZA(),t.TgZ(8,"h4"),t._uU(9,"Sebastian K\xf6hler, Michael Gargano, Nicolas Matentzoglu, Leigh C Carmody, David Lewis-Smith, Nicole A Vasilevsky, Daniel Danis, Ganna Balagura, Gareth Baynam, Amy M Brower, Tiffany J Callahan, Christopher G Chute, Johanna L Est, Peter D Galer, Shiva Ganesan, Matthias Griese, Matthias Haimel, Julia Pazmandi, Marc Hanauer, Nomi L Harris, Michael J Hartnett, Maximilian Hastreiter, Fabian Hauck, Yongqun He, Tim Jeske, Hugh Kearney, Gerhard Kindle, Christoph Klein, Katrin Knoflach, Roland Krause, David Lagorce, Julie A McMurry, Jillian A Miller, Monica C Munoz-Torres, Rebecca L Peters, Christina K Rapp, Ana M Rath, Shahmir A Rind, Avi Z Rosenberg, Michael M Segal, Markus G Seidel, Damian Smedley, Tomer Talmy, Yarlalu Thomas, Samuel A Wiafe, Julie Xian, Zafer Y\xfcksel, Ingo Helbig, Christopher J Mungall, Melissa A Haendel, Peter N Robinson, The Human Phenotype Ontology in 2021, Nucleic Acids Research, Volume 49, Issue D1, 8 January 2021, Pages D1207\u2013D1217, https://doi.org/10.1093/nar/gkaa1043"),t.qZA()())}}),a})(),g=(()=>{class a{constructor(){}ngOnInit(){}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-license"]],decls:35,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["routerLink","/citation"],["href","https://github.com/obophenotype/human-phenotype-ontology/issues","target","__blank"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"License"),t.qZA(),t.TgZ(3,"p"),t._uU(4,"All of the following license restrictions apply to all files provided by the HPO project:"),t.qZA(),t.TgZ(5,"ol")(6,"li"),t._uU(7," The HPO vocabularies, annotation files, tools and documentation are freely available. "),t.qZA(),t.TgZ(8,"li"),t._uU(9," The HPO is copyrighted to protect the integrity of the vocabularies, which means that changes to the HPO vocabularies need to be done by HPO developers. However, anyone can download the HPO and use the ontologies or other HPO files under three conditions:"),t._UZ(10,"br"),t.TgZ(11,"ul")(12,"li"),t._uU(13,"That the Human Phenotype Ontology Consortium is "),t.TgZ(14,"strong"),t._uU(15,"acknowledged"),t.qZA(),t._uU(16," and "),t.TgZ(17,"strong")(18,"a",2),t._uU(19,"cited"),t.qZA()(),t._uU(20," properly. "),t.qZA(),t.TgZ(21,"li"),t._uU(22," That any HPO Consortium file(s) displayed publicly "),t.TgZ(23,"i"),t._uU(24,"include the date(s) and/or version number(s)"),t.qZA(),t._uU(25," of the relevant HPO file(s). "),t.qZA(),t.TgZ(26,"li"),t._uU(27," That neither the content of the HPO file(s) nor the logical relationships embedded within the HPO file(s) be altered in any way. (Content additions and modifications have to be suggested using our "),t.TgZ(28,"a",3),t._uU(29,"issue tracker"),t.qZA(),t._uU(30,".) "),t.qZA()()(),t.TgZ(31,"li"),t._uU(32," We suggest that you follow standard scientific citation practice if you use the HPO in publications. "),t.qZA(),t.TgZ(33,"li"),t._uU(34,"Users of the HPO should add the following statement to their online presence. This service/product uses the Human Phenotype Ontology (version information). Find out more at http://www.human-phenotype-ontology.org We request that the HPO logo be included as well. "),t.qZA()()())},dependencies:[T.yS]}),a})();var y=r(4475),d=r(2289),U=r(3267);const G=["sidenav"],te=function(a){return{active:a}};function ne(a,c){if(1&a){const i=t.EpF();t.TgZ(0,"mat-list-item",10),t.NdJ("click",function(){const H=t.CHM(i).$implicit,F=t.oxw();return t.KtG(F.setCurrentDate(H))}),t.TgZ(1,"div",11),t._uU(2),t.qZA()()}if(2&a){const i=c.$implicit,f=t.oxw();t.Q6J("ngClass",t.VKq(2,te,f.currentSelected==i)),t.xp6(2),t.hij(" ",i," ")}}function x(a,c){if(1&a&&(t.TgZ(0,"div",12)(1,"div",13)(2,"span",14),t._uU(3),t.qZA(),t.TgZ(4,"span",15),t._uU(5),t.qZA(),t._UZ(6,"br"),t.qZA(),t._UZ(7,"br")(8,"div",16),t.qZA()),2&a){const i=c.$implicit;t.xp6(3),t.Oqu(i.title),t.xp6(2),t.Oqu(i.date),t.xp6(3),t.Q6J("innerHtml",i.body,t.oJD)}}function j(a,c){1&a&&(t.TgZ(0,"div",1)(1,"h3"),t._uU(2,"Sorry something went wrong. Please try again, if this error continues "),t.TgZ(3,"a",17),t._uU(4,"contact"),t.qZA(),t._uU(5," the developers."),t.qZA()())}let J=(()=>{class a{constructor(i,f,O,H){this.newsService=i,this.route=f,this.currentSelected="",this.teaserDate="",this.error=!1,this.mobileQuery=H.matchMedia("(max-width: 600px)"),this._mobileQueryListener=()=>O.detectChanges(),this.mobileQuery.addEventListener("change",this._mobileQueryListener)}ngOnInit(){this.route.params.subscribe(i=>{this.teaserDate=i.id}),this.newsService.getUniqueDates().subscribe(i=>{i?(this.dates=i,this.currentSelected=this.teaserDate&&i.includes(this.teaserDate)?this.teaserDate:i[0],this.updateNewsItems()):this.error=!0})}ngOnDestroy(){this.mobileQuery.removeEventListener("change",this._mobileQueryListener)}setCurrentDate(i){this.sidenav.close(),this.currentSelected=i,this.updateNewsItems()}updateNewsItems(){this.newsService.getNewsByDate(this.currentSelected).subscribe(i=>{i?this.content=i:this.error=!0})}}return a.\u0275fac=function(i){return new(i||a)(t.Y36(o.Y),t.Y36(T.gz),t.Y36(t.sBO),t.Y36(d.vx))},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-news"]],viewQuery:function(i,f){if(1&i&&t.Gf(G,5),2&i){let O;t.iGM(O=t.CRH())&&(f.sidenav=O.first)}},decls:14,vars:7,consts:[[1,"news","container"],[1,"center"],[1,"by-month-wrapper"],["mat-stroked-button","","color","primary",1,"toggler",3,"click"],[1,"hpo-sidenav-container"],["fixedTopGap","56",3,"mode","fixedInViewport"],["sidenav",""],[3,"ngClass","click",4,"ngFor","ngForOf"],["class","news-section mat-elevation-z4",4,"ngFor","ngForOf"],["class","center",4,"ngIf"],[3,"ngClass","click"],[1,"news-dates"],[1,"news-section","mat-elevation-z4"],[1,"news-title"],[1,"hr"],[1,"news-date"],[1,"news-body",3,"innerHtml"],["routerLink","/contact"]],template:function(i,f){if(1&i){const O=t.EpF();t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"News & Updates"),t.qZA(),t.TgZ(3,"div",2)(4,"button",3),t.NdJ("click",function(){t.CHM(O);const F=t.MAs(8);return t.KtG(F.toggle())}),t._uU(5,"Select Month"),t.qZA()(),t.TgZ(6,"mat-sidenav-container",4)(7,"mat-sidenav",5,6)(9,"mat-nav-list"),t.YNc(10,ne,3,4,"mat-list-item",7),t.qZA()(),t.TgZ(11,"mat-sidenav-content"),t.YNc(12,x,9,3,"div",8),t.qZA()(),t.YNc(13,j,6,0,"div",9),t.qZA()}2&i&&(t.xp6(6),t.Udp("margin-top",f.mobileQuery.matches?56:0,"px"),t.xp6(1),t.Q6J("mode",f.mobileQuery.matches?"over":"side")("fixedInViewport",f.mobileQuery.matches),t.xp6(3),t.Q6J("ngForOf",f.dates),t.xp6(2),t.Q6J("ngForOf",f.content),t.xp6(1),t.Q6J("ngIf",f.error))},dependencies:[n.mk,n.sg,n.O5,e.lW,A.Hk,A.Tg,U.JX,U.TM,U.Rh,v.oO,T.yS],styles:[".news-title[_ngcontent-%COMP%]{font-size:20px;font-weight:800}.news-date[_ngcontent-%COMP%]{font-size:14px;font-style:italic;font-weight:400;float:right}.news-timeline[_ngcontent-%COMP%]   h4[_ngcontent-%COMP%]{margin:10px 0}.news-section[_ngcontent-%COMP%]{padding:15px;margin-bottom:25px}.news-timeline[_ngcontent-%COMP%]   .news-dates[_ngcontent-%COMP%]{border-right:solid 1px lightgray;line-height:30px}.news-timeline[_ngcontent-%COMP%]   .news-dates[_ngcontent-%COMP%]   p[_ngcontent-%COMP%]{cursor:pointer}.news-timeline[_ngcontent-%COMP%]   .active[_ngcontent-%COMP%]{font-weight:700}.hr[_ngcontent-%COMP%]{width:-moz-fit-content!important;width:fit-content!important}.hpo-sidenav-container[_ngcontent-%COMP%]{min-height:60%}.news.container[_ngcontent-%COMP%]{height:100%}.toggler[_ngcontent-%COMP%]{vertical-align:middle}.toggler[_ngcontent-%COMP%]:hover{cursor:pointer;opacity:.8}.by-month-wrapper[_ngcontent-%COMP%]{text-align:center;margin-bottom:5px}"]}),a})(),Y=(()=>{class a{constructor(){}ngOnInit(){}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-disclaimer"]],decls:20,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["mat-stroked-button","","target","__blank","href","https://www.jax.org/privacy-policy"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"Warranty, Disclaimer, and Copyright Notice"),t.qZA(),t.TgZ(3,"p"),t._uU(4,"By using this site, you signify your assent to the Terms and Conditions set forth below. If you do not agree to all of these Terms and Conditions of use, do not use this site."),t.qZA(),t.TgZ(5,"p"),t._uU(6,'The contents of the Human Phenotype Ontology (HPO) Site, such as text and other material contained on the HPO Site ("Content") are for informational and educational purposes only. The use of the HPO Site and the Content is at your own risk. '),t.qZA(),t.TgZ(7,"p"),t._uU(8,"When using the HPO Site, information will be transmitted over a medium that may be beyond the control and jurisdiction of The Jackson Laboratory. Accordingly, The Jackson Laboratory assumes no liability for or relating to the delay, failure, interruption, or corruption of any data or other information transmitted in connection with use of the HPO Site."),t.qZA(),t.TgZ(9,"p"),t._uU(10,'The HPO Site and the Content are provided on an "as is" basis. THE JACKSON LABORATORY, TO THE FULLEST EXTENT PERMITTED BY LAW, DISCLAIMS ALL WARRANTIES, EITHER EXPRESS OR IMPLIED, STATUTORY OR OTHERWISE, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY, NON-INFRINGEMENT OF THIRD PARTIES\' RIGHTS, AND FITNESS FOR PARTICULAR PURPOSE. Without limiting the foregoing, The Jackson Laboratory makes no representations or warranties about the accuracy, reliability, completeness, currentness, or timeliness of the Content, software, text, or communications provided on or through the use of the HPO Site.'),t.qZA(),t.TgZ(11,"p"),t._uU(12,"In no event shall The Jackson Laboratory be liable for any damages (including, without limitation, incidental and consequential damages, personal injury/wrongful death, lost profits, or damages resulting from lost data or business interruption) resulting from the use of or inability to use the HPO Site or the Content, whether based on warranty, contract, tort, or any other legal theory, and whether or not The Jackson Laboratory is advised of the possibility of such damages. The Jackson Laboratory shall not be liable for any personal injury, including death, caused by your use or misuse of the HPO Site or content."),t.qZA(),t.TgZ(13,"p"),t._uU(14,"This HPO Site and content are provided to enhance knowledge and encourage progress in the scientific community and are to be used only for research and educational purposes. The HPO Site and content are not intended to be used for medical advice, diagnosis, or treatment."),t.qZA(),t.TgZ(15,"p"),t._uU(16,"The Jackson Laboratory authorizes you to view or download a single copy of the Content on the HPO Site solely for your personal, noncommercial use. Any reproduction or use for commercial purpose is prohibited without the prior express written permission of The Jackson Laboratory."),t.qZA(),t.TgZ(17,"p")(18,"a",2),t._uU(19," Click here to view The Jackson Laboratory privacy policy"),t.qZA()()())},dependencies:[e.zs]}),a})(),oe=(()=>{class a{constructor(){}ngOnInit(){}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-about"]],decls:103,vars:0,consts:[["fxLayout","row wrap",1,"about-us","container"],["fxFlexFill","",1,"center","pageTitle"],["fxFlex","70","fxFlexOffset","15"],[1,"sub-title"],["href","https://www.ncbi.nlm.nih.gov/pubmed/22504886","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/27899602","target","__blank"],["routerLink","/browse/term/HP:0000517"],["routerLink","/browse/term/HP:0012372"],["routerLink","https://hpo.jax.org/app/browse/term/HP:0000118"],["mat-flat-button","","color","primary","routerLink","/browse/term/HP:0000001"],["href","https://www.sciencedirect.com/topics/medicine-and-dentistry/human-phenotype-ontology","mat-stroked-button","","target","_blank",1,"science-direct"],["fxFlex","70","fxFlexOffset","15","fxLayout","row","fxLayoutAlign","space-evenly center",1,"about-us-banner"],["routerLink","/community",1,"banner-card"],[1,"icon-wrapper"],["routerLink","/resources/publications",1,"banner-card"],["routerLink","/citation",1,"banner-card"],["fxFlex","70","fxFlexOffset","15",1,"history"],["href","https://www.charite.de/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/18950739","target","__blank"],["href","http://www.lbl.gov/","target","__blank"],["href","https://www.cam.ac.uk/","target","__blank"],["href","https://uoregon.edu/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774506/","target","__blank"],["href","http://www.sanger.ac.uk/","target","__blank"],["href","https://monarchinitiative.org/","target","__blank"],["mat-flat-button","","color","primary","href","https://monarchinitiative.org/about/monarch","target","_blank",1,"mat-btn"],["fxFlex","70","fxFlexOffset","15","fxLayout","row","fxLayout.lt-md","column","fxLayoutAlign","center center",1,"contact-banner"],["fxFlex","","fxLayout","row","fxLayoutAlign","center center",1,"contact-banner-title-wrapper"],["fxFlex","",1,"contact-banner-title"],[1,"contact-icon"],["fxFlex.lt-md","100","fxLayout","row","fxLayoutAlign.gt-sm","end center","fxLayoutAlign","space-evenly center",1,"contact-banner-btn-wrapper"],["mat-stroked-button","","routerLink","/faq",1,"contact-btn"],["mat-stroked-button","","routerLink","/contact",1,"contact-btn"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"The Human Phenotype Ontology"),t.qZA(),t.TgZ(3,"div",2)(4,"h3",3),t._uU(5,"Who we are!"),t.qZA(),t.TgZ(6,"p"),t._uU(7," The Human Phenotype Ontology (HPO) project provides an ontology of medically relevant phenotypes, disease-phenotype annotations, and the algorithms that operate on these. The HPO can be used to support differential diagnostics, translational research, and a number of applications in computational biology by providing the means to "),t.TgZ(8,"em"),t._uU(9,"compute"),t.qZA(),t._uU(10," over the clinical phenotype. The HPO is being used for computational deep phenotyping and precision medicine as well as integration of clinical data into translational research. "),t.TgZ(11,"a",4),t._uU(12,"Deep phenotyping"),t.qZA(),t._uU(13," can be defined as the precise and comprehensive analysis of phenotypic abnormalities in which the individual components of the phenotype are observed and described. The HPO is being increasingly adopted as a standard for phenotypic abnormalities by diverse groups such as international rare disease organizations, registries, clinical labs, biomedical resources, and clinical software tools and will thereby contribute toward nascent efforts at global data exchange for identifying disease etiologies ("),t.TgZ(14,"a",5),t._uU(15,"K\xf6hler et al, 2017"),t.qZA(),t._uU(16,"). "),t.qZA(),t.TgZ(17,"p"),t._uU(18," The HPO currently contains over 13,000 terms arranged in a directed acyclic graph and are connected by is-a (subclass-of) edges, such that a term represents a more specific or limited instance of its parent term(s). All relationships in the HPO are is-a relationships, i.e. simple class-subclass relationships. For instance, "),t.TgZ(19,"a",6)(20,"em"),t._uU(21,"Abnormal lens morphology"),t.qZA()(),t._uU(22," is-a "),t.TgZ(23,"a",7)(24,"em"),t._uU(25,"Abnormal eye morphology"),t.qZA()(),t._uU(26,". The relationships are transitive, meaning that they are inherited up all paths to the root. "),t.TgZ(27,"a",8)(28,"em"),t._uU(29,"Phenotypic abnormality"),t.qZA()(),t._uU(30," is the main subontology of the HPO and contains descriptions of clinical abnormalities. Additional subontologies are provided to describe inheritance patterns, onset/clinical course and modifiers of abnormalities. "),t.qZA(),t.TgZ(31,"button",9),t._uU(32," Explore Ontology "),t.TgZ(33,"mat-icon"),t._uU(34,"keyboard_arrow_right"),t.qZA()(),t._uU(35," \xa0 "),t.TgZ(36,"a",10),t._uU(37,"HPO on Science Direct"),t.qZA()(),t.TgZ(38,"div",11)(39,"div",12)(40,"div",13)(41,"mat-icon"),t._uU(42,"groups"),t.qZA()(),t.TgZ(43,"h3"),t._uU(44,"Our Team"),t.qZA()(),t.TgZ(45,"div",14)(46,"div",13)(47,"mat-icon"),t._uU(48,"menu_book"),t.qZA()(),t.TgZ(49,"h3"),t._uU(50,"Publications"),t.qZA()(),t.TgZ(51,"div",15)(52,"div",13)(53,"mat-icon"),t._uU(54,"history_edu"),t.qZA()(),t.TgZ(55,"h3"),t._uU(56,"Cite Us"),t.qZA()()(),t.TgZ(57,"div",16)(58,"h3",3),t._uU(59,"Where it started!"),t.qZA(),t.TgZ(60,"p"),t._uU(61,"The Human Phenotype Ontology was initially developed at the "),t.TgZ(62,"a",17),t._uU(63,"Charite\u0301 University Hospital"),t.qZA(),t._uU(64," in Berlin, Germany by Peter N Robinson and Sebastian K\xf6hler with support from Sandra Doelken and Sebastian Bauer. Very soon after the "),t.TgZ(65,"a",18),t._uU(66,"initial publication"),t.qZA(),t._uU(67," of the HPO in 2008, a collaboration was initiated with researchers from the "),t.TgZ(68,"a",19),t._uU(69,"Lawrence Berkeley National Laboratory"),t.qZA(),t._uU(70," (Chris Mungall, Suzi Lewis, and Nicole Washington), the "),t.TgZ(71,"a",20),t._uU(72,"University of Cambridge"),t.qZA(),t._uU(73," (Michael Ashburner Paul Schofield, George Gkoutos), and the "),t.TgZ(74,"a",21),t._uU(75,"University of Oregon"),t.qZA(),t._uU(76," (Melissa Haendel, Monte Westerfield). These groups have developed a scheme for "),t.TgZ(77,"a",22),t._uU(78,"Linking human diseases to animal models using ontology-based phenotype annotation"),t.qZA(),t._uU(79,". In parallel, the DECIPHER project at the "),t.TgZ(80,"a",23),t._uU(81,"Wellcome Trust Sanger Institute"),t.qZA(),t._uU(82," led by Helen Firth and Matt Hurles, conducted clinical workshops to extend the HPO and have provided an enormous amount of support and input in subsequent years. "),t.qZA(),t.TgZ(83,"p"),t._uU(84," The initially unfunded collaboration grew over the years, and has now become the "),t.TgZ(85,"a",24),t._uU(86,"Monarch Initiative"),t.qZA(),t._uU(87,", now an NIH-supported project that aims to integrate biological information using semantics, and to use computational reasoning to enable phenotype comparison both within and across species, with the ultimate goal of improving biomedical research. "),t.qZA(),t.TgZ(88,"a",25),t._uU(89," Learn more about the Monarch Initiative "),t.TgZ(90,"mat-icon"),t._uU(91,"keyboard_arrow_right"),t.qZA()()(),t.TgZ(92,"div",26)(93,"div",27)(94,"h2",28)(95,"mat-icon",29),t._uU(96,"contact_support"),t.qZA(),t._uU(97," Still have some questions? "),t.qZA()(),t.TgZ(98,"div",30)(99,"button",31),t._uU(100,"Frequently Asked Questions"),t.qZA(),t.TgZ(101,"button",32),t._uU(102,"Contact us"),t.qZA()()()())},dependencies:[e.lW,e.zs,q.Hw,m.xw,m.Wh,m.UT,m.s9,m.yH,T.rH,T.yS],styles:[".about-us[_ngcontent-%COMP%]   .sub-title[_ngcontent-%COMP%]{text-transform:uppercase;color:#2f4f4f;opacity:.6}.about-us[_ngcontent-%COMP%]   p[_ngcontent-%COMP%]{font-size:16px}.about-us[_ngcontent-%COMP%]   .mat-flat-button.mat-primary[_ngcontent-%COMP%]{background-color:#ec9736;font-weight:700}.about-us[_ngcontent-%COMP%]   .about-us-banner[_ngcontent-%COMP%]{background-color:#29657c;color:#fff;padding:4rem 0;margin:2.5rem 0}.about-us[_ngcontent-%COMP%]   .banner-card[_ngcontent-%COMP%]{text-align:center;opacity:1;cursor:pointer}.about-us[_ngcontent-%COMP%]   .banner-card[_ngcontent-%COMP%]:hover{opacity:.85}.about-us[_ngcontent-%COMP%]   .banner-card[_ngcontent-%COMP%]   h3[_ngcontent-%COMP%]{margin:.5rem 0 0;text-transform:uppercase}.about-us[_ngcontent-%COMP%]   .banner-card[_ngcontent-%COMP%]   .icon-wrapper[_ngcontent-%COMP%]{text-align:center}.about-us[_ngcontent-%COMP%]   .banner-card[_ngcontent-%COMP%]   .icon-wrapper[_ngcontent-%COMP%]   .mat-icon[_ngcontent-%COMP%]{width:36px;font-size:36px;height:36px}.about-us[_ngcontent-%COMP%]   .history[_ngcontent-%COMP%]{margin-bottom:4rem}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]{background-color:#29657c;padding:1rem 2rem;color:#fff}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]   .contact-banner-title-wrapper[_ngcontent-%COMP%]{align-items:center}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]   .contact-banner-title[_ngcontent-%COMP%]{display:flex;vertical-align:middle}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]   .contact-icon[_ngcontent-%COMP%]{font-size:32px;height:32px;width:32px;margin:0 .5rem}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]   .contact-banner-btn-wrapper[_ngcontent-%COMP%]   .contact-btn[_ngcontent-%COMP%]{border-color:#fff}.about-us[_ngcontent-%COMP%]   .contact-banner[_ngcontent-%COMP%]   .contact-banner-btn-wrapper[_ngcontent-%COMP%]   .contact-btn[_ngcontent-%COMP%]:last-child{margin-left:1rem}"]}),a})();var ae=r(4850),Q=r(2340),ie=r(529);let X=(()=>{class a{constructor(i){this.http=i,this.contributors=[]}getContributors(){return this.http.get(Q.N.HPO_CONTRIBUTORS_URL,{responseType:"text"}).pipe((0,ae.U)(i=>this.buildContributors(i.split("\n"))))}buildContributors(i){if(i)for(const f of i){const O=f.split(","),H=O.shift(),F=O.shift(),Pe=O.join(",").replace(/['"]+/g,"");this.contributors.push({firstName:F,lastName:H,location:Pe})}return this.contributors}getTeamOrganizations(){return this.http.get(Q.N.HPO_TEAM_HPO_URL,{responseType:"json"})}}return a.\u0275fac=function(i){return new(i||a)(t.LFG(ie.eN))},a.\u0275prov=t.Yz7({token:a,factory:a.\u0275fac}),a})();var V=r(8739),M=r(2143),re=r(7730),K=r(3848),se=r(266);function le(a,c){if(1&a){const i=t.EpF();t.TgZ(0,"button",72),t.NdJ("click",function(){const H=t.CHM(i).$implicit,F=t.oxw();return t.KtG(F.scrollToView(H.name))}),t._uU(1),t.qZA()}if(2&a){const i=c.$implicit;t.xp6(1),t.Oqu(i.name)}}function ce(a,c){if(1&a&&(t.TgZ(0,"p",79),t._uU(1),t.qZA()),2&a){const i=t.oxw().$implicit;t.xp6(1),t.Oqu(i.description)}}function he(a,c){1&a&&t._UZ(0,"span",80)}function ue(a,c){1&a&&(t.TgZ(0,"mat-icon",86),t._uU(1,"hourglass_empty"),t.qZA())}function ge(a,c){if(1&a){const i=t.EpF();t.TgZ(0,"div",81),t.NdJ("click",function(){const H=t.CHM(i).$implicit,F=t.oxw(2);return t.KtG(F.navigate(H))}),t.TgZ(1,"div",82)(2,"mat-icon"),t._uU(3,"person"),t.qZA()(),t.TgZ(4,"h3",83),t._uU(5),t.YNc(6,ue,2,0,"mat-icon",84),t.qZA(),t.TgZ(7,"span",85),t._uU(8),t.qZA()()}if(2&a){const i=c.$implicit;t.xp6(5),t.hij(" ",i.name," "),t.xp6(1),t.Q6J("ngIf",i.alumni),t.xp6(2),t.Oqu(i.role)}}function pe(a,c){if(1&a&&(t.TgZ(0,"div",73)(1,"div",74)(2,"h2",75),t._uU(3),t.qZA(),t.YNc(4,ce,2,1,"p",76),t.YNc(5,he,1,0,"span",77),t.YNc(6,ge,9,3,"div",78),t.qZA(),t._UZ(7,"hr"),t.qZA()),2&a){const i=c.$implicit;t.Q6J("id",i.name.toLowerCase()),t.xp6(3),t.Oqu(i.name),t.xp6(1),t.Q6J("ngIf",i.description),t.xp6(1),t.Q6J("ngIf",i.description),t.xp6(1),t.Q6J("ngForOf",i.members)}}function de(a,c){1&a&&t._UZ(0,"mat-header-row")}function me(a,c){1&a&&t._UZ(0,"mat-row")}function fe(a,c){1&a&&(t.TgZ(0,"mat-header-cell"),t._uU(1," Last Name"),t.qZA())}function _e(a,c){if(1&a&&(t.TgZ(0,"mat-cell"),t._uU(1),t.qZA()),2&a){const i=c.$implicit;t.xp6(1),t.hij(" ",i.lastName," ")}}function s(a,c){1&a&&(t.TgZ(0,"mat-header-cell"),t._uU(1," First Name"),t.qZA())}function u(a,c){if(1&a&&(t.TgZ(0,"mat-cell"),t._uU(1),t.qZA()),2&a){const i=c.$implicit;t.xp6(1),t.hij(" ",i.firstName," ")}}function l(a,c){1&a&&(t.TgZ(0,"mat-header-cell"),t._uU(1," Location"),t.qZA())}function _(a,c){if(1&a&&(t.TgZ(0,"mat-cell"),t._uU(1),t.qZA()),2&a){const i=c.$implicit;t.xp6(1),t.hij(" ",i.location," ")}}function b(a,c){if(1&a&&(t.TgZ(0,"mat-table",87,88),t.YNc(2,de,1,0,"mat-header-row",89),t.YNc(3,me,1,0,"mat-row",90),t.ynx(4,91),t.YNc(5,fe,2,0,"mat-header-cell",92),t.YNc(6,_e,2,1,"mat-cell",93),t.BQk(),t.ynx(7,94),t.YNc(8,s,2,0,"mat-header-cell",92),t.YNc(9,u,2,1,"mat-cell",93),t.BQk(),t.ynx(10,95),t.YNc(11,l,2,0,"mat-header-cell",92),t.YNc(12,_,2,1,"mat-cell",93),t.BQk(),t.qZA()),2&a){const i=t.oxw();t.Q6J("dataSource",i.contributorsSource),t.xp6(2),t.Q6J("matHeaderRowDef",i.displayedColumns),t.xp6(1),t.Q6J("matRowDefColumns",i.displayedColumns)}}const w=function(){return[10,20]},L=T.Bz.forChild([{path:"",component:B},{path:"about",component:oe},{path:"citation",component:p},{path:"contact",component:Z},{path:"license",component:g},{path:"faq",component:y.v},{path:"faq/:id",component:y.v},{path:"news",component:J},{path:"news/:id",component:J},{path:"community",component:(()=>{class a{constructor(i,f){this.teamService=i,this.utilityService=f,this.displayedColumns=["lastName","firstName","location"]}ngOnInit(){this.teamService.getTeamOrganizations().subscribe(i=>{this.organizations=i}),this.teamService.getContributors().subscribe(i=>{this.contributorsSource=new M.by(i),this.contributorsSource.paginator=this.paginator})}scrollToView(i){document.getElementById(i.toLowerCase()).scrollIntoView({behavior:"smooth"})}navigate(i){i.alumni||this.utilityService.openExternalUrl(i.link)}}return a.\u0275fac=function(i){return new(i||a)(t.Y36(X),t.Y36(re.t))},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-community"]],viewQuery:function(i,f){if(1&i&&t.Gf(V.NW,5),2&i){let O;t.iGM(O=t.CRH())&&(f.paginator=O.first)}},decls:265,vars:5,consts:[[1,"container","community"],[1,"center","pageTitle"],[1,"quick-links"],["mat-align-tabs","start"],["label","Organizational Contributors","fxLayout","row wrap","fxLayoutAlign","center center"],["fxLayout","row wrap","fxLayoutAlign","center center",1,"org-navigation"],["mat-stroked-button","","color","primary",3,"click",4,"ngFor","ngForOf"],["class","organization-wrapper",3,"id",4,"ngFor","ngForOf"],["label","Individual Contributors"],[1,"contributors-table"],[3,"dataSource",4,"ngIf"],["showFirstLastButtons","","aria-label","Select page of periodic elements",3,"pageSizeOptions"],["label","Workshops"],["routerLink","/contact"],["href","https://www.ern-eye.eu/","target","__blank"],["href","https://stanfordhealthcare.org/doctors/w/matthew-wheeler.html","target","__blank"],["target","__blank","href","https://www.niaid.nih.gov/"],["target","__blank","href","https://www.ncbi.nlm.nih.gov/pubmed/25949529"],["label","Users"],["href","https://www.genome.gov/27544402/the-undiagnosed-diseases-program/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/28603714","target","__blank"],["href","http://www.orpha.net/consor/cgi-bin/index.php","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/29425702","target","__blank"],["href","https://www.genomecanada.ca/","target","__blank"],["href","http://www.informatics.jax.org/","target","__blank"],["href","https://www.genomicsengland.co.uk/","target","__blank"],["href","http://ecaruca.radboudumc.nl:8080/ecaruca/","target","__blank"],["href","https://decipher.sanger.ac.uk/","target","__blank"],["href","https://www.ddduk.org/","target","__blank"],["href","https://www.gwascentral.org/","target","__blank"],["href","http://jjwanglab.org/gwasdb","target","__blank"],["href","http://www.irdirc.org/","target","__blank"],["href","https://www.clinicalgenome.org/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/gtr/","target","__blank"],["href","http://rd-connect.eu/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/25029978","target","__blank"],["href","http://geno2mp.gs.washington.edu/Geno2MP/#/","target","__blank"],["href","http://www.genomes2people.org/the-medseq-project/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/clinvar/intro/","target","__blank"],["href","http://www.sequenceontology.org/","target","__blank"],["href","http://www.sequenceontology.org/so_wiki/index.php/Using_Phenotype_Ontologies_in_Human_GVF","target","__blank"],["href","https://esid.org/Working-Parties/Registry-Working-Party/Diagnosis-criteria","target","__blank"],["label","Other Tools"],[1,"row"],["href","https://www.fdna.com/","target","_blank"],["href","https://cartagenia.com/","target","_blank"],["href","http://gepado.de/en/","target","_blank"],["href","http://www.biodiscovery.com/nxclinical/","target","_blank"],["href","http://www.diploid.com/moon","target","__blank"],["href","https://www.centogene.com/","target","__blank"],["href","http://simulconsult.com/index.html","target","__blank"],["href","https://www.fabricgenomics.com/","target","_blank"],["href","http://resources.qiagenbioinformatics.com/white-papers/IVA_white_paper_FINAL.pdf"],["href","https://track.health/"],["href","https://pryzm.health/"],["href","http://extasy.esat.kuleuven.be/","target","_blank"],["href","http://phen-gen.org/","target","_blank"],["href","http://phenovar-dev.udes.genap.ca/","target","_blank"],["href","http://www.yandell-lab.org/software/phevor.html","target","_blank"],["href","https://track.health/","target","_blank"],["href","https://decipher.sanger.ac.uk/","target","_blank"],["href","https://phenopolis.org/","target","_blank"],["href","https://www.ddduk.org/","target","_blank"],["href","https://phenomecentral.org/","target","_blank"],["href","http://omimexplorer.research.bcm.edu","target","_blank"],["href","https://phenotips.org/","target","_blank"],["href","http://www.webgestalt.org/option.php","target","_blank"],["href","http://www.inetbio.org/morphin/","target","_blank"],["href","http://great.stanford.edu/public/html/","target","_blank"],["href","https://toppgene.cchmc.org/","target","_blank"],["href","http://supfam.org/SUPERFAMILY/","target","_blank"],["href","http://www.hetop.eu/hetop/","target","_blank"],["mat-stroked-button","","color","primary",3,"click"],[1,"organization-wrapper",3,"id"],["fxLayout","row wrap","fxFlex","100",1,"organization"],["fxFlex","100",1,"organization-name"],["class","organization-description","fxFlex","50","fxFlexOffset","25","fxFlex.lt-md","100","fxFlexOffset.lt-md","0",4,"ngIf"],["fxFlex","25",4,"ngIf"],["fxFlex","25","class","member",3,"click",4,"ngFor","ngForOf"],["fxFlex","50","fxFlexOffset","25","fxFlex.lt-md","100","fxFlexOffset.lt-md","0",1,"organization-description"],["fxFlex","25"],["fxFlex","25",1,"member",3,"click"],[1,"member-icon-wrapper"],[1,"member-name"],["matTooltip","Alumni","matTooltipPosition","above","matTooltipHideDelay","100000",4,"ngIf"],[1,"member-role"],["matTooltip","Alumni","matTooltipPosition","above","matTooltipHideDelay","100000"],[3,"dataSource"],["table",""],[4,"matHeaderRowDef"],[4,"matRowDef","matRowDefColumns"],["matColumnDef","lastName"],[4,"matHeaderCellDef"],[4,"matCellDef"],["matColumnDef","firstName"],["matColumnDef","location"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"Our Community"),t.qZA(),t.TgZ(3,"div",2)(4,"mat-tab-group",3)(5,"mat-tab",4)(6,"div",5),t.YNc(7,le,2,1,"button",6),t.qZA(),t.YNc(8,pe,8,5,"div",7),t.qZA(),t.TgZ(9,"mat-tab",8),t._UZ(10,"div",9),t.YNc(11,b,13,3,"mat-table",10),t._UZ(12,"mat-paginator",11),t.qZA(),t.TgZ(13,"mat-tab",12)(14,"p"),t._uU(15," This is a partial list of workshops (\u201chackathons\u201d) conducted over the last decade to add content to the HPO for specific areas of medicine. Please "),t.TgZ(16,"a",13),t._uU(17,"contact"),t.qZA(),t._uU(18," the HPO team if you would like to work with us on HPO terms for an area of medicine. "),t.qZA(),t.TgZ(19,"ul")(20,"li"),t._uU(21,"2017: Workshop on Ophthalmology phenotypes and rare diseases of the "),t.TgZ(22,"a",14),t._uU(23,"European Reference Network for Rare Eye Diseases"),t.qZA()(),t.TgZ(24,"li"),t._uU(25,"2017: Workshop on cardiology at Stanford University coordinated by "),t.TgZ(26,"a",15),t._uU(27,"Matthew Wheeler"),t.qZA()(),t.TgZ(28,"li"),t._uU(29,"2017: Workshop on Immunology terms at the "),t.TgZ(30,"a",16),t._uU(31,"National Institute for Allergy and Infectious Diseases"),t.qZA(),t._uU(32," coordinated by Morgan Similuk "),t.qZA(),t.TgZ(33,"li"),t._uU(34,"2015: Workshop with the UK's National Institute for Health Research (NIHR) Rare Disease initiatives on projects such as NIHR RD-TRC (Rare Disease\u2014Translational Research Collaboration, http://rd.trc.nihr.ac.uk) coordinated by Patrick Chinnery, Mareike Veltman and Roger James. "),t.qZA(),t.TgZ(35,"li"),t._uU(36,"2013: Workshop on epilepsy phenotypes and EEG abnormalities led by Prof. Ingo Helbig (currently Children\u2019s Hospital of Philadelphia, University of Pennsylvania) "),t.qZA(),t.TgZ(37,"li"),t._uU(38,"2012-2016: Work on bleeding phenotypes with contributions from multiple members of a team led by Prof. Willem Ouwehand of the University of Cambridge (see "),t.TgZ(39,"a",17),t._uU(40,"https://www.ncbi.nlm.nih.gov/pubmed/25949529"),t.qZA(),t._uU(41,") "),t.qZA(),t.TgZ(42,"li"),t._uU(43,"2012: 2-day workshop in Berlin, Germany, to extend dental terminology in the HPO led by Prof. Isabelle Bailleul-Forestier (Paul Sabatier University - Toulouse, France) "),t.qZA(),t.TgZ(44,"li"),t._uU(45,"2009: 5-day workshop at the Sanger Institute with UK clinicians involved with the DECIPHER project"),t.qZA()()(),t.TgZ(46,"mat-tab",18)(47,"p"),t._uU(48,"\tThe following list provides an overview of the groups who are currently using HPO. If you notice an omission, please contact us. "),t.qZA(),t.TgZ(49,"ul")(50,"li")(51,"a",19),t._uU(52,"NIH Undiagnosed disease program (UDP)"),t.qZA(),t._uU(53,". See also this paper that describes the UDP and its use of HPO: "),t.TgZ(54,"a",20),t._uU(55,"Gall T, et al (2017) Defining Disease, Diagnosis, and Translational Medicine within a Homeostatic Perturbation Paradigm: The National Institutes of Health Undiagnosed Diseases Program Experience. "),t.TgZ(56,"i"),t._uU(57,"Front Med (Lausanne)"),t.qZA(),t.TgZ(58,"b"),t._uU(59,"4"),t.qZA(),t._uU(60,":62"),t.qZA(),t._uU(61,"."),t.qZA(),t.TgZ(62,"li")(63,"a",21),t._uU(64,"Orphanet"),t.qZA(),t._uU(65,". See "),t.TgZ(66,"a",22),t._uU(67,"HIPBI-RD (Harmonising phenomics information for a better interoperability in the rare disease field)"),t.qZA(),t._uU(68," for information about work to integrate ORDO (the Orphanet Rare Disease Ontology), HPO (the Human Phenotype Ontology) as well as PhenoTips software for the capture and sharing of structured phenotypic data for RD patients."),t.qZA(),t.TgZ(69,"li")(70,"a",23),t._uU(71,"Genome Canada"),t.qZA(),t._uU(72,". "),t.qZA(),t.TgZ(73,"li")(74,"a",24),t._uU(75,"Mouse Genome Informatics (MGI)"),t.qZA()(),t.TgZ(76,"li")(77,"a",25),t._uU(78,"Genomics England; 100,000 Genomes Project"),t.qZA()(),t.TgZ(79,"li")(80,"a",26),t._uU(81,"ECARUCA"),t.qZA()(),t.TgZ(82,"li")(83,"a",27),t._uU(84,"DECIPHER"),t.qZA()(),t.TgZ(85,"li")(86,"a",28),t._uU(87,"Deciphering Developmental Disorders (DDD)"),t.qZA()(),t.TgZ(88,"li")(89,"a",29),t._uU(90,"GWAS Central"),t.qZA()(),t.TgZ(91,"li")(92,"a",30),t._uU(93,"GWASdb"),t.qZA()(),t.TgZ(94,"li")(95,"a",31),t._uU(96,"International Rare Diseases Research Consortium (IRDiRC)"),t.qZA()(),t.TgZ(97,"li")(98,"a",32),t._uU(99,"ClinGen"),t.qZA()(),t.TgZ(100,"li")(101,"a",33),t._uU(102,"NCBI Genetic Testing Registry"),t.qZA()(),t.TgZ(103,"li")(104,"a",34),t._uU(105,"RD-Connect"),t.qZA(),t._uU(106,". See this "),t.TgZ(107,"a",35),t._uU(108,"description"),t.qZA(),t._uU(109," of RD-CONNECT."),t.qZA(),t.TgZ(110,"li")(111,"a",36),t._uU(112,"Geno2MP"),t.qZA(),t._uU(113,"."),t.qZA(),t.TgZ(114,"li")(115,"a",37),t._uU(116,"MedSeq"),t.qZA(),t._uU(117,"."),t.qZA(),t.TgZ(118,"li")(119,"a",38),t._uU(120,"NCBI ClinVar"),t.qZA()(),t.TgZ(121,"li")(122,"a",39),t._uU(123,"Sequence Ontology (SO)"),t.qZA(),t._uU(124,". See this "),t.TgZ(125,"a",40),t._uU(126,"description"),t.qZA(),t._uU(127," of the use of phenotype ontologies in human GVF files."),t.qZA(),t.TgZ(128,"li")(129,"a",41),t._uU(130,"European Society for Immunodeficiencies"),t.qZA()()()(),t.TgZ(131,"mat-tab",42)(132,"p"),t._uU(133,"The HPO project welcomes other groups to use the HPO resources to develop tools and algorithms. This page has a list of academic and commercial tools that use the HPO. We invite feedback about omissions or errors in this list."),t.qZA(),t._UZ(134,"br"),t.TgZ(135,"h3"),t._uU(136,"Commerical entitites"),t.qZA(),t.TgZ(137,"div",43)(138,"ul")(139,"li")(140,"a",44),t._uU(141,"FDNA"),t.qZA(),t._uU(142,": Face2Gene."),t.qZA(),t.TgZ(143,"li")(144,"a",45),t._uU(145,"Cartagenia BENCH"),t.qZA()(),t.TgZ(146,"li")(147,"a",46),t._uU(148,"GEPADO \u2013 Software Solutions for Genetics - GmbH"),t.qZA()(),t.TgZ(149,"li")(150,"a",47),t._uU(151,"BioDiscovery\u2019s NxClinical"),t.qZA()(),t.TgZ(152,"li"),t._uU(153,"Diploid: Diagnosing rare diseases: "),t.TgZ(154,"a",48),t._uU(155," Moon"),t.qZA()(),t.TgZ(156,"li")(157,"a",49),t._uU(158,"Centogene"),t.qZA()(),t.TgZ(159,"li")(160,"a",50),t._uU(161,"SimulConsult"),t.qZA(),t.TgZ(162,"ul")(163,"li"),t._uU(164,"Michael M. Segal"),t.qZA(),t.TgZ(165,"li"),t._uU(166,"Ayman W. El-Hattab"),t.qZA(),t.TgZ(167,"li"),t._uU(168,"Nicole Nadeau"),t.qZA(),t.TgZ(169,"li"),t._uU(170,"Kristina Brumme"),t.qZA(),t.TgZ(171,"li"),t._uU(172,"Robert Sundel"),t.qZA()()(),t.TgZ(173,"li")(174,"a",51),t._uU(175,"Fabric Genomics"),t.qZA()(),t.TgZ(176,"li")(177,"a",52),t._uU(178,"Qiagen"),t.qZA()(),t.TgZ(179,"li")(180,"a",53),t._uU(181,"Track.Health"),t.qZA()(),t.TgZ(182,"li")(183,"a",54),t._uU(184,"Pryzm.Health"),t.qZA()()()(),t._UZ(185,"hr"),t.TgZ(186,"h2"),t._uU(187,"Phenotype-driven diagnostics"),t.qZA(),t.TgZ(188,"ul")(189,"li")(190,"a",55),t._uU(191,"eXtasy"),t.qZA()(),t.TgZ(192,"li")(193,"a",56),t._uU(194,"Phen-Gen"),t.qZA()(),t.TgZ(195,"li")(196,"a",57),t._uU(197,"PhenoVar"),t.qZA()(),t.TgZ(198,"li")(199,"a",58),t._uU(200,"Phevor"),t.qZA()()(),t.TgZ(201,"h3"),t._uU(202,"Patient Engagement and Self Phenotyping (ePRO, PROM, eConsent, eCRF)"),t.qZA(),t.TgZ(203,"ul")(204,"li")(205,"a",59),t._uU(206,"Track.Health"),t.qZA()()(),t._UZ(207,"hr"),t.TgZ(208,"h3"),t._uU(209,"Platforms for phenotype-driven analysis"),t.qZA(),t.TgZ(210,"ul")(211,"li")(212,"a",60),t._uU(213,"DECIPHER"),t.qZA(),t._uU(214,": Mapping the clinical genome."),t.qZA(),t.TgZ(215,"li")(216,"a",61),t._uU(217,"Phenopolis"),t.qZA(),t._uU(218,": An Open Platform for Harmonization & Analysis of Sequencing & Phenotype Data. "),t.qZA(),t.TgZ(219,"li")(220,"a",62),t._uU(221,"DDD"),t.qZA(),t._uU(222," Deciphering Developmental Disorders."),t.qZA(),t.TgZ(223,"li")(224,"a",63),t._uU(225,"PhenomeCentral"),t.qZA(),t._uU(226,": A repository for clinicians and scientists to find additional cases of the same rare disorder. "),t.qZA(),t.TgZ(227,"li")(228,"a",64),t._uU(229,"OMIM Explorer"),t.qZA()()(),t._UZ(230,"hr"),t.TgZ(231,"h3"),t._uU(232,"Phenotype entry and databasing"),t.qZA(),t.TgZ(233,"ul")(234,"li")(235,"a",65),t._uU(236,"Phenotips"),t.qZA(),t._uU(237,": a software tool for collecting and analyzing phenotypic information of patients with genetic disorders. "),t.qZA()(),t._UZ(238,"hr"),t.TgZ(239,"h3"),t._uU(240,"Network analysis"),t.qZA(),t.TgZ(241,"ul")(242,"li")(243,"a",66),t._uU(244,"WebGestalt"),t.qZA(),t._uU(245,": WEB-based GEne SeT AnaLysis Toolkit. "),t.qZA(),t.TgZ(246,"li")(247,"a",67),t._uU(248,"MORPHIN (Model Organisms Projected on a Human Integrated Gene Network)"),t.qZA()(),t.TgZ(249,"li")(250,"a",68),t._uU(251,"GREAT"),t.qZA()(),t.TgZ(252,"li")(253,"a",69),t._uU(254,"ToppGene"),t.qZA()(),t.TgZ(255,"li")(256,"a",70),t._uU(257,"Superfamily Structural classification of protein"),t.qZA()()(),t._UZ(258,"hr"),t.TgZ(259,"h3"),t._uU(260,"Terminology servers"),t.qZA(),t.TgZ(261,"ul")(262,"li")(263,"a",71),t._uU(264,"HeTOP"),t.qZA()()()()()()()),2&i&&(t.xp6(7),t.Q6J("ngForOf",f.organizations),t.xp6(1),t.Q6J("ngForOf",f.organizations),t.xp6(3),t.Q6J("ngIf",f.contributorsSource),t.xp6(1),t.Q6J("pageSizeOptions",t.DdM(4,w)))},dependencies:[n.sg,n.O5,e.lW,q.Hw,K.SP,K.uX,M.BZ,M.fO,M.as,M.w1,M.Dz,M.nj,M.ge,M.ev,M.XQ,M.Gk,V.NW,se.gM,m.xw,m.Wh,m.UT,m.yH,T.yS],styles:[".community[_ngcontent-%COMP%]   .pageTitle[_ngcontent-%COMP%]{border:0}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .contributors-table[_ngcontent-%COMP%]{padding:1.5rem}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .org-navigation[_ngcontent-%COMP%]{padding:2rem}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .org-navigation[_ngcontent-%COMP%]   button[_ngcontent-%COMP%]{margin:.5rem}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]{padding:1rem}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]{padding:2rem;text-align:center;cursor:pointer}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]   [_ngcontent-%COMP%]:hover{opacity:.8}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]   .member-icon-wrapper[_ngcontent-%COMP%]{text-align:center}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]   .member-name[_ngcontent-%COMP%]{margin:.5rem 0 0;vertical-align:middle}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]   .member-name[_ngcontent-%COMP%]   .mat-icon[_ngcontent-%COMP%]{height:14px;width:14px;font-size:14px;cursor:pointer}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .member[_ngcontent-%COMP%]   .member-role[_ngcontent-%COMP%]{text-transform:uppercase;font-size:1rem}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .organization-name[_ngcontent-%COMP%]{text-align:center;margin:1rem 0}.community[_ngcontent-%COMP%]   .quick-links[_ngcontent-%COMP%]   .organization[_ngcontent-%COMP%]   .organization-description[_ngcontent-%COMP%]{font-style:italic;font-size:1.2rem;color:#2f4f4f}"]}),a})()},{path:"funding",component:(()=>{class a{constructor(){}ngOnInit(){}}return a.\u0275fac=function(i){return new(i||a)},a.\u0275cmp=t.Xpm({type:a,selectors:[["app-funding"]],decls:70,vars:0,consts:[[1,"container","funding"],[1,"center","pageTitle"],["href","https://www.charite.de/","target","__blank"],["href","https://www.jax.org/","target","__blank"],["href","http://www.dfg.de/","target","__blank"],["href","http://www.erare.eu/","target","__blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/29425702","target","__blank"],["href","https://ec.europa.eu/programmes/horizon2020/"],["href","http://solve-rd.eu/","target","__blank"],["href","https://www.nih.gov/","target","__blank"],[1,"funding-thumbnails"],["fxLayout","row wrap","fxLayoutAlign","center"],["fxFlex","30","fxFlexOffset","10"],["href","http://www.dfg.de","target","__blank"],["src","assets/funding-images/DFG.jpg"],["href","https://www.bmbf.de/en/index.html","target","__blank"],["src","assets/funding-images/bmbf.png"],["src","assets/funding-images/solve.png"],["src","assets/funding-images/hipbi.png"],["src","assets/funding-images/NIH.jpg"],["href","https://ec.europa.eu/programmes/horizon2020/","target","__blank"],["src","assets/funding-images/H2020-Web.png"],["href"," https://www.charite.de/","target","__blank"],["src","assets/charite-logo.png"],["src","assets/jax_logo.png"]],template:function(i,f){1&i&&(t.TgZ(0,"div",0)(1,"h1",1),t._uU(2,"Funding and Grant Support"),t.qZA(),t.TgZ(3,"p"),t._uU(4,"The HPO project gratefully acknowledges funding from a number of sources since its inception in 2007."),t.qZA(),t.TgZ(5,"ul")(6,"li")(7,"a",2),t._uU(8,"Charit\xe9-Universit\xe4tsmedizin Berlin"),t.qZA(),t._uU(9," institutional funding. "),t.qZA(),t.TgZ(10,"li")(11,"a",3),t._uU(12,"The Jackson Laboratory"),t.qZA(),t._uU(13," institutional funding."),t.qZA(),t.TgZ(14,"li"),t._uU(15,"Deutsche Forschungsgemeinschaft ( "),t.TgZ(16,"a",4),t._uU(17,"DFG"),t.qZA(),t._uU(18," ): 2007-2013 (DFG-Projektnummer 103545043). "),t.qZA(),t.TgZ(19,"li"),t._uU(20," Bundesministerium fur Bildung und Forschung ( "),t.TgZ(21,"a",5),t._uU(22,"E-RARE program"),t.qZA(),t._uU(23," ): "),t.TgZ(24,"ul")(25,"li")(26,"a",6),t._uU(27,"HIPBI-RD"),t.qZA(),t._uU(28,". HIPBI-RD has received funding from the European Union's Horizon 2020 Research and Innovation program under the ERA-NET Cofund Action n\xb0 643578. It was supported by the French National Research Agency (ANR), the German Federal Ministry of Education and Research (BMBF), German Research Foundation (DFG), and the Canadian Institutes for Health Research (CIHR) under the frame of E-Rare-3, the ERA-NET for research on rare diseases. "),t.qZA()()(),t.TgZ(29,"li"),t._uU(30," European Commission ("),t.TgZ(31,"a",7),t._uU(32,"Horizon 2020"),t.qZA(),t._uU(33,"): "),t.TgZ(34,"a",8),t._uU(35,"SOLVE-RD"),t.qZA()(),t.TgZ(36,"li"),t._uU(37," National Institutes of Health ("),t.TgZ(38,"a",9),t._uU(39,"NIH"),t.qZA(),t._uU(40,"): National Institutes of Health (NIH) Monarch Initiative [NIH OD #5R24OD011883]; Biomedical Data Translator, NCATS [OT3 TR002019-01S2]; Data Commons OD/NHBLI [1 OT OD025464-01; 1 OT3 HL142479-01] "),t.qZA()(),t.TgZ(41,"div",10)(42,"div",11)(43,"div",12)(44,"a",13),t._UZ(45,"img",14),t.qZA()(),t.TgZ(46,"div",12)(47,"a",15),t._UZ(48,"img",16),t.qZA()()(),t.TgZ(49,"div",11)(50,"div",12)(51,"a",8),t._UZ(52,"img",17),t.qZA()(),t.TgZ(53,"div",12)(54,"a",6),t._UZ(55,"img",18),t.qZA()()(),t.TgZ(56,"div",11)(57,"div",12)(58,"a",9),t._UZ(59,"img",19),t.qZA()(),t.TgZ(60,"div",12)(61,"a",20),t._UZ(62,"img",21),t.qZA()()(),t.TgZ(63,"div",11)(64,"div",12)(65,"a",22),t._UZ(66,"img",23),t.qZA()(),t.TgZ(67,"div",12)(68,"a",3),t._UZ(69,"img",24),t.qZA()()()()())},dependencies:[m.xw,m.Wh,m.UT,m.yH],styles:[".funding[_ngcontent-%COMP%]   ul[_ngcontent-%COMP%]{list-style-type:none;line-height:25px}.funding[_ngcontent-%COMP%]   .funding-thumbnails[_ngcontent-%COMP%]   img[_ngcontent-%COMP%]{max-height:150px;max-width:150px}.funding[_ngcontent-%COMP%]   .funding-thumbnails[_ngcontent-%COMP%]   .row[_ngcontent-%COMP%]{margin-bottom:15px}.funding[_ngcontent-%COMP%]   .funding-thumbnails[_ngcontent-%COMP%]   img[_ngcontent-%COMP%]:hover{opacity:.6}"]}),a})()},{path:"data",loadChildren:()=>Promise.resolve().then(r.bind(r,7734)).then(a=>a.DataModule)},{path:"resources",loadChildren:()=>Promise.resolve().then(r.bind(r,7475)).then(a=>a.ResourcesModule)},{path:"tools",loadChildren:()=>Promise.resolve().then(r.bind(r,9908)).then(a=>a.ToolsModule)},{path:"disclaimer",component:Y}]);let ee=(()=>{class a{}return a.\u0275fac=function(i){return new(i||a)},a.\u0275mod=t.oAB({type:a}),a.\u0275inj=t.cJS({imports:[T.Bz,L,T.Bz]}),a})();var Ze=r(9908),be=r(7770),ye=r(2474),Te=r(886),Ae=r(7734),Ue=r(7475);let we=(()=>{class a{}return a.\u0275fac=function(i){return new(i||a)},a.\u0275mod=t.oAB({type:a}),a.\u0275inj=t.cJS({providers:[Te.o,X],imports:[n.ez,Ae.DataModule,Ze.ToolsModule,Ue.ResourcesModule,ye.c,be.t,ee]}),a})()},9908:(W,C,r)=>{r.r(C),r.d(C,{ToolsModule:()=>D});var n=r(6895),T=r(3917),o=r(4650),t=r(1576);const S=T.Bz.forChild([{path:"tools",children:[{path:"phenomizer",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-phenomizer"]],decls:13,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","http://compbio.charite.de/phenomizer","target","_blank"],["href","https://www.ncbi.nlm.nih.gov/pubmed/19800049","target","_blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/tools-images/phenomizer.png",1,"static-images"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"Phenomizer"),o.qZA(),o.TgZ(3,"p"),o._uU(4," The Phenomizer is a "),o.TgZ(5,"a",2),o._uU(6,"web-based application"),o.qZA(),o._uU(7,". for clinical diagnostics in human genetics using semantic similarity searches in ontologies "),o.TgZ(8,"a",3),o._uU(9,"K\xf6hler et al., AJHG, October 2009"),o.qZA(),o._uU(10,". A detailed manual can be found in the online app. "),o.qZA(),o.TgZ(11,"div",4),o._UZ(12,"img",5),o.qZA()())},dependencies:[t.xw,t.Wh]}),h})()},{path:"phenogramviz",component:(()=>{class h{constructor(){this.pageTitle="Tools / PhenoGramViz",this.pageIntro=""}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-phenogramviz"]],decls:17,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://www.ncbi.nlm.nih.gov/pubmed/25280750/","target","_blank"],["href","https://phenomics.github.io/software-phenoviz.html","target","_blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/tools-images/phenogramviz.jpg",1,"static-images"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"PhenoGramViz"),o.qZA(),o.TgZ(3,"p"),o._uU(4," Clinical evaluation of CNVs identified via techniques such as array comparative genome hybridisation (aCGH) involves the inspection of lists of known and unknown duplications and deletions with the goal of distinguishing pathogenic from benign CNVs. "),o.TgZ(5,"a",2),o._uU(6,"PhenoGramViz"),o.qZA(),o._uU(7," is a tool that provides a phenotype-based approach to analyzing candidate CNVs on the basis of phenotype matching. "),o.qZA(),o.TgZ(8,"p"),o._uU(9," A key step in this process is the comparison of the individual's phenotypic abnormalities with those associated with Mendelian disorders of the genes affected by the CNV. PhenoGramViz additionally implements a comparison of model organism phenotypes with patient phenotypes in order to support the identification of novel disease genes and medical evaluation of CNVs. "),o.qZA(),o.TgZ(10,"ul")(11,"li")(12,"a",3),o._uU(13,"PhenoGramViz"),o.qZA(),o._uU(14,": Automatically analyze and visualize gene-to-phenotype relations for a set of genes affected by CNV of a patient and a set of HPO-terms "),o.qZA()(),o.TgZ(15,"div",4),o._UZ(16,"img",5),o.qZA()())},dependencies:[t.xw,t.Wh]}),h})()},{path:"exomiser",component:(()=>{class h{constructor(){this.pageTitle="Tools / Exomiser",this.pageIntro=""}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-exomiser"]],decls:18,vars:0,consts:[[1,"container"],[1,"pageTitle","center"],["href","https://www.ncbi.nlm.nih.gov/pubmed/24162188","target","_blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/tools-images/exomiser1.jpg",1,"static-images"],["src","assets/tools-images/exomiser2.jpg",1,"static-images"],["target","__blank","href","https://monarchinitiative.org/page/exomes"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"Exomiser"),o.qZA(),o.TgZ(3,"p"),o._uU(4," Whole-exome sequencing (WES) has revolutionized research into novel disease-gene discovery However, many cases remain unsolved due to the sheer number of candidate variants remaining after common filtering strategies such as removing low quality and common variants and those deemed unlikely to be pathogenic. "),o.TgZ(5,"a",2),o._uU(6,"Exomiser"),o.qZA(),o._uU(7," filters a whole-exome data set by removing off-target, common, and synonymous variants from further consideration and evaluates the remaining variants based on the predicted pathogenicity and minor allele frequency (variant score). Optionally, an assumed mode of inheritance is used to further filter genes with variants present in a pattern compatible with the assumed mode of inheritance (e.g., homozygous or compound heterozygous for autosomal recessive). These genes are then assigned a phenotypic relevance score based on comparison with mouse and zebrafish models with mutations in orthologous genes. The final ranking is calculated as the sum of the individual scores to yield the PHIVE score. "),o.qZA(),o.TgZ(8,"div",3),o._UZ(9,"img",4),o.qZA(),o.TgZ(10,"p"),o._uU(11,"The Exomiser uses a cross-species phenotype matching algorithm. The user enters a human phenotype as a list of HPO terms. All genes with variants that survive the initial filtering steps are then screened for mouse models with phenotypic to the human disease. "),o.qZA(),o.TgZ(12,"div",3),o._UZ(13,"img",5),o.qZA(),o.TgZ(14,"p"),o._uU(15,"The Exomiser is available as a downloadable Java application Exomiser. An online demonstration is available at the "),o.TgZ(16,"a",6),o._uU(17,"Monarch Initiative website"),o.qZA()()())},dependencies:[t.xw,t.Wh]}),h})()},{path:"genomiser",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-genomiser"]],decls:12,vars:0,consts:[["fxLayout","column",1,"container"],[1,"center","pageTitle"],["href","http://exomiser.monarchinitiative.org/exomiser/","target","_blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/tools-images/genomiser.jpg"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"Genomiser"),o.qZA(),o.TgZ(3,"p"),o._uU(4," The interpretation of non-coding variants still constitutes a major challenge in the application of whole-genome sequencing in Mendelian disease, especially for single-nucleotide and other small non-coding variants. The Genomiser, an analysis framework that is able not only to score the relevance of variation in the non-coding genome, but also to associate regulatory variants to specific Mendelian diseases. Genomiser scores variants through either existing methods such as CADD or a bespoke machine learning method and combines these with allele frequency, regulatory sequences, chromosomal topological domains, and phenotypic relevance to discover variants associated to specific Mendelian disorders. "),o.qZA(),o.TgZ(5,"p"),o._uU(6,"The Genomiser is available as part of the downloadable Java application "),o.TgZ(7,"a",2),o._uU(8,"Exomiser"),o.qZA(),o._uU(9,". "),o.qZA(),o.TgZ(10,"div",3),o._UZ(11,"img",4),o.qZA()())},dependencies:[t.xw,t.Wh]}),h})()},{path:"external",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-externalhpo"]],decls:125,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],[1,"row"],["href","https://www.fdna.com/","target","_blank"],["href","https://cartagenia.com/","target","_blank"],["href","http://gepado.de/en/","target","_blank"],["href","http://www.biodiscovery.com/nxclinical/","target","_blank"],["href","http://www.diploid.com/moon","target","__blank"],["href","https://www.centogene.com/","target","__blank"],["href","http://simulconsult.com/index.html","target","__blank"],["href","https://www.fabricgenomics.com/","target","_blank"],["href","http://resources.qiagenbioinformatics.com/white-papers/IVA_white_paper_FINAL.pdf"],["href","https://track.health/"],["href","https://pryzm.health/"],["href","http://extasy.esat.kuleuven.be/","target","_blank"],["href","http://phen-gen.org/","target","_blank"],["href","http://phenovar-dev.udes.genap.ca/","target","_blank"],["href","http://www.yandell-lab.org/software/phevor.html","target","_blank"],["href","https://track.health/","target","_blank"],["href","https://decipher.sanger.ac.uk/","target","_blank"],["href","https://phenopolis.org/","target","_blank"],["href","https://www.ddduk.org/","target","_blank"],["href","https://phenomecentral.org/","target","_blank"],["href","http://omimexplorer.research.bcm.edu","target","_blank"],["href","https://phenotips.org/","target","_blank"],["href","http://www.webgestalt.org/option.php","target","_blank"],["href","http://www.inetbio.org/morphin/","target","_blank"],["href","http://great.stanford.edu/public/html/","target","_blank"],["href","https://toppgene.cchmc.org/","target","_blank"],["href","http://supfam.org/SUPERFAMILY/","target","_blank"],["href","http://www.hetop.eu/hetop/","target","_blank"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"External Tools"),o.qZA(),o.TgZ(3,"p"),o._uU(4,"The HPO project welcomes other groups to use the HPO resources to develop tools and algorithms. This page has a list of academic and commercial tools that use the HPO. We invite feedback about omissions or errors in this list."),o.qZA(),o._UZ(5,"br"),o.TgZ(6,"h3"),o._uU(7,"Commerical entitites"),o.qZA(),o.TgZ(8,"div",2)(9,"ul")(10,"li")(11,"a",3),o._uU(12,"FDNA"),o.qZA(),o._uU(13,": Face2Gene."),o.qZA(),o.TgZ(14,"li")(15,"a",4),o._uU(16,"Cartagenia BENCH"),o.qZA()(),o.TgZ(17,"li")(18,"a",5),o._uU(19,"GEPADO \u2013 Software Solutions for Genetics - GmbH"),o.qZA()(),o.TgZ(20,"li")(21,"a",6),o._uU(22,"BioDiscovery\u2019s NxClinical"),o.qZA()(),o.TgZ(23,"li"),o._uU(24,"Diploid: Diagnosing rare diseases: "),o.TgZ(25,"a",7),o._uU(26," Moon"),o.qZA()(),o.TgZ(27,"li")(28,"a",8),o._uU(29,"Centogene"),o.qZA()(),o.TgZ(30,"li")(31,"a",9),o._uU(32,"SimulConsult"),o.qZA()(),o.TgZ(33,"li")(34,"a",10),o._uU(35,"Fabric Genomics"),o.qZA()(),o.TgZ(36,"li")(37,"a",11),o._uU(38,"Qiagen"),o.qZA()(),o.TgZ(39,"li")(40,"a",12),o._uU(41,"Track.Health"),o.qZA()(),o.TgZ(42,"li")(43,"a",13),o._uU(44,"Pryzm.Health"),o.qZA()()()(),o._UZ(45,"hr"),o.TgZ(46,"h2"),o._uU(47,"Phenotype-driven diagnostics"),o.qZA(),o.TgZ(48,"ul")(49,"li")(50,"a",14),o._uU(51,"eXtasy"),o.qZA()(),o.TgZ(52,"li")(53,"a",15),o._uU(54,"Phen-Gen"),o.qZA()(),o.TgZ(55,"li")(56,"a",16),o._uU(57,"PhenoVar"),o.qZA()(),o.TgZ(58,"li")(59,"a",17),o._uU(60,"Phevor"),o.qZA()()(),o.TgZ(61,"h3"),o._uU(62,"Patient Engagement and Self Phenotyping (ePRO, PROM, eConsent, eCRF)"),o.qZA(),o.TgZ(63,"ul")(64,"li")(65,"a",18),o._uU(66,"Track.Health"),o.qZA()()(),o._UZ(67,"hr"),o.TgZ(68,"h3"),o._uU(69,"Platforms for phenotype-driven analysis"),o.qZA(),o.TgZ(70,"ul")(71,"li")(72,"a",19),o._uU(73,"DECIPHER"),o.qZA(),o._uU(74,": Mapping the clinical genome."),o.qZA(),o.TgZ(75,"li")(76,"a",20),o._uU(77,"Phenopolis"),o.qZA(),o._uU(78,": An Open Platform for Harmonization & Analysis of Sequencing & Phenotype Data. "),o.qZA(),o.TgZ(79,"li")(80,"a",21),o._uU(81,"DDD"),o.qZA(),o._uU(82," Deciphering Developmental Disorders."),o.qZA(),o.TgZ(83,"li")(84,"a",22),o._uU(85,"PhenomeCentral"),o.qZA(),o._uU(86,": A repository for clinicians and scientists to find additional cases of the same rare disorder. "),o.qZA(),o.TgZ(87,"li")(88,"a",23),o._uU(89,"OMIM Explorer"),o.qZA()()(),o._UZ(90,"hr"),o.TgZ(91,"h3"),o._uU(92,"Phenotype entry and databasing"),o.qZA(),o.TgZ(93,"ul")(94,"li")(95,"a",24),o._uU(96,"Phenotips"),o.qZA(),o._uU(97,": a software tool for collecting and analyzing phenotypic information of patients with genetic disorders. "),o.qZA()(),o._UZ(98,"hr"),o.TgZ(99,"h3"),o._uU(100,"Network analysis"),o.qZA(),o.TgZ(101,"ul")(102,"li")(103,"a",25),o._uU(104,"WebGestalt"),o.qZA(),o._uU(105,": WEB-based GEne SeT AnaLysis Toolkit. "),o.qZA(),o.TgZ(106,"li")(107,"a",26),o._uU(108,"MORPHIN (Model Organisms Projected on a Human Integrated Gene Network)"),o.qZA()(),o.TgZ(109,"li")(110,"a",27),o._uU(111,"GREAT"),o.qZA()(),o.TgZ(112,"li")(113,"a",28),o._uU(114,"ToppGene"),o.qZA()(),o.TgZ(115,"li")(116,"a",29),o._uU(117,"Superfamily Structural classification of protein"),o.qZA()()(),o._UZ(118,"hr"),o.TgZ(119,"h3"),o._uU(120,"Terminology servers"),o.qZA(),o.TgZ(121,"ul")(122,"li")(123,"a",30),o._uU(124,"HeTOP"),o.qZA()()()())}}),h})()},{path:"workbench",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-other"]],decls:14,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],["href","https://github.com/TheJacksonLaboratory/HPOworkbench","target","_blank"],["fxLayout","row","fxLayoutAlign","center"],["src","assets/tools-images/HPOworkbench.png",1,"static-images"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2," HPO Workbench "),o.qZA(),o.TgZ(3,"p"),o._uU(4," HPO Workbench is a Java app designed as a browser for HPO terms and annotated diseases. HPO Workbench has a series of functions that can be used by collaborators to review and make suggestions for extending, correcting, or otherwise revising the HPO or the disease annotations. "),o.qZA(),o.TgZ(5,"p"),o._uU(6," HPO Workbench may also be of interest for browsing and exploring the HPO and disease annotation data. "),o.qZA(),o.TgZ(7,"p"),o._uU(8,"The source code of HPO Workbench as well as an executable file can be downloaded from the "),o.TgZ(9,"a",2),o._uU(10,"Project repository"),o.qZA(),o._uU(11,". "),o.qZA(),o.TgZ(12,"div",3),o._UZ(13,"img",4),o.qZA()())},dependencies:[t.xw,t.Wh]}),h})()},{path:"hpo-browser",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-hpobrowser"]],decls:29,vars:0,consts:[[1,"container","other-browsers"],[1,"center","pageTitle"],["fxLayout","row wrap"],["fxFlex","80","fxFlexOffset","10","fxFlex.gt-sm","30","fxFlexOffset.gt-sm","15"],["href","https://www.ebi.ac.uk/ols/ontologies/hp","target","__blank"],["src","assets/other-browser-images/ols.png",1,"static-images"],["href","http://www.ontobee.org/ontology/HP","target","__blank"],["src","assets/other-browser-images/ontobee.png",1,"static-images"],["target","__blank","href","https://bioportal.bioontology.org/ontologies/HP"],["src","assets/other-browser-images/bioportal.png",1,"static-images"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"Other HPO Browsers"),o.qZA(),o.TgZ(3,"div",2)(4,"div",3)(5,"h3"),o._uU(6,"The Ontology Lookup Service (OLS)"),o.qZA(),o.TgZ(7,"p"),o._uU(8," The "),o.TgZ(9,"a",4),o._uU(10,"OLS"),o.qZA(),o._uU(11," is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions. You can browse the ontologies through the website as well as programmatically via the OLS API. "),o.qZA(),o._UZ(12,"img",5),o.qZA(),o.TgZ(13,"div",3)(14,"h3"),o._uU(15,"Ontobee"),o.qZA(),o.TgZ(16,"p")(17,"a",6),o._uU(18,"Ontobee"),o.qZA(),o._uU(19," is a linked-data server designed for ontologies. Ontobee is aimed to facilitate ontology data sharing, visualization, query, integration, and analysis."),o.qZA(),o._UZ(20,"img",7),o.qZA(),o.TgZ(21,"div",3)(22,"h3"),o._uU(23,"BioPortal"),o.qZA(),o.TgZ(24,"p")(25,"a",8),o._uU(26,"BioPortal"),o.qZA(),o._uU(27," is the ontology browser of the National Center for Biomedical Ontology."),o.qZA(),o._UZ(28,"img",9),o.qZA()()())},dependencies:[t.xw,t.UT,t.yH],styles:[".other-browsers[_ngcontent-%COMP%]   .static-images[_ngcontent-%COMP%]{width:100%;height:300px;box-shadow:0 1px 5px}"]}),h})()},{path:"loinc2hpo",component:(()=>{class h{constructor(){}ngOnInit(){}}return h.\u0275fac=function(p){return new(p||h)},h.\u0275cmp=o.Xpm({type:h,selectors:[["app-loinc"]],decls:35,vars:0,consts:[[1,"container"],[1,"center","pageTitle"],[1,"subtitle"],["href","https://npjdigitalmedcommunity.nature.com/users/256992-aaron-zhang/posts/48542-integrating-laboratory-tests-for-deep-phenotyping-and-biomarker-discovery","target","_blank"],[1,"material-icons"],["href","https://loinc.org/","target","_blank"],["href","https://hpo.jax.org/app/","target","_blank"],["src","assets/hpo-diagram.png",1,"loinc-img"],["href","https://www.hl7.org/fhir/index.html","target","_blank"],["href","https://www.i2b2.org/","target","_blank"],["href","https://www.ohdsi.org/data-standardization/the-common-data-model/","target","_blank"],["src","assets/loinc2hpo.png",1,"loinc-img"]],template:function(p,g){1&p&&(o.TgZ(0,"div",0)(1,"h1",1),o._uU(2,"LOINC 2 HPO"),o.qZA(),o.TgZ(3,"h3",2),o._uU(4,"Integrating laboratory tests for deep phenotyping and biomarker discovery "),o.TgZ(5,"a",3)(6,"i",4),o._uU(7," launch "),o.qZA()()(),o.TgZ(8,"p"),o._uU(9,"Electronic health records (EHRs) have been widely adopted in US hospitals since the last decade, generating a vast amount of data waiting to be explored. Clinical laboratory tests are one of the key components of EHR. Because of the objectivity of laboratory tests and the sheer volume that each person can accumulate in routine health care, they are a gold mine for extracting detailed phenotypic profiles of patient, a practice that we referred to \u201cdeep phenotyping\u201d. Laboratory tests are uniquely identified with "),o.TgZ(10,"a",5),o._uU(11,"Laboratory Observation Identifier Names and Codes"),o.qZA(),o._uU(12," (LOINC), which is a universal code system that defines various kinds of clinical laboratory tests and other measurements. The entire LOINC database have more than 86,000 codes, however, multiple codes are used to represent different ways of measuring the same metabolite. For example, there are five different types of laboratory tests that measure nitrite in urine, which differ in the kind of assays utilized, output formats, or the output units. Although such technical details are useful for uniquely identifying the laboratory tests, they pose a challenge if one hopes to combine them for statistical analysis as they require domain expertise and are hard to scale."),o.qZA(),o.TgZ(13,"p"),o._uU(14,"Our group has contributed to patient phenotyping by developing the "),o.TgZ(15,"a",6),o._uU(16,"Human Phenotype Ontology"),o.qZA(),o._uU(17," (HPO), which is a vocabulary for describing patient abnormal phenotypes. HPO currently (April 2019) contains ~14,000 terms, many of them represent the medical implications of clinical laboratory tests, such as Hypokalemia and Nitrituria. The terms form a hierarchical tree structure, where the root terms are more generic and the leaf terms are more specific. According to the hierarchical structure of the HPO, if a patient is annotated to a certain phenotype such as Sinus venosus atrial septal defect, then that patient is inferred to have all of the more general ancestor terms such as Atrial septal defect and so on. In addition, HPO also contains disease annotations for all rare and many common diseases. HPO has been adopted by organizations such as the 100,000 Genomes Project, the NIH Undiagnosed Diseases Network and Project, and many others and become the de facto standard for computational genomic diagnosis. In the current paper, we represent laboratory test results with HPO terms, thereby integrating similar laboratory tests together and extracting detailed patient profiles at the same time."),o.qZA(),o._UZ(18,"img",7),o.TgZ(19,"p"),o._uU(20,"The essential component of our approach is an accurate map between laboratory test results to their medical implications in HPO terms. For instance, a test for the concentration of potassium in the blood (LOINC:6298-4). If the result is high, our procedure infers the corresponding HPO term for Hyperkalemia (HP:0002153). Analogously, a low result is mapped to Hypokalemia (HP:0002900). The HPO is an ontology of abnormal phenotypes, and thus there is no term that specifically represents a normal test result. However, computational analysis can record negated HPO terms, and the normal test result is represented as NOT Abnormal blood potassium concentration (HP:0011042). Other types of laboratory tests, such as ordinal type that have negative or positive findings, or nominal type that have a list of potential findings, are also mapped to their corresponding HPO terms. We have a team of multiple MDs and biomedical PhDs to ensure that we achieve the highest possible quality of mappings."),o.qZA(),o.TgZ(21,"p"),o._uU(22,"With the mapping in hand, our next step was to devise an algorithm that checks the value of a laboratory test with the reference ranges and then find the correct HPO term for the corresponding result. Because laboratory tests are reported in different formats depending on vendors, we targeted toward laboratory tests encoded as observations in the "),o.TgZ(23,"a",8),o._uU(24,"FHIR"),o.qZA(),o._uU(25," format, which is vendor-independent standards for transmitting EHR data. However, it is quite feasible to adapt our software to other formats, such as "),o.TgZ(26,"a",9),o._uU(27,"i2b2"),o.qZA(),o._uU(28," or "),o.TgZ(29,"a",10),o._uU(30,"OMOP"),o.qZA(),o._uU(31,"."),o.qZA(),o._UZ(32,"img",11),o.TgZ(33,"p"),o._uU(34,"In our study, we also demonstrated our approach with an EHR dataset on 15,681 de-identified patients with asthma or asthma-like symptoms. We were able to transform 88.6% into HPO terms. On average, each patient were assigned to 633 HPO term-encoded phenotypes, which collapsed to 57 unique HPO terms because patients were tested multiple times with the same laboratory test over the tracking period. Using a logistic regression approach, we identified numerous HPO terms significantly associated with frequent prednisone prescriptions and acute asthma diagnosis, including many terms related to well known prednisone side effects as well as many published asthma biomarkers. Because our approach is very generic, it can be applied to many readily available dataset sitting in data warehouses and EHR systems to study other diseases of interest. Our library can also be coupled with more sophisticated statistical and machine-learning approaches to perform phenotype-driven analysis."),o.qZA()())},styles:[".loinc-img[_ngcontent-%COMP%]{max-width:500px;width:100%;margin:auto;display:block}.subtitle[_ngcontent-%COMP%]   .material-icons[_ngcontent-%COMP%]{font-size:16px!important}"]}),h})()}]}]);let I=(()=>{class h{}return h.\u0275fac=function(p){return new(p||h)},h.\u0275mod=o.oAB({type:h}),h.\u0275inj=o.cJS({imports:[T.Bz,S,T.Bz]}),h})();var B=r(2474);let D=(()=>{class h{}return h.\u0275fac=function(p){return new(p||h)},h.\u0275mod=o.oAB({type:h}),h.\u0275inj=o.cJS({imports:[n.ez,I,B.c]}),h})()}}]);