<div class="container faq" fxLayout="column wrap">
  <h1 class="center pageTitle">Frequently Asked Questions</h1>
  <br>
  <div class="center">
    <a mat-button href="https://github.com/TheJacksonLaboratory/hpo-web/issues/new" target="__blank" class="faq-btn">Request
      a Feature or Report a Bug</a>
  </div>
  <div class="faq-container" fxLayout="row wrap">
    <div fxFlex="100" fxFlex.gt-sm="50" fxFlexOffset.gt-sm="25">
      <h3>
        Ontology
      </h3>
      <mat-accordion>
        <mat-expansion-panel [expanded]="openPanel == 1" id="hpo-relation">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How is HPO related to the Monarch Initiative?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>The HPO is the clinical window of the <a href="https://monarchinitiative.org/" target="__blank">Monarch
            Initiative</a> which is an NIH supported project that integrates, aligns, and re-distributes
            cross-species gene, genotype, variant, disease, and phenotype data. <a href="https://monarchinitiative.org/"
                                                                                   target="__blank">The Monarch Browser</a>
            allows users to browse not only human diseases but to see their links to other organisms, genes, phenotypes, and
            pathways, which
            may be of especial interest to translational researchers.</p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 2" id="content-change">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>Am I allowed to change the contents of HPO?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>The HPO is a community-driven project. We request that all proposed changes be suggested on our <a
            target="__blank" href="https://github.com/obophenotype/human-phenotype-ontology/issues">GitHub tracker</a> so
            that they will be of use to all users of the HPO. </p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 3" id="collaboration">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How to collaborate with the HPO ?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>We welcome the participation of interested colleagues. We anticipate that the structure of the HPO will
            continue to be refined and completed for some time to come.
            Groups or persons with expert knowledge in a particular domain of human phenotyping in a medical genetics
            setting are invited to contribute their knowledge on a collaborative basis.
            Please contact us to discuss details.</p>
          <p>We are currently transitioning from annotations derived in a semiautomated fashion from the OMIM database to
            manually curated annotations. Given the number of human hereditary diseases,
            this is no small task. We encourage participation of colleagues with an interest in specific hereditary diseases
            to improve the annotations for ‘their’ disease. Please get in contact with us to discuss this form of
            collaboration.
            We are also preparing an online system for manual annotations.</p>
          <p>The issue tracker is <a href="https://github.com/obophenotype/human-phenotype-ontology/issues"
                                     target="__blank">available on Github.</a></p>
          <h4>Collaborations for translations</h4>
          <p>We are working with several people to translate more content of HPO into a multitude of languages. Several
            translations are being actively developed at the moment (e.g. german).</p>
          <p>The translations are considered a project coordinated by the HPO team and are hosted on <a
            href="https://github.com/Phenomics/HPO-translations" target="__blank">GitHub</a>. Please do not start
            translating HPO content without contacting us beforehand.
            This might cause duplicated translations and other problems. </p>
          <p> Please contact us <a routerLink="/contact">here</a>.</p>
        </mat-expansion-panel>
        <mat-expansion-panel>
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>What is the medical focus of the Human Phenotype Ontology?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>
            The medical focus of the HPO in its initial decade (2007-2017) was on rare, mainly Mendelian diseases. The
            construction of the initial version of the HPO in
            2007/2008 was performed by generating an ontology based on descriptions in the Clinical Synopsis of the
            <a href="https://omim.org/" target="__blank">Online Mendelian Inheritance in Man (OMIM) database</a>. Since the
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/18950739" target="__blank">initial publication of the HPO in 2008</a>,
            the HPO
            team has held regular workshops with clinicians to refine and extend the clinical terminology of the HPO in specific
            areas
            such as cardiology or immunology. We have added textual definitions, computational logical definitions, and roughly
            6,000 new terms since then.
          </p>
          <p>
            The focus of the HPO will be extended to other areas of medicine in coming years. A <a
            href="https://www.ncbi.nlm.nih.gov/pubmed/26119816" target="__blank">pilot project</a>
            on common-disease annotations was published in 2015.
          </p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 4" id="ontology">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>What is an ontology?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>
            A terminology contains a list of items (often called "terms") that represent the concepts of a domain. An
            ontology is a terminology that additionally specifies formal semantic relationships of concepts.
            For instance, a terminology of wines would include a list of items such as Chardonnay, Cabernet Sauvignon,
            Rioja, Riesling, Sauvignon blanc, etc., but an ontology of wine might have a term for "red wine" and "Rioja"; It
            also can specify that "Rioja" is a type of "red wine".
            Ontologies and terminologies can be used as tools for standardizing and exchanging data. Ontologies such as the
            HPO typically enable sophisticated computational algorithms that exploit the semantic relations between terms.
          </p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 5" id="susceptibility">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How does the HPO project model susceptibility?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>
            The word susceptibility can be defined as a state of being liable to be influenced or harmed by a particular
            thing. In genetics, the word susceptibility can be used with a variety of meanings.
          <ol>
            <li>To refer to an allele that confers some (usually small) increased risk of developing a polygenic (complex)
              disease. A disease such as Asthma for instance currently has 733 such loci listed
              in <a href="https://www.ebi.ac.uk/gwas/efotraits/EFO_0000270" target="_blank">GWAS catalog</a>, as well as ten
              entries in <a href="https://omim.org/entry/600807" target="_blank">OMIM</a>. The
              HPO project does not consider these "susceptibility" entries in OMIM to be separate diseases.
            </li>
            <li>
              Similarly, some congenital anomalies such as Cleft palate are polygenic diseases, with over 150 loci in <a
              href="https://www.ebi.ac.uk/gwas/efotraits/HP_0000175" target="_blank">GWAS catalog</a>. These
              loci are treated analogously.
            </li>
            <li>Susceptibility to a cancer that is inherited as a Mendelian trait; the classic example is <a
              href="https://omim.org/entry/604370" target="_blank">BRCA1</a>. The susceptibility associated with
              a pathogenic BRCA1 mutation is substantially higher than the susceptibility associated with an allele related
              to a polygenic disease; the susceptibility is also transmitted as a Mendelian trait.
              The clinical aspects may differ in important ways for distinct cancer susceptibility loci. For instance, the
              risk of ovarian cancer is higher for BRCA1 mutation carriers than for BRCA2 mutation carriers (
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/28632866" target="_blank">Kuchenbaecker et al, 2017</a>).
            </li>
            <li>Inherited susceptibility to certain kinds of infectious disease may occur as a Mendelian trait. For example,
              WHIM syndrome confers suceptibility to human papilloma virus infection induced warts,
              condyloma acuminata and carcinomas (<a href="https://www.ncbi.nlm.nih.gov/pubmed/19057201/" target="_blank">Kawai
                and Malech, 2009</a>). Susceptibility to infections may also be a polygenic trait. The HPO
              models Mendelian traits as diseases, e.g., <a href="https://hpo.jax.org/app/browse/disease/OMIM:193670"
                                                            target="_blank">WHIM syndrome</a>).
            </li>
            <li>In some cases, susceptibility to diseases such as neurodegeneration can be inherited as a Mendelian trait,
              including Huntington disease, some forms of Parkinson disease, and Alzheimer disease.
              <a href="https://hpo.jax.org/app/browse/disease/OMIM:146500" target="_blank">Mutliple system atrophy</a> may
              be occur sporadically or in rare cases be inherited as a Mendelian trait. For this reason,
              HPO models this as a distinct disease.
            </li>
          </ol>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 5" id="ontology-structure">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How are the ontology terms structured?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>
            Most ontologies are structured as directed acyclic graphs (DAG), which are similar to hierarchies but differ in that
            a more specialized term (child)
            can be related to more than one less specialized term (parent).
            Cycles (cyclic paths in the graph) are not allowed. The relationship of the terms of the HPO to
            one another is displayed in the DAG. For instance, the term <a routerLink="/browse/term/HP:0001964">Aplasia/Hypoplasia
            of metatarsal bones</a> is a child of both
            <a routerLink="/browse/term/HP:0006494">Aplasia/Hypoplasia involving bones of the feet</a> and <a
            routerLink="/browse/term/HP:0001964">Abnormal metatarsal morphology</a>. The ability to encode multiple
            parents in a DAG adds to the flexibility and descriptiveness of the ontology.
            This would not be possible with a simple hierarchical system. The <strong>&nbsp;is-a&nbsp;</strong> relationship is
            transitive, meaning that
            annotations are inherited up all paths to the root. For instance,
            <a routerLink="/browse/term/HP:0001711">Abnormal morphology of the left ventricle</a>
            <strong>&nbsp;is-a&nbsp;</strong>
            <a routerLink="/browse/term/HP:0001713">Abnormal cardiac ventricle morphology</a>.
          </p>
          <p>
            Each term in the HPO describes a clinical abnormality. These may be general terms, such as <em>Abnormal ear
            morphology</em> or
            very specific ones such as <em>Chorioretinal atrophy</em>.
            The terms have a unique ID such as HP:0001140 and a label such as Epibulbar dermoid.
            Most terms have textual definitions such as An epibulbar dermoid is a benign tumor typically found at the
            junction of the cornea and sclera (limbal epibullar dermoid). The source of the definition must be indicated.
            Many terms have synonyms. For instance, Epibulbar dermoids is taken to be a synonym of Epibulbar dermoids.</p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 6" id="ontology-research">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>What is an ontology and how is it used for medical genetics research?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>
            The word ontology is derived from Greek words meaning the study of existence and being. More recently, the word
            ontology has been used in computer
            science to describe systems that describe concepts within some domain and relationships
            between those concepts. For instance, the <a href="http://www.geneontology.org/" target="__blank">Gene Ontology
            consortium</a>
            has developed an extensive ontology describing molecular functions, biological processes,
            and cellular locations over the last decade and a number of groups have supplied annotations
            using the GO terms to gene products of many organisms. The study of human phenotypes in the context of hereditary
            and
            common disease has the potential to lead to great insight on the function of genes and genetic networks.
            The HPO provides computational resources that allow large-scale computational analysis of the human phenome.
          </p>
        </mat-expansion-panel>
      </mat-accordion>
      <h3>
        Annotation
      </h3>
      <mat-accordion>
        <mat-expansion-panel [expanded]="openPanel == 6" id="annotation-use">
          <mat-expansion-panel-header>
            <mat-panel-title>
              <strong>How do I use HPO Annotations?</strong>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>The terms of the HPO describe phenotypic abnormalities and do not directly describe diseases. Instead, the HPO
            project uses so-called annotations to describe the connection between diseases and HPO terms. For instance, the
            disease,
            <a routerLink="/browse/disease/ORPHA:171719">Cutis laxa-Marfanoid syndrome </a> is annotated by the HPO terms
            for <a routerLink="/browse/term/HP:0002827">Hip dislocation</a>, <a routerLink="/browse/term/HP:0001371">Flexion
              contracture</a> and many more.</p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 7" id="genes-phenotypes">
          <mat-expansion-panel-header>
            <mat-panel-title>
            <span fxHide fxShow.gt-sm>
              <strong>What is the difference between genes_to_phenotypes.txt and phenotype_to_genes.txt?</strong>
            </span>
              <span fxShow fxHide.gt-sm>
                 <strong>genes_to_phenotypes.txt and phenotype_to_genes.txt?</strong>
            </span>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>A question that is asked very often, deals with the asymmetry of the files:</p>
          <ul>
            <li><i>genes_to_phenotypes.txt</i></li>
            <li><i>phenotype_to_genes.txt</i></li>
          </ul>
          <p>The difference is that one file uses the transitivity of the annotations and the other does not. I.e. in <i>genes_to_phenotypes.txt</i>
            we list for each gene the most specific HPO-classes (and not all the ancestors).
            In <i>phenotype_to_genes.txt</i> we show each phenotype that has at least one gene associated with it.
            Additionally, we have mapped genes to the ancestor classes of each associated phenotype. For example given that:
          </p>
          <ul>
            <li>HP_1 subclass_of HP_3</li>
            <li>HP_1 subclass_of HP_3</li>
          </ul>
          <p>then <i>genes_to_phenotypes.txt</i></p>
          <ul>
            <li><strong>geneA</strong> annotated to HP_1</li>
            <li><strong>geneB</strong> annotated to HP_2</li>
          </ul>
          <p>then <i>phenotype_to_genes.txt</i> contains:</p>
          <ul>
            <li>HP_3 annotates <strong>geneA</strong> and <strong>geneB</strong></li>
          </ul>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 8" id="frequencies">
          <mat-expansion-panel-header>
            <mat-panel-title>
            <span fxHide fxShow.gt-sm>
              <strong>What is the difference between “ALL_FREQUENCIES” and “FREQUENT_FEATURES”?</strong>
            </span>
              <span fxShow fxHide.gt-sm>
                 <strong >“ALL_FREQUENCIES” vs “FREQUENT_FEATURES”?</strong>
            </span>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>Each annotation has a frequency associated, i.e. how many patients with that disease have this clinical
            feature. “ALL_FREQUENCIES” denotes that we do not use any filter.
            “FREQUENT_FEATURES” means that we exclude all features seen in less than 50% of the patients.</p>
        </mat-expansion-panel>
        <mat-expansion-panel [expanded]="openPanel == 9" id="association-rules">
          <mat-expansion-panel-header>
            <mat-panel-title>
            <span fxHide fxShow.gt-sm>
              <strong>What is your procedure for associating genes with HPO phenotypes?</strong>
            </span>
              <span fxShow fxHide.gt-sm>
                 <strong>Genes to Phenotypes Procedure?</strong>
            </span>
            </mat-panel-title>
          </mat-expansion-panel-header>
          <p>The following file is used to associate genes with phenotypes.</p>
          <ul>
            <li>
              <i>genes_to_phenotypes.txt</i>
            </li>
          </ul>
          <p>
            In many cases, there are multiple diseases that are associated with mutations in the same gene (for instance,
            mutations in the LMNA gene are associated with multiple diseases including Cardiomyopathy,
            dilated, 1A [MIM:115200], Charcot-Marie-Tooth disease, type 2B1 [605588], Emery-Dreifuss muscular dystrophy 2,
            AD [181350], and Hutchinson-Gilford progeria [176670]. Each of these diseases is associated with a
            different spectrum of phenotypic features. The gene-to-phenotype association in our databases takes the set of
            all phenotypic features (HPO terms) associated with one or more diseases caused by mutation in the gene under
            consideration. Thus, LMNA is associated with Alopecia (which is a feature of Hutchinson-Gilford progeria) as
            well as Ventricular arrhythmia (which is a feature of Cardiomyopathy, dilated, 1A).
          </p>
        </mat-expansion-panel>
      </mat-accordion>
    </div>
  </div>
</div>
